<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.16">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>TBD</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">TBD</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p> </p>
          </div>
  </div>
    
    
  </div>
  
<div>
  <div class="abstract">
    <div class="abstract-title">Abstract</div>
    <p>This is the abstract for the paper.</p>
  </div>
</div>

</header>

<section id="introduction" class="level1">
<h1>Introduction</h1>
<!-- basic introduction to the field, comprehensible to a scientist in any discipline -->
<p>Dental calculus is quickly becoming the go-to substance for exploring health and diet in past populations. Studies using archaeological dental calculus span a wide range of topics in different regions and time periods. These include characterisation of the oral microbiome and its evolution in past populations <span class="citation" data-cites="velskoMicrobialDifferences2019 adlerSequencingAncient2013 warinnerPathogensHost2014 kazarinaPostmedievalMicrobial2021 yatesOralMicrobiome2021">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al. 2019</a>; <a href="#ref-adlerSequencingAncient2013" role="doc-biblioref">Adler et al. 2013</a>; <a href="#ref-warinnerPathogensHost2014" role="doc-biblioref">Warinner et al. 2014</a>; <a href="#ref-kazarinaPostmedievalMicrobial2021" role="doc-biblioref">Kazarina et al. 2021</a>; <a href="#ref-yatesOralMicrobiome2021" role="doc-biblioref">Fellows Yates et al. 2021</a>)</span>, extraction of microbotanical remains <span class="citation" data-cites="henryCalculusSyria2008 hardyStarchGranulesDental2009 mickleburghNewInsights2012 maHumanDiet2022">(<a href="#ref-henryCalculusSyria2008" role="doc-biblioref">Henry and Piperno 2008</a>; <a href="#ref-hardyStarchGranulesDental2009" role="doc-biblioref">Hardy et al. 2009</a>; <a href="#ref-mickleburghNewInsights2012" role="doc-biblioref">Mickleburgh and Pagán-Jiménez 2012</a>; <a href="#ref-maHumanDiet2022" role="doc-biblioref"><strong>maHumanDiet2022?</strong></a>)</span> and other residues to infer dietary patterns and nicotine-use <span class="citation" data-cites="buckleyDentalCalculus2014 hendyProteomicCalculus2018 eerkensDentalCalculus2018">(<a href="#ref-buckleyDentalCalculus2014" role="doc-biblioref">Buckley et al. 2014</a>; <a href="#ref-hendyProteomicCalculus2018" role="doc-biblioref">Hendy et al. 2018</a>; <a href="#ref-eerkensDentalCalculus2018" role="doc-biblioref">Eerkens et al. 2018</a>)</span>.</p>
<!-- more detailed background, comprehensible to scientists in related disciplines -->
<!-- dental plaque/calculus formation and calculus composition -->
<p>Dental calculus is formed by the mineralisation of dental plaque. Dental plaque is an oral biofilm and is part of the normal state of the oral cavity; however, if left unchecked, plaque can lead to infections such as dental caries and periodontitis <span class="citation" data-cites="marshDentalPlaque2006">(<a href="#ref-marshDentalPlaque2006" role="doc-biblioref">Marsh 2006</a>)</span>. Shortly after teeth are cleaned (whether mechanically or otherwise), a salivary pellicle adsorbs to the surface of the tooth, in most cases enamel, forming the acquired dental pellicle. The pellicle is comprised mainly of proteins and, in addition to protecting the tooth against mechanical and chemical decay, provides a viable surface for bacterial attachment <span class="citation" data-cites="yaoIdentificationProtein2003">(<a href="#ref-yaoIdentificationProtein2003" role="doc-biblioref"><strong>yaoIdentificationProtein2003?</strong></a>)</span>. Shortly after adsorbing to the enamel, early-coloniser species, such as those within genus <em>Streptococcus</em> and <em>Actinomyces</em>, adhere to the pellicle through reversible long-range physicochemical forces and irreversible short range cell-host<!--check--> interactions <span class="citation" data-cites="marshDentalPlaque2006">(<a href="#ref-marshDentalPlaque2006" role="doc-biblioref">Marsh 2006</a>)</span>. Once the surface has been populated by the specialists in surface-attachment, other species of bacteria can attach to their surface through cell-cell interactions, allowing adhesion of species that are not otherwise capable of adhering to a surface. After accumulation and multiplication of bacteria, this, now, diverse community of is able to secrete polysaccharides, proteins, lipids, and nucleic acids, into their immediate environment to form a biofilm matrix <span class="citation" data-cites="flemmingBiofilmsEmergent2016">(<a href="#ref-flemmingBiofilmsEmergent2016" role="doc-biblioref"><strong>flemmingBiofilmsEmergent2016?</strong></a>)</span>. The matrix provides an adaptive advantage to the organisms within through resistance to antibiotics and mechanical removal, as well as transporting nutrients from outside the biofilm and facilitating distribution of resources between bacterial communities within the biofilm <span class="citation" data-cites="petersonViscoelasticityBiofilms2015 jainIsolationCharacterization2013">(<a href="#ref-petersonViscoelasticityBiofilms2015" role="doc-biblioref"><strong>petersonViscoelasticityBiofilms2015?</strong></a>; <a href="#ref-jainIsolationCharacterization2013" role="doc-biblioref"><strong>jainIsolationCharacterization2013?</strong></a>)</span>.</p>
<p>The composition of a biofilm matrix is largely water (around 90%), with the remaining content consisting of microbes, extracellular polysaccharides (EPS), DNA, RNA, and proteins <span class="citation" data-cites="bergerOralBiofilms2018">(<a href="#ref-bergerOralBiofilms2018" role="doc-biblioref"><strong>bergerOralBiofilms2018?</strong></a>)</span>. Biofilms can become susceptible to calcification under certain microenvironmental conditions. These include an increased concentration of salts and a decrease in statherin and proline-rich proteins in saliva, rises in local plaque pH, and increased hydrolysis of urea <span class="citation" data-cites="whiteDentalCalculus1997 wongCalciumPhosphate2002">(<a href="#ref-whiteDentalCalculus1997" role="doc-biblioref"><strong>whiteDentalCalculus1997?</strong></a>; <a href="#ref-wongCalciumPhosphate2002" role="doc-biblioref"><strong>wongCalciumPhosphate2002?</strong></a>)</span>. Under these conditions, the biofilm environment becomes favourable to increased precipitation and decreased dissolution of calcium phosphate salts within saliva and the plaque biofilm. The resulting supersaturation of calcium phosphate salts are the main drivers of biofilm mineralisation <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref"><strong>jinSupragingivalCalculus2002?</strong></a>)</span>. Mineralisation generally starts from within the biofilm matrix as a result of nucleation, followed by mineralisation of the matrix and, subsequently, bacterial cells. The susceptibility of crystallisation in bacteria depends on the composition and concentration of membrane-associated components, such as proteolipids and phospholipids <span class="citation" data-cites="jinSupragingivalCalculus2002 whiteDentalCalculus1997">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref"><strong>jinSupragingivalCalculus2002?</strong></a>; <a href="#ref-whiteDentalCalculus1997" role="doc-biblioref"><strong>whiteDentalCalculus1997?</strong></a>)</span>. Binding of calcium to bacterial membranes is facilitated by phospholipid molecules within the cell membrane, followed by association of phosphates with the bound calcium to form calcium phosphate complexes. These complexes are active in promoting the formation and deposition of hydroxyapatite within biofilms <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref"><strong>jinSupragingivalCalculus2002?</strong></a>)</span>.</p>
<p>The primary minerals in dental calculus are hydroxyapatite (HAP), octacalcium phosphate (OCP), whitlockite (WHT), and brushite (BHT). During initial mineralisation the main mineral component is BHT, which shifts to HAP in more mature dental calculus <span class="citation" data-cites="jinSupragingivalCalculus2002 hayashizakiSiteSpecific2008">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref"><strong>jinSupragingivalCalculus2002?</strong></a>; <a href="#ref-hayashizakiSiteSpecific2008" role="doc-biblioref"><strong>hayashizakiSiteSpecific2008?</strong></a>)</span>. The exact elemental composition of dental calculus varies by individual due to various factors, including diet <span class="citation" data-cites="hayashizakiSiteSpecificMineral2008 jiFluorideMagnesium2000">(<a href="#ref-hayashizakiSiteSpecificMineral2008" role="doc-biblioref"><strong>hayashizakiSiteSpecificMineral2008?</strong></a>; <a href="#ref-jiFluorideMagnesium2000" role="doc-biblioref"><strong>jiFluorideMagnesium2000?</strong></a>)</span>.</p>
<p>The role of bacteria in dental calculus formation is still not clear, and dental calculus formation has been induced in bacteria-free rodents <span class="citation" data-cites="glasBiophysicalStudies1962">(<a href="#ref-glasBiophysicalStudies1962" role="doc-biblioref"><strong>glasBiophysicalStudies1962?</strong></a>)</span>. However, given the abundance of bacteria present within human dental plaque, the structure of calculus will reflect the presence of bacteria with human dental calculus containing a heterogenous mineral composition across the biofilm due to the differing mineralisation properties in bacteria, and also directly influences the porosity of calculus <span class="citation" data-cites="omelonReviewPhosphate2013 rohanizadehUltrastructuralStudy2005">(<a href="#ref-omelonReviewPhosphate2013" role="doc-biblioref"><strong>omelonReviewPhosphate2013?</strong></a>; <a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref"><strong>rohanizadehUltrastructuralStudy2005?</strong></a>)</span>. <!-- rephrase: need to emphasise the importance of bacteria in human dental calculus - better proxy --> <!-- biofilm maturation and mineralisation --> <!-- dental calculus composition and distribution --> The different susceptibility of certain bacteria to mineralise may explain differences in bacterial profiles in plaque and dental calculus <span class="citation" data-cites="velskoMicrobialDifferences2019">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al. 2019</a>)</span></p>
<p>Attachment of dental calculus to enamel is further solidified by fusion of dental calculus with enamel rods <span class="citation" data-cites="rohanizadehUltrastructuralStudy2005 whiteDentalCalculus1997">(<a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref"><strong>rohanizadehUltrastructuralStudy2005?</strong></a>; <a href="#ref-whiteDentalCalculus1997" role="doc-biblioref"><strong>whiteDentalCalculus1997?</strong></a>)</span>.</p>
<p>The organic component of dental calculus consists of proteins and lipids, likely incorporated from bacteria, saliva, and food <span class="citation" data-cites="whiteDentalCalculus1997">(<a href="#ref-whiteDentalCalculus1997" role="doc-biblioref"><strong>whiteDentalCalculus1997?</strong></a>)</span>. Calculus forming above the gumline (supragingival) has a lower inorganic content than calculus forming below the gumline (subgingival) <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref"><strong>jinSupragingivalCalculus2002?</strong></a>)</span>.</p>
<!-- oral biofilm models. History, types, etc. -->
<p>Oral biofilm models are commonly used in dental research to assess the efficacy of certain treatments on dental pathogens <span class="citation" data-cites="filochePlaqueMicrocosm2007 extercateAAA2010">(<a href="#ref-filochePlaqueMicrocosm2007" role="doc-biblioref"><strong>filochePlaqueMicrocosm2007?</strong></a>; <a href="#ref-extercateAAA2010" role="doc-biblioref"><strong>extercateAAA2010?</strong></a>)</span>. These are often short-term models grown over a few days, but there also exist longer term models used to develop dental calculus <span class="citation" data-cites="middletonVitroCalculus1965 sissonsMultistationPlaque1991 wongCalciumPhosphate2002">(<a href="#ref-middletonVitroCalculus1965" role="doc-biblioref"><strong>middletonVitroCalculus1965?</strong></a>; <a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al. 1991</a>; <a href="#ref-wongCalciumPhosphate2002" role="doc-biblioref"><strong>wongCalciumPhosphate2002?</strong></a>)</span>. There are multiple different types of models ranging from simplistic agar plate or multiwell-plate models <span class="citation" data-cites="ceriCalgaryBiofilmDevice1999 extercateAAA2010">(<a href="#ref-ceriCalgaryBiofilmDevice1999" role="doc-biblioref"><strong>ceriCalgaryBiofilmDevice1999?</strong></a>; <a href="#ref-extercateAAA2010" role="doc-biblioref"><strong>extercateAAA2010?</strong></a>)</span>, to more complex setups like the constant depth film fermenter (CDFF) <span class="citation" data-cites="petersConstantDepth1988">(<a href="#ref-petersConstantDepth1988" role="doc-biblioref"><strong>petersConstantDepth1988?</strong></a>)</span> and the multi-station artificial mouth (MAM) <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al. 1991</a>)</span>. The more complex models have the benefit of a continuous flow of saliva or saliva-like medium and control over the environment, while the multiwell-plate models offer the advantage of generating more samples over the same amount of time <span class="citation" data-cites="mcbainBiofilmModels2009">(<a href="#ref-mcbainBiofilmModels2009" role="doc-biblioref"><strong>mcbainBiofilmModels2009?</strong></a>)</span>. Simplistic models restricted to a select subset of oral bacteria are often more reproducible than models using whole saliva, while the latter are more representative of the <em>in vivo</em> oral microbiome complexity <span class="citation" data-cites="roderStudyingBacterial2016 mcbainBiofilmModels2009">(<a href="#ref-roderStudyingBacterial2016" role="doc-biblioref"><strong>roderStudyingBacterial2016?</strong></a>; <a href="#ref-mcbainBiofilmModels2009" role="doc-biblioref"><strong>mcbainBiofilmModels2009?</strong></a>)</span>.</p>
<!-- clearly stating the general problem being addressed by this particular study -->
<p>We present an oral biofilm model that can serve as a viable proxy for dental calculus, and provide a method for fundamental research on dental calculus in the past. The need for such a model is warranted by the different questions that are asked by archaeologists compared to clinical dentistry. We are interested in learning more about how dietary residues and microremains become trapped in calculus, and how the methods we use may inadvertently bias our interpretations; questions that are best addressed in a lab using a model. We used FTIR to verify the mineral composition, and metagenomic classification to characterise the bacterial composition, and compared our results against modern and archaeological human dental calculus. We found that the mineral and organic components mimic that of the modern reference calculus used for comparison, while the bacterial classification revealed a similar but distinct community structure.<!--The bacterial composition --> In addition to the benefit of increased control over parameters involved in calculus formation and dietary incorporation, our method also provides unlimited material for experimentation, rather than using the limited archaeological material currently available.</p>
</section>
<section id="materials-and-methods" class="level1">
<h1>Materials and Methods</h1>
</section>
<section id="materials-and-methods-1" class="level1">
<h1>Materials and methods</h1>

<!-- Create table for all samples and indicate which samples underwent DNA, FTIR, etc. -->
<section id="biofilm-growth" class="level2">
<h2 class="anchored" data-anchor-id="biofilm-growth">Biofilm growth</h2>
<p>In this study we employ a multispecies oral biofilm model following a modified protocol from Sissons and colleagues <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">1991</a>)</span> and Shellis <span class="citation" data-cites="shellisSyntheticSalivaCultural1978">(<a href="#ref-shellisSyntheticSalivaCultural1978" role="doc-biblioref">1978</a>)</span>. The setup comprises a polypropylene 24 deepwell PCR plate (KingFisher 97003510) with a lid containing 24 pegs (substrata), which is autoclaved at 120°C, 1 bar overpressure, for 20 mins.</p>
<p>The artificial saliva (AS) is a modified version of the basal medium mucin (BMM) described by Sissons and colleagues <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">1991</a>)</span>. It is a complex medium containing 2.5 g/l partially purified mucin from porcine stomach (Type III, Sigma M1778), 5 g/l trypticase peptone (Roth 2363.1), 10 g/l proteose peptone (Oxoid LP0085), 5 g/l yeast extract (BD 211921), 2.5 g/l KCl, 0.35 g/l NaCl, 1.8 mmol/l CaCl<sub>2</sub>, 5.2 mmol/l Na<sub>2</sub>HPO<sub>4</sub> <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al. 1991</a>)</span>, 6.4 mmol/l NaHCO<sub>3</sub> <span class="citation" data-cites="shellisSyntheticSalivaCultural1978">(<a href="#ref-shellisSyntheticSalivaCultural1978" role="doc-biblioref">Shellis 1978</a>)</span>, 2.5 mg/l haemin. This is subsequently adjusted to pH 7 with NaOH pellets and stirring, autoclaved (15 min, 120°C, 1 bar overpressure), and supplemented with 5.8 (mu)mol/l menadione, 5 mmol/l urea, and 1 mmol/l arginine <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al. 1991</a>)</span>.</p>
<!-- add to protocols.io -->
<p>Fresh whole saliva (WS) for inoculation was provided by a 31-year-old male donor with no history of caries, who abstained from oral hygiene for 24 hours, and no food was consumed two hours prior to donation. No antibiotics were taken up to six months prior to donation. The saliva was filtered through a sterilised (with bleach) nylon cloth to remove particulates. Substrata were inoculated with 1 ml/well of a two-fold dilution of WS in sterilised 20% glycerine for four hours at 36°C, to allow attachment of the salivary pellicle and plaque-forming bacteria. After initial inoculation, the substrata were transferred to a new plate containing 1 ml/well AS and incubated at 36°C, 30 rpm. The inoculation process was repeated on days 3 and 5. AS was partially refreshed once per day and fully refreshed every three days, throughout the experiment, by transferring the substrata to a new plate containing AS. To feed the bacteria, the substrata were transferred to a new plate, containing 5% (w/v) sucrose, for six minutes twice daily, except on inoculation days (days 0, 3, and 5), where the samples only received one sucrose treatment after inoculation.</p>
<p>Starch treatments were initiated on day 9 to avoid starch granule counts being affected by <span class="math inline">\(alpha\)</span>-amylase hydrolysis from inoculation saliva. An <span class="math inline">\(\alpha\)</span>-amylase assay confirmed the absence of any <span class="math inline">\(\alpha\)</span>-amylase activity in the system <span class="citation" data-cites="bartholdyInvestigatingBiases2021">(<a href="#ref-bartholdyInvestigatingBiases2021" role="doc-biblioref"><strong>bartholdyInvestigatingBiases2021?</strong></a>)</span>. Starch treatments replaced sucrose treatments, occurring twice per day for six minutes. The starch treatments involved transferring the substrata to a new plate containing a 0.25% (w/v) starch from potato (Roth 9441.1) solution, a 0.25% (w/v) starch from wheat (Sigma S5127) solution, and a 0.5% (w/v) mixture of equal concentrations (w/v) wheat and potato. All starch solutions were created in a 5% (w/v) sucrose solution. Before transferring biofilm samples to the starch treatments, the starch plates were agitated to keep the starches in suspension in the solutions, and during treatments, the rpm was increased to 60.</p>
<p>After 15 days, mineralisation was encouraged with a calcium phosphate monofluorophosphate urea (CPMU) solution containing 20 mmol/l CaCl<sub>2</sub>, 12 mmol/l NaH<sub>2</sub>PO<sub>4</sub>, 5 mmol/l Na<sub>2</sub>PO<sub>3</sub>F, 500 mmol/l Urea <span class="citation" data-cites="pearceConcomitantDepositionStrontium1987 sissonsMultistationPlaque1991">(<a href="#ref-pearceConcomitantDepositionStrontium1987" role="doc-biblioref">Pearce and Sissons 1987</a>; <a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al. 1991</a>)</span>, and (0.04 g/l MgCl). The substrata were submerged in 1 ml/well CPMU five times daily, every two hours, for six minutes. During the mineralisation period, starch treatments were reduced to once per day after the five CPMU treatments. This cycle was repeated for 10 days until the end of the experiment on day 24 (see @ref(fig:protocol-fig) for an overview of protocol). A more detailed protocol is available at<!-- protocols.io -->.</p>
<p>All laboratory work was conducted in sterile conditions under a laminar flow hood to prevent starch and bacterial contamination. Control samples were included to detect starch contamination.</p>
</section>
<section id="metagenomics" class="level2">
<h2 class="anchored" data-anchor-id="metagenomics">Metagenomics</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 36 × 2
   `#SampleID`  Env            
   &lt;chr&gt;        &lt;chr&gt;          
 1 LIB030.A0117 library control
 2 SYN001.A0101 saliva         
 3 SYN002.A0101 saliva         
 4 SYN003.A0101 saliva         
 5 SYN014.A0101 medium         
 6 SYN014.C0101 medium         
 7 SYN015.B0101 medium         
 8 SYN015.D0101 medium         
 9 SYN015.E0101 medium         
10 SYN015.F0101 medium         
# … with 26 more rows</code></pre>
</div>
</div>
<!-- Samples sent to Jena -->
<p>A total of 36 samples were taken during the experiment from the donated saliva, artificial saliva, and from the biofilm end-product on day 24. DNA extraction was performed at the archaeogenetic facility at the Max Planck Institute for the Science of Human History (Jena, Germany). Extractions were performed in duplicates<!-- ?? -->. A total of <!-- sample size --> DNA extracts.</p>
<!-- DNA extraction details -->
<p>DNeasy PowerSoil Kit from QIAGEN. C2 inhibitor removal step skipped, going directly to C3 step.</p>
<p>were paired-end sequenced on a NextSeq (2 color chemistry) to 150bp</p>
<!-- EAGER settings -->
<section id="preprocessing" class="level3">
<h3 class="anchored" data-anchor-id="preprocessing">Preprocessing</h3>
<p>Processing of the raw DNA reads was conducted using the nf-core/eager, v2.4.4 pipeline <span class="citation" data-cites="yatesEAGER2020">(<a href="#ref-yatesEAGER2020" role="doc-biblioref">Fellows Yates et al. 2020</a>)</span>. Adapter removal and read merging was performed using AdapterRemoval, v2.3.2 <span class="citation" data-cites="AdapterRemovalv2">(<a href="#ref-AdapterRemovalv2" role="doc-biblioref">Schubert, Lindgreen, and Orlando 2016</a>)</span>. Merged reads were mapped to the human reference genome (GRCh38 <!--link-->) using BWA, v0.7.17-r1188 <span class="citation" data-cites="BWA">(<a href="#ref-BWA" role="doc-biblioref">Li and Durbin 2009</a>)</span> with default settings (-n 0.01; -l 32), and unmapped reads were extracted using Samtools, v1.12.</p>
<p>Metagenomic classification was conducted in kraken, v2.1.2 using the Standard database <span class="citation" data-cites="kraken2">(<a href="#ref-kraken2" role="doc-biblioref">Wood, Lu, and Langmead 2019</a>)</span>. <!-- cuperdec [@yatesOralMicrobiome2021]--></p>
<p>Reference samples downloaded from HMP, ENA, and using SRA Toolkit<!-- reference supp. mat.--> and processed in the same way as the artificial samples. Only paired reads were processed, singletons were removed.</p>
</section>
<section id="authentication" class="level3">
<h3 class="anchored" data-anchor-id="authentication">Authentication</h3>
<!-- sourcetracker -->
<p>Species with lower than 0.001% relative abundance were removed. SourceTracker <span class="citation" data-cites="knightsSourceTracker2011">(<a href="#ref-knightsSourceTracker2011" role="doc-biblioref"><strong>knightsSourceTracker2011?</strong></a>)</span> was used to estimate source composition of the oral biofilm model samples using a Bayesian framework. Sampes were compared with oral and environmental controls to detect potential external contamination. The R package <em>decontam</em> v1.16.0 <span class="citation" data-cites="R-decontam">(<a href="#ref-R-decontam" role="doc-biblioref">Davis et al. 2018</a>)</span> was used to identify potential contaminants using DNA concentrations with a probability threshold of 0.95 and negative controls with a probability threshold of 0.05. Putative contaminants were filtered out of the OTU tables for all downstream analyses. Authentication methods are described in more detail in the Supplementary material.<!--../06-reports/metagen-authentication.Rmd--></p>
</section>
<section id="community-composition" class="level3">
<h3 class="anchored" data-anchor-id="community-composition">Community composition</h3>
<p>Relative abundances of communities were calculated at the species- and genus-level, as recommended for compositional data <span class="citation" data-cites="gloorMicrobiomeDatasets2017">(<a href="#ref-gloorMicrobiomeDatasets2017" role="doc-biblioref"><strong>gloorMicrobiomeDatasets2017?</strong></a>)</span>. Shannon index calculated on species OTU tables of all artificial and oral reference samples using the vegan 2.6.2 R package <span class="citation" data-cites="Rvegan">(<a href="#ref-Rvegan" role="doc-biblioref"><strong>Rvegan?</strong></a>)</span>. Sparse principal components analysis (sPCA) was performed on model biofilm samples to … and a separate sPCA analysis was performed on biofilm model end-products and oral reference samples using the mixOmics R package 6.20.0 <span class="citation" data-cites="RmixOmics">(<a href="#ref-RmixOmics" role="doc-biblioref"><strong>RmixOmics?</strong></a>)</span>.</p>
</section>
<section id="differential-abundance" class="level3">
<h3 class="anchored" data-anchor-id="differential-abundance">Differential abundance</h3>
</section>
</section>
<section id="ftir" class="level2">
<h2 class="anchored" data-anchor-id="ftir">FTIR</h2>
<div class="cell" data-layout-align="center">

</div>
<p>To determine the mineral composition and level of crystallisation of the model dental calculus samples, we used Fourier Transform Infrared (FTIR) spectroscopy. We compared the spectra of model dental calculus with spectra archaeological dental calculus and used a built-in Omnic search library for mineral identification <span class="citation" data-cites="weinerInfraredSpectroscopy2010">(<a href="#ref-weinerInfraredSpectroscopy2010" role="doc-biblioref"><strong>weinerInfraredSpectroscopy2010?</strong></a>)</span>. The archaeological sample was dental calculus from an isolated tooth from Middenbeemster, a rural, 19th century Dutch site. Samples were analysed at the Laboratory for Sedimentary Archaeology, Haifa University. The analysis was conducted with a Thermo Nicolet is5 spectrometer in transmission, at 4 cm<span class="math inline">\(^{-1}\)</span> resolution, with an average of 32 scans between wavenumbers 4000 and 400 cm<span class="math inline">\(^{-1}\)</span>.</p>
<p>Analysis was conducted on 28 biofilm samples from days 7, 12, 16, 20, and 24. Some samples from the same sampling day had to be combined to provide enough material for analysis. Samples analysed for FTIR originated from a different experiment than the metagenomic samples, following the same protocol (as described above). Samples were analysed following the method presented in <span class="citation" data-cites="asscherAtomicDisorder2011">(<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref"><strong>asscherAtomicDisorder2011?</strong></a>)</span>. A few <span class="math inline">\(\mu\)</span>g of each sample were repeatedly ground together with KBr and pressed in a 7 mm die under two tons of pressure using a Specac mini-pellet press (Specac Ltd., GS01152). Repeated measurements of the splitting factor were taken after each grind, and a grind curve was produced following <span class="citation" data-cites="asscherAtomicDisorder2011">(<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref"><strong>asscherAtomicDisorder2011?</strong></a>)</span>. Samples were ground and analysed up to six times (sample suffix a-f) for the grinding curve. Grinding curves were prepared for samples from days 16, 20, and 24. No grind curves were produced for samples from days 7 and 12. These were largely composed of organics and proteins, and did not form enough carbonated hydroxtapatite for analysis. The splitting factor of carbonate hydroxyapatite was calculated using a macro script, following <span class="citation" data-cites="weinerStatesPreservation1990">(<a href="#ref-weinerStatesPreservation1990" role="doc-biblioref"><strong>weinerStatesPreservation1990?</strong></a>)</span>, by dividing the sum of the height of the absorptions at 603 cm<span class="math inline">\(^{-1}\)</span> and 567 cm<span class="math inline">\(^{-1}\)</span> in the height of the valley between them. Following <span class="citation" data-cites="asscherAtomicDisorder2011">(<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref"><strong>asscherAtomicDisorder2011?</strong></a>)</span> we plotted the IRSF against the full width at half maximum (FWHM) of the main absorption at 1035, and compared our grinding curves to the ones produced by <span class="citation" data-cites="asscherAtomicDisorder2011">(<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref"><strong>asscherAtomicDisorder2011?</strong></a>)</span>.</p>
<p>Splitting factors of the doublet, and FWHM of the main PO<sub>4</sub> peak at 1040 were calculated from the grinding measurements, and plotted against each other to create grinding curves to explore crystallinity (crystal size) and the order and disorder. Disorder is a steep slope and large FWHM.</p>
</section>
<section id="statistics" class="level2">
<h2 class="anchored" data-anchor-id="statistics">Statistics</h2>
<!-- R packages used -->
<p>Statistical analysis was conducted in R Statistical Software R version 4.2.0 (2022-04-22) (Vigorous Calisthenics) <span class="citation" data-cites="Rbase">(<a href="#ref-Rbase" role="doc-biblioref"><strong>Rbase?</strong></a>)</span>. Data cleaning and wrangling performed with packages from <strong>tidyverse</strong> <span class="citation" data-cites="tidyverse2019">(<a href="#ref-tidyverse2019" role="doc-biblioref"><strong>tidyverse2019?</strong></a>)</span>. Plots were created using ggplot2 <span class="citation" data-cites="Rggplot">(<a href="#ref-Rggplot" role="doc-biblioref"><strong>Rggplot?</strong></a>)</span>.</p>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="metagenomic-analysis" class="level2">
<h2 class="anchored" data-anchor-id="metagenomic-analysis">Metagenomic analysis</h2>
<section id="sample-authentication" class="level3">
<h3 class="anchored" data-anchor-id="sample-authentication">Sample authentication</h3>
<!-- SourceTracker -->
<p>The sources of taxa were estimated using SourceTracker2 <span class="citation" data-cites="knightsSourceTracker2011">(<a href="#ref-knightsSourceTracker2011" role="doc-biblioref"><strong>knightsSourceTracker2011?</strong></a>)</span>. The results suggest that the majority of taxa across samples have an oral microbial signature. SourceTracker2 results were compared to a database of oral taxa <span class="citation" data-cites="fellowsyatesEvolutionChanging2021">(<a href="#ref-fellowsyatesEvolutionChanging2021" role="doc-biblioref"><strong>fellowsyatesEvolutionChanging2021?</strong></a>)</span> to prevent removal of samples where oral taxa were assigned to a non-oral source, as some similar taxa with a signature from multiple sources are often classified as “Unknown” <span class="citation" data-cites="velskoMicrobialDifferences2019">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al. 2019</a>)</span>. A few samples suspected of containing a large proportion of contamination were removed (SYN015.F0101,SYN015.G0101,SYN015.H0101,SYN017.F0101,SYN017.G0101,SYN018.H0101,SYN013.I0101,SYN016.I0101). The removed samples were predominantly medium samples from later in the experiment, and a few calculus samples (see Supp mat).</p>
</section>
<section id="decrease-in-community-diversity-across-experiment" class="level3">
<h3 class="anchored" data-anchor-id="decrease-in-community-diversity-across-experiment">Decrease in community diversity across experiment</h3>
<!-- species diversity -->
</section>
<section id="lower-diversity-in-artificial-samples-than-oral-references" class="level3">
<h3 class="anchored" data-anchor-id="lower-diversity-in-artificial-samples-than-oral-references">Lower diversity in artificial samples than oral references</h3>
<!-- alpha diversity -->
<!-- beta diversity -->
</section>
</section>
<section id="ftir-1" class="level2">
<h2 class="anchored" data-anchor-id="ftir-1">FTIR</h2>
<div id="fig-experiment-spectra" class="cell quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-1.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(a) A</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-2.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(b) B</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-3" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-3.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(c) C</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-4" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-4.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(d) D</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure 1: Select spectra from all experiment sampling days.</figcaption><p></p>
</figure>
</div>
<!-- needs to be condensed!! -->
<p>Day 7 spectra have large O–H and amide A absorbance bands in stretching mode around 3400 cm<span class="math inline">\(^{-1}\)</span>, as well as three marked CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. There is a clear amide I peak at 1650 and a less pronounced amide II peak at 1545 cm<span class="math inline">\(^{-1}\)</span>. In the ‘fingerprint’ region, C–O<span class="math inline">\(_3^{2-}\)</span> at 1450 and 1400 absorbance bands corresponding to the v<sub>3</sub> asummetric stretching vibrations, P–O<sub>4</sub> absorbance band corresponding to the v<sub>3</sub> asymmetric stretching vibrations at 1080 cm<span class="math inline">\(^{-1}\)</span>, and minor phosphate absorption bands around 500 cm<span class="math inline">\(^{-1}\)</span> in sample F7.1A6, but absent in sample F7.2D1. The absorption bands at 1080 cm<span class="math inline">\(^{-1}\)</span>, and 1040-1047 cm<span class="math inline">\(^{-1}\)</span> and minor bending absorption bands of the phosphate doublet around 605 cm<span class="math inline">\(^{-1}\)</span> and 560 cm<span class="math inline">\(^{-1}\)</span> in sample F7.2D1, but absent in sample F7.1A6. The absorption bands at 1080 cm<span class="math inline">\(^{-1}\)</span> could be a C–N stretching mode from aliphatic amines, but may also come from silicate contamination (typical quartz doublet at 797, 780). The relative absorbance of O–H and Amide I and II bands are higher than the phosphate bands, representing a relatively higher content of lipids and proteins than inorganic content. Large variation between spectra.</p>
<p>Day 12, amide I and II continue to be the dominant peaks, and a higher ratio of both amide and O–H to PO<sub>4</sub> v<sub>3</sub> absorbance bands. Three marked CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. Reduced variation between two of the three spectra.</p>
<p>Day 16, the ratio of O–H and amides to PO<sub>4</sub> has shifted, with the main peak shifting to the PO<sub>4</sub> v<sub>3</sub> absorbance band at 1040 (except in sample F16.1A2). A well-defined PO<sub>4</sub> doublet at 600 and 560 is present. Small CO<span class="math inline">\(_3^{2-}\)</span> asymmetric stretching at 1450 cm<span class="math inline">\(^{-1}\)</span> and 1415 cm<span class="math inline">\(^{-1}\)</span>, and stretching vibrations at 875-870 cm<span class="math inline">\(^{-1}\)</span>. Decreased variability between the spectra, with most spectra exhibiting a higher phosphate-to-protein/lipid ratio.</p>
<p>Day 24, large O–H and amide A absorbance bands in stretching mode around 3400 cm<span class="math inline">\(^{-1}\)</span>, as well as three minor CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. Main peak of spectra is PO<sub>4</sub> v<sub>3</sub> at 1040 cm<span class="math inline">\(^{-1}\)</span>, well-defined PO<sub>4</sub> doublet at 600-550 cm<span class="math inline">\(^{-1}\)</span>. Amide I band, with small amide II and III bands. Carbonate peaks also present. Very little variation between all of the spectra.</p>
<p>The main difference in the samples across the experiment, with increasing age, is an increase in the main PO<sub>4</sub> v<sub>3</sub> peak at around 1040 cm<span class="math inline">\(^{-1}\)</span>, appearance of the phosphate doublet around 600 and 560 cm<span class="math inline">\(^{-1}\)</span> between day 12 and 16, an increase in the carbonate peaks at 1458-1450 cm<span class="math inline">\(^{-1}\)</span> and 1415-1420 cm<span class="math inline">\(^{-1}\)</span>, an increase in the ratio of the Amide I/II to peaks at around 1650 and 1540 cm<span class="math inline">\(^{-1}\)</span>, and a reduction of the C–H<sub>2</sub> and C–H<sub>3</sub> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>.</p>
<p>High lipid and protein content consistent with the presence of extracellular polysaccharides and bacteria within a matrix. Microbial DNA and RNA may be visible from peaks around 1200–800 cm<span class="math inline">\(^{-1}\)</span> on days 7 and 12, which are later obscured by the increasing phosphate peak at 1040 cm<span class="math inline">\(^{-1}\)</span> <span class="citation" data-cites="jainIsolationCharacterization2013">(<a href="#ref-jainIsolationCharacterization2013" role="doc-biblioref"><strong>jainIsolationCharacterization2013?</strong></a>)</span>. The presence of water indicated by the O–H stretch is also consistent with a biofilm, which is around 90% water <span class="citation" data-cites="bergerOralBiofilms2018">(<a href="#ref-bergerOralBiofilms2018" role="doc-biblioref"><strong>bergerOralBiofilms2018?</strong></a>)</span>. As the samples mature, the ratio of proteins and lipids to phosphates shift from predominance of organic content to inorganic content in the form of carbonated hydroxyapatite.</p>
<p>Early spectra, days 7 and 12, are also similar to collagen spectra, with the OH absorbance band at around 3400 cm<span class="math inline">\(^{-1}\)</span>, amide I, II, and III peaks at 1659, 1552, and 1240 cm<span class="math inline">\(^{-1}\)</span>, respectively <span class="citation" data-cites="rohanizadehUltrastructuralStudy2005 martinezcortizasLinkingStructural2020">(<a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref"><strong>rohanizadehUltrastructuralStudy2005?</strong></a>; <a href="#ref-martinezcortizasLinkingStructural2020" role="doc-biblioref"><strong>martinezcortizasLinkingStructural2020?</strong></a>)</span>.</p>
<p>The archaeological and modern reference spectra are largely indistinguishable and consist of an O–H absorbance band (3400 cm<span class="math inline">\(^{-1}\)</span>), CH<sub>3</sub> bands (3000–2900 cm<span class="math inline">\(^{-1}\)</span>), carbonate (1420, 1458-1450, 875-870 cm<span class="math inline">\(^{-1}\)</span>), amide I band (1650 cm<span class="math inline">\(^{-1}\)</span>), and phosphates (1040, 604, 566 cm<span class="math inline">\(^{-1}\)</span>).</p>
<p>The biofilm samples from the end of the experiment are similar to both reference samples. The main difference is a lower organic component in reference samples seen as a reduced amide I peak at around 1637 compared to the carbonate peak at around 1420, and an absence of amide II and III. Also a reduction in CH<sub>3</sub> bands at 3000-2900 cm<span class="math inline">\(^{-1}\)</span>.</p>
<section id="grinding-curves" class="level3">
<h3 class="anchored" data-anchor-id="grinding-curves">Grinding curves</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-grind-curve-inset" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-grind-curve-inset-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 2: (A) Grinding curves of multiple materials; and (B) calculus-only materials, including biofilm samples from three days, and an archaeological calclulus sample.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<!-- FTIR spectra comparison of artificial and reference dental calculus -->
<p>Samples were compared to the results of <span class="citation" data-cites="asscherAtomicDisorder2011">(<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref"><strong>asscherAtomicDisorder2011?</strong></a>)</span> and <span class="citation" data-cites="asscherVariationsAtomic2011">(<a href="#ref-asscherVariationsAtomic2011" role="doc-biblioref"><strong>asscherVariationsAtomic2011?</strong></a>)</span>, and the slopes of the trend lines for our model calculus are similar to those of fresh bone and dentin. No appreciable differences between days 16, 20, and 24. The archaeological dental calculus does show a slightly increased slope compared to model calculus from the three sampling days used in the grind curve.</p>
</section>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
</section>
<section id="discussion-1" class="level1">
<h1>Discussion</h1>
<!-- Big summary/important statement for the science -->
<p>In this study we present a [long-term] oral biofilm model with mineralisation to produce model dental calculus. The aim of the model is to address a variety of research questions related to dental calculus, both in the present and in the past. We also assess the viability of our model as a proxy for dental calculus.</p>
<p>used metagenomic classification and FTIR to in bacterial and mineral content. The performance of our model as a proxy for dental calculus was evaluated using metagenomic screening for bacterial composition, and FTIR for mineral structure. <!-- Scientific findings as extension of previous work --></p>
<!-- Findings' scientific implications -->
<!-- secondary findings -->
<!-- metagenomics -->
<p>While our model is not an anaerobic system, the anaerobes seem to have outcompeted aerobes and facultative anaerobes over the course of the experiment. This may be a result of biofilm microenvironments, where species can produce favourable environments within biofilms<!--cite-->.</p>
<!-- FTIR discussion-->
<p>Experimental samples are indicative of biofilm growth and maturation, including mineralisation, represented by a reduction in proteins and lipids (1460, 1420, 870) and the increasing intensity of peaks related to carbonated hydroxyapatite (1040, 600-550 cm<span class="math inline">\(^{-1}\)</span>) over the course of the experiment.</p>
<p>The presence of collagen in dental calculus (and saliva) is still not clear. It may be a result of external contamination <span class="citation" data-cites="mackiePreservationMetaproteome2017">(<a href="#ref-mackiePreservationMetaproteome2017" role="doc-biblioref"><strong>mackiePreservationMetaproteome2017?</strong></a>)</span> or end-products of collagen degradation (e.g.&nbsp;carboxyterminal telopeptide of type I collagen, ICTP) which may or may not be related to periodontitis <span class="citation" data-cites="gursoySalivaryCollagen2013">(<a href="#ref-gursoySalivaryCollagen2013" role="doc-biblioref"><strong>gursoySalivaryCollagen2013?</strong></a>)</span>. It is more likely that the proline-rich proteins (secondary amine) present in the samples are attributed to the biofilm matrix.</p>
<p>The artificial samples from day 24 resemble both the modern reference samples and the archaeological sample in mineral composition and crystallinity. The steeper slope of the archaeological sample may suggest larger crystals, which is to be expected, as the inorganic crystals expand into the space left by degrading organic matter over time <span class="citation" data-cites="weinerBiologicalMaterials2010">(<a href="#ref-weinerBiologicalMaterials2010" role="doc-biblioref"><strong>weinerBiologicalMaterials2010?</strong></a>)</span>. More samples are needed to see if this is consistent in archaeological samples. Overall, the spectra of the artificial calculus show an increase in the inorganic component and a decrease in the organic component over time. The grinding curves for days 16, 20, and 24 were very similar, suggesting that the CPMU solution (introduced on day 16) may not have much of an effect.</p>
<!-- How this engineering will facilitate future research -->
<!-- Forward-looking statements about the field as a whole -->
<!-- a summary/importance of the engineering aspect of the work -->
<!-- How it facilitates scientific research -->
<!-- limitations of the platform show which questions would be unanswered -->
<p>Research relating to <span class="math inline">\(alpha\)</span>-amylase activity and its effect on the oral biome (whether dietary or otherwise) requires addition of amylase to the microcosm, as it is not present in the naturally in the system (as shown in <!--cite starch paper-->). The reason for the absence of <span class="math inline">\(alpha\)</span>-amylase is not clear… There is of course no renewable source for <span class="math inline">\(alpha\)</span>-amylase once the inoculations have been completed; furthermore, there does not seem to be any retention of <span class="math inline">\(alpha\)</span>-amylase by amylase-binding streptococci (ABS). ABS are present in the system, but do not seem to be binding amylase. At this point we are unable to explain why. <!-- Is it possible that the mixture of sucrose in the starch solutions eliminates the need to feed on starch? --></p>
<!-- dominance of aerobes may affect interpretation of dietary studies 
Although samples were quite consitent within the experiment-->
<!-- BacDive limited info - more than 50% missing oxygen tolerance info-->
<!-- modern calculus references samples were all from the same geographic origin -->
<p>Single donor may have affected diversity of model. Donated saliva had a lower mean alpha-diversity than other saliva samples.</p>
<!-- limitations of FTIR and DNA analysis for validation of the model. -->
<!-- How it will facilitate the specific topic of research (e.g. archaeology) -->
<p>This method can be used for a wide range of methods-testing (e.g.&nbsp;DNA, proteomics, etc.) as well as training for various sampling methods, and contamination protocol testing. The model also has dietary applications, which can be seen elsewhere <!-- [bartholdy] -->.</p>

</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-adlerSequencingAncient2013" class="csl-entry" role="doc-biblioentry">
Adler, C. J., K. Dobney, L. S. Weyrich, J. Kaidonis, A. W. Walker, W. Haak, C. J. Bradshaw, et al. 2013. <span>“Sequencing Ancient Calcified Dental Plaque Shows Changes in Oral Microbiota with Dietary Shifts of the <span>Neolithic</span> and <span>Industrial</span> Revolutions.”</span> <em>Nature Genetics</em> 45 (4): 450–55, 455e1. <a href="https://doi.org/10.1038/ng.2536">https://doi.org/10.1038/ng.2536</a>.
</div>
<div id="ref-buckleyDentalCalculus2014" class="csl-entry" role="doc-biblioentry">
Buckley, Stephen, Donatella Usai, Tina Jakob, Anita Radini, and Karen Hardy. 2014. <span>“Dental <span>Calculus Reveals Unique Insights</span> into <span>Food Items</span>, <span>Cooking</span> and <span>Plant Processing</span> in <span>Prehistoric Central Sudan</span>.”</span> <em>PLOS ONE</em> 9 (7): e100808. <a href="https://doi.org/10.1371/journal.pone.0100808">https://doi.org/10.1371/journal.pone.0100808</a>.
</div>
<div id="ref-R-decontam" class="csl-entry" role="doc-biblioentry">
Davis, Nicole M., Diana M. Proctor, Susan P. Holmes, David A. Relman, and Benjamin J. Callahan. 2018. <span>“Simple Statistical Identification and Removal of Contaminant Sequences in Marker-Gene and Metagenomics Data.”</span> <em>Microbiome</em> 6 (1): 226. <a href="https://doi.org/10.1186/s40168-018-0605-2">https://doi.org/10.1186/s40168-018-0605-2</a>.
</div>
<div id="ref-eerkensDentalCalculus2018" class="csl-entry" role="doc-biblioentry">
Eerkens, Jelmer W., Shannon Tushingham, Korey J. Brownstein, Ramona Garibay, Katherine Perez, Engel Murga, Phil Kaijankoski, Jeffrey S. Rosenthal, and David R. Gang. 2018. <span>“Dental Calculus as a Source of Ancient Alkaloids: <span>Detection</span> of Nicotine by <span>LC-MS</span> in Calculus Samples from the <span>Americas</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 18 (April): 509–15. <a href="https://doi.org/10.1016/j.jasrep.2018.02.004">https://doi.org/10.1016/j.jasrep.2018.02.004</a>.
</div>
<div id="ref-yatesEAGER2020" class="csl-entry" role="doc-biblioentry">
Fellows Yates, James A., Thiseas C. Lamnidis, Maxime Borry, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U. Garcia, Judith Neukamm, and Alexander Peltzer. 2020. <span>“Reproducible, Portable, and Efficient Ancient Genome Reconstruction with Nf-Core/Eager.”</span> <em>bioRxiv</em>, October, 2020.06.11.145615. <a href="https://doi.org/10.1101/2020.06.11.145615">https://doi.org/10.1101/2020.06.11.145615</a>.
</div>
<div id="ref-yatesOralMicrobiome2021" class="csl-entry" role="doc-biblioentry">
Fellows Yates, James A., Irina M. Velsko, Franziska Aron, Cosimo Posth, Courtney A. Hofman, Rita M. Austin, Cody E. Parker, et al. 2021. <span>“The Evolution and Changing Ecology of the <span>African</span> Hominid Oral Microbiome.”</span> <em>Proceedings of the National Academy of Sciences</em> 118 (20). <a href="https://doi.org/10.1073/pnas.2021655118">https://doi.org/10.1073/pnas.2021655118</a>.
</div>
<div id="ref-hardyStarchGranulesDental2009" class="csl-entry" role="doc-biblioentry">
Hardy, Karen, Tony Blakeney, Les Copeland, Jennifer Kirkham, Richard Wrangham, and Matthew Collins. 2009. <span>“Starch Granules, Dental Calculus and New Perspectives on Ancient Diet.”</span> <em>Journal of Archaeological Science</em> 36 (2): 248–55. <a href="https://doi.org/10.1016/j.jas.2008.09.015">https://doi.org/10.1016/j.jas.2008.09.015</a>.
</div>
<div id="ref-hendyProteomicCalculus2018" class="csl-entry" role="doc-biblioentry">
Hendy, Jessica, Christina Warinner, Abigail Bouwman, Matthew J. Collins, Sarah Fiddyment, Roman Fischer, Richard Hagan, et al. 2018. <span>“Proteomic Evidence of Dietary Sources in Ancient Dental Calculus.”</span> <em>Proceedings. Biological Sciences</em> 285 (1883): 20180977. <a href="https://doi.org/10.1098/rspb.2018.0977">https://doi.org/10.1098/rspb.2018.0977</a>.
</div>
<div id="ref-henryCalculusSyria2008" class="csl-entry" role="doc-biblioentry">
Henry, Amanda G., and Dolores R. Piperno. 2008. <span>“Using Plant Microfossils from Dental Calculus to Recover Human Diet: A Case Study from <span>Tell</span> Al-<span>Raqā</span>’i, <span>Syria</span>.”</span> <em>Journal of Archaeological Science</em> 35 (7): 1943–50. <a href="https://doi.org/10.1016/j.jas.2007.12.005">https://doi.org/10.1016/j.jas.2007.12.005</a>.
</div>
<div id="ref-kazarinaPostmedievalMicrobial2021" class="csl-entry" role="doc-biblioentry">
Kazarina, Alisa, Elina Petersone-Gordina, Janis Kimsis, Jevgenija Kuzmicka, Pawel Zayakin, Žans Griškjans, Guntis Gerhards, and Renate Ranka. 2021. <span>“The <span>Postmedieval Latvian Oral Microbiome</span> in the <span>Context</span> of <span>Modern Dental Calculus</span> and <span>Modern Dental Plaque Microbial Profiles</span>.”</span> <em>Genes</em> 12 (2): 309. <a href="https://doi.org/10.3390/genes12020309">https://doi.org/10.3390/genes12020309</a>.
</div>
<div id="ref-BWA" class="csl-entry" role="doc-biblioentry">
Li, Heng, and Richard Durbin. 2009. <span>“Fast and Accurate Short Read Alignment with <span>Burrows</span>–<span>Wheeler</span> Transform.”</span> <em>Bioinformatics</em> 25 (14): 1754–60. <a href="https://doi.org/10.1093/bioinformatics/btp324">https://doi.org/10.1093/bioinformatics/btp324</a>.
</div>
<div id="ref-marshDentalPlaque2006" class="csl-entry" role="doc-biblioentry">
Marsh, Philip D. 2006. <span>“Dental Plaque as a Biofilm and a Microbial Community – Implications for Health and Disease.”</span> <em>BMC Oral Health</em> 6 (S1): S14. <a href="https://doi.org/10.1186/1472-6831-6-S1-S14">https://doi.org/10.1186/1472-6831-6-S1-S14</a>.
</div>
<div id="ref-mickleburghNewInsights2012" class="csl-entry" role="doc-biblioentry">
Mickleburgh, Hayley L., and Jaime R. Pagán-Jiménez. 2012. <span>“New Insights into the Consumption of Maize and Other Food Plants in the Pre-<span>Columbian Caribbean</span> from Starch Grains Trapped in Human Dental Calculus.”</span> <em>Journal of Archaeological Science</em> 39 (7): 2468–78. <a href="https://doi.org/10.1016/j.jas.2012.02.020">https://doi.org/10.1016/j.jas.2012.02.020</a>.
</div>
<div id="ref-pearceConcomitantDepositionStrontium1987" class="csl-entry" role="doc-biblioentry">
Pearce, E. I. F., and C. H. Sissons. 1987. <span>“The <span>Concomitant Deposition</span> of <span>Strontium</span> and <span>Fluoride</span> in <span>Dental Plaque</span>.”</span> <em>Journal of Dental Research</em> 66 (10): 1518–22. <a href="https://doi.org/10.1177/00220345870660100101">https://doi.org/10.1177/00220345870660100101</a>.
</div>
<div id="ref-AdapterRemovalv2" class="csl-entry" role="doc-biblioentry">
Schubert, Mikkel, Stinus Lindgreen, and Ludovic Orlando. 2016. <span>“<span>AdapterRemoval</span> V2: Rapid Adapter Trimming, Identification, and Read Merging.”</span> <em>BMC Research Notes</em> 9 (February): 88. <a href="https://doi.org/10.1186/s13104-016-1900-2">https://doi.org/10.1186/s13104-016-1900-2</a>.
</div>
<div id="ref-shellisSyntheticSalivaCultural1978" class="csl-entry" role="doc-biblioentry">
Shellis, R. P. 1978. <span>“A Synthetic Saliva for Cultural Studies of Dental Plaque.”</span> <em>Archives of Oral Biology</em> 23 (6): 485–89. <a href="https://doi.org/10.1016/0003-9969(78)90081-X">https://doi.org/10.1016/0003-9969(78)90081-X</a>.
</div>
<div id="ref-sissonsMultistationPlaque1991" class="csl-entry" role="doc-biblioentry">
Sissons, C. H., T. W. Cutress, M. P. Hoffman, and J. St J. Wakefield. 1991. <span>“A <span class="nocase">Multi-station Dental Plaque Microcosm</span> (<span>Artificial Mouth</span>) for the <span>Study</span> of <span>Plaque Growth</span>, <span>Metabolism</span>, <span class="nocase">pH</span>, and <span>Mineralization</span>:”</span> <em>Journal of Dental Research</em>. <a href="https://doi.org/10.1177/00220345910700110301">https://doi.org/10.1177/00220345910700110301</a>.
</div>
<div id="ref-velskoMicrobialDifferences2019" class="csl-entry" role="doc-biblioentry">
Velsko, Irina M., James A. Fellows Yates, Franziska Aron, Richard W. Hagan, Laurent A. F. Frantz, Louise Loe, Juan Bautista Rodriguez Martinez, et al. 2019. <span>“Microbial Differences Between Dental Plaque and Historic Dental Calculus Are Related to Oral Biofilm Maturation Stage.”</span> <em>Microbiome</em> 7 (1): 102. <a href="https://doi.org/10.1186/s40168-019-0717-3">https://doi.org/10.1186/s40168-019-0717-3</a>.
</div>
<div id="ref-warinnerPathogensHost2014" class="csl-entry" role="doc-biblioentry">
Warinner, Christina, J. F. Rodrigues, R. Vyas, C. Trachsel, N. Shved, J. Grossmann, A. Radini, et al. 2014. <span>“Pathogens and Host Immunity in the Ancient Human Oral Cavity.”</span> <em>Nature Genetics</em> 46 (4): 336–44. <a href="https://doi.org/10.1038/ng.2906">https://doi.org/10.1038/ng.2906</a>.
</div>
<div id="ref-kraken2" class="csl-entry" role="doc-biblioentry">
Wood, Derrick E., Jennifer Lu, and Ben Langmead. 2019. <span>“Improved Metagenomic Analysis with <span>Kraken</span> 2.”</span> <em>Genome Biology</em> 20 (1): 257. <a href="https://doi.org/10.1186/s13059-019-1891-0">https://doi.org/10.1186/s13059-019-1891-0</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    toggleBodyColorMode(window.document.querySelector("link#quarto-bootstrap"));
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>