<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.16">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Bjørn Peare Bartholdy">
<meta name="author" content="Irina M. Velsko">
<meta name="author" content="Shira Gur-Arieh">
<meta name="author" content="Zandra Fagernäs">
<meta name="author" content="Sophie Seng">
<meta name="author" content="Amanda G. Henry">

<title>Assessing the validity of a mineralised oral biofilm model as a suitable proxy for dental calculus</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Assessing the validity of a mineralised oral biofilm model as a suitable proxy for dental calculus</h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    Bjørn Peare Bartholdy <a href="https://orcid.org/0000-0003-3985-1016" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a>
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Leiden University
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    Irina M. Velsko 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Max Planck Institute for Evolutionary Anthropology
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    Shira Gur-Arieh 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Haifa University
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    Zandra Fagernäs 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Max Planck Institute for Evolutionary Anthropology
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    Sophie Seng 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Max Planck Institute for Evolutionary Anthropology
          </p>
      </div>
      <div class="quarto-title-meta-contents">
    Amanda G. Henry 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Leiden University
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    
  </div>
  
<div>
  <div class="abstract">
    <div class="abstract-title">Abstract</div>
    <p>This is the abstract for the paper.</p>
  </div>
</div>

</header>

<section id="introduction" class="level1">
<h1>Introduction</h1>
<!-- basic introduction to the field, comprehensible to a scientist in any discipline -->
<p>Dental calculus is quickly becoming the go-to substance for exploring health and diet in past populations. Studies using archaeological dental calculus span a wide range of topics in different regions and time periods. These include characterisation of the oral microbiome and its evolution in past populations <span class="citation" data-cites="velskoMicrobialDifferences2019 adlerSequencingAncient2013 warinnerPathogensHost2014 kazarinaPostmedievalMicrobial2021 yatesOralMicrobiome2021">(<a href="#ref-adlerSequencingAncient2013" role="doc-biblioref">Adler et al., 2013</a>; <a href="#ref-yatesOralMicrobiome2021" role="doc-biblioref">Fellows Yates et al., 2021</a>; <a href="#ref-kazarinaPostmedievalMicrobial2021" role="doc-biblioref">Kazarina et al., 2021</a>; <a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al., 2019</a>; <a href="#ref-warinnerPathogensHost2014" role="doc-biblioref">Warinner et al., 2014</a>)</span>, extraction of microbotanical remains <span class="citation" data-cites="henryCalculusSyria2008 hardyStarchGranulesDental2009 mickleburghNewInsights2012 maHumanDiet2022">(<a href="#ref-hardyStarchGranulesDental2009" role="doc-biblioref">Hardy et al., 2009</a>; <a href="#ref-henryCalculusSyria2008" role="doc-biblioref">Henry &amp; Piperno, 2008</a>; <a href="#ref-maHumanDiet2022" role="doc-biblioref">Ma et al., 2022</a>; <a href="#ref-mickleburghNewInsights2012" role="doc-biblioref">Mickleburgh &amp; Pagán-Jiménez, 2012</a>)</span> and other residues to infer dietary patterns and nicotine-use <span class="citation" data-cites="buckleyDentalCalculus2014 hendyProteomicCalculus2018 eerkensDentalCalculus2018">(<a href="#ref-buckleyDentalCalculus2014" role="doc-biblioref">Buckley et al., 2014</a>; <a href="#ref-eerkensDentalCalculus2018" role="doc-biblioref">Eerkens et al., 2018</a>; <a href="#ref-hendyProteomicCalculus2018" role="doc-biblioref">Hendy et al., 2018</a>)</span>.</p>
<!-- more detailed background, comprehensible to scientists in related disciplines -->
<!-- dental plaque/calculus formation and calculus composition -->
<p>Dental calculus is formed by the mineralisation of dental plaque. Dental plaque is an oral biofilm and is part of the normal state of the oral cavity; however, if left unchecked, plaque can lead to infections such as dental caries and periodontitis <span class="citation" data-cites="marshDentalPlaque2006">(<a href="#ref-marshDentalPlaque2006" role="doc-biblioref">Marsh, 2006</a>)</span>. Shortly after teeth are cleaned (whether mechanically or otherwise), a salivary pellicle adsorbs to the surface of the tooth, in most cases enamel, forming the acquired dental pellicle. The pellicle is comprised mainly of proteins and, in addition to protecting the tooth against mechanical and chemical decay, provides a viable surface for bacterial attachment <span class="citation" data-cites="yaoIdentificationProtein2003">(<a href="#ref-yaoIdentificationProtein2003" role="doc-biblioref">Yao et al., 2003</a>)</span>. Shortly after adsorbing to the enamel, early-coloniser species, such as those within genus <em>Streptococcus</em> and <em>Actinomyces</em>, adhere to the pellicle through reversible long-range physicochemical forces and irreversible short range cell-host<!--check--> interactions <span class="citation" data-cites="marshDentalPlaque2006">(<a href="#ref-marshDentalPlaque2006" role="doc-biblioref">Marsh, 2006</a>)</span>. Once the surface has been populated by the specialists in surface-attachment, other species of bacteria can attach to their surface through cell-cell interactions, allowing adhesion of species that are not otherwise capable of adhering to a surface. After accumulation and multiplication of bacteria, this, now, diverse community of is able to secrete polysaccharides, proteins, lipids, and nucleic acids, into their immediate environment to form a biofilm matrix <span class="citation" data-cites="flemmingBiofilmsEmergent2016">(<a href="#ref-flemmingBiofilmsEmergent2016" role="doc-biblioref">Flemming et al., 2016</a>)</span>. The matrix provides an adaptive advantage to the organisms within through resistance to antibiotics and mechanical removal, as well as transporting nutrients from outside the biofilm and facilitating distribution of resources between bacterial communities within the biofilm <span class="citation" data-cites="petersonViscoelasticityBiofilms2015 jainIsolationCharacterization2013">(<a href="#ref-jainIsolationCharacterization2013" role="doc-biblioref">Jain et al., 2013</a>; <a href="#ref-petersonViscoelasticityBiofilms2015" role="doc-biblioref">Peterson et al., 2015</a>)</span>.</p>
<p>The composition of a biofilm matrix is largely water (around 90%), with the remaining content consisting of microbes, extracellular polysaccharides (EPS), DNA, RNA, and proteins <span class="citation" data-cites="bergerOralBiofilms2018">(<a href="#ref-bergerOralBiofilms2018" role="doc-biblioref">Berger et al., 2018</a>)</span>. Biofilms can become susceptible to calcification under certain microenvironmental conditions. These include an increased concentration of salts and a decrease in statherin and proline-rich proteins in saliva, rises in local plaque pH, and increased hydrolysis of urea <span class="citation" data-cites="whiteDentalCalculus1997 wongCalciumPhosphate2002">(<a href="#ref-whiteDentalCalculus1997" role="doc-biblioref">White, 1997</a>; <a href="#ref-wongCalciumPhosphate2002" role="doc-biblioref">Wong et al., 2002</a>)</span>. Under these conditions, the biofilm environment becomes favourable to increased precipitation and decreased dissolution of calcium phosphate salts within saliva and the plaque biofilm. The resulting supersaturation of calcium phosphate salts are the main drivers of biofilm mineralisation <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref">Jin &amp; Yip, 2002</a>)</span>. Mineralisation generally starts from within the biofilm matrix as a result of nucleation, followed by mineralisation of the matrix and, subsequently, bacterial cells. The susceptibility of crystallisation in bacteria depends on the composition and concentration of membrane-associated components, such as proteolipids and phospholipids <span class="citation" data-cites="jinSupragingivalCalculus2002 whiteDentalCalculus1997">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref">Jin &amp; Yip, 2002</a>; <a href="#ref-whiteDentalCalculus1997" role="doc-biblioref">White, 1997</a>)</span>. Binding of calcium to bacterial membranes is facilitated by phospholipid molecules within the cell membrane, followed by association of phosphates with the bound calcium to form calcium phosphate complexes. These complexes are active in promoting the formation and deposition of hydroxyapatite within biofilms <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref">Jin &amp; Yip, 2002</a>)</span>.</p>
<p>The primary minerals in dental calculus are hydroxyapatite (HAP), octacalcium phosphate (OCP), whitlockite (WHT), and brushite (BHT). During initial mineralisation the main mineral component is BHT, which shifts to HAP in more mature dental calculus <span class="citation" data-cites="jinSupragingivalCalculus2002 hayashizakiSiteSpecific2008">(<a href="#ref-hayashizakiSiteSpecific2008" role="doc-biblioref">Hayashizaki et al., 2008</a>; <a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref">Jin &amp; Yip, 2002</a>)</span>. The exact elemental composition of dental calculus varies by individual due to various factors, including diet <span class="citation" data-cites="hayashizakiSiteSpecific2008 jiFluorideMagnesium2000">(<a href="#ref-hayashizakiSiteSpecific2008" role="doc-biblioref">Hayashizaki et al., 2008</a>; <a href="#ref-jiFluorideMagnesium2000" role="doc-biblioref">Ji et al., 2000</a>)</span>.</p>
<p>The role of bacteria in dental calculus formation is still not clear, and dental calculus formation has been induced in bacteria-free rodents <span class="citation" data-cites="glasBiophysicalStudies1962">(<a href="#ref-glasBiophysicalStudies1962" role="doc-biblioref">Glas &amp; Krasse, 1962</a>)</span>. However, given the abundance of bacteria present within human dental plaque, the structure of calculus will reflect the presence of bacteria with human dental calculus containing a heterogenous mineral composition across the biofilm due to the differing mineralisation properties in bacteria, and also directly influences the porosity of calculus <span class="citation" data-cites="omelonReviewPhosphate2013 rohanizadehUltrastructuralStudy2005">(<a href="#ref-omelonReviewPhosphate2013" role="doc-biblioref">Omelon et al., 2013</a>; <a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref">Rohanizadeh &amp; LeGeros, 2005</a>)</span>. <!-- rephrase: need to emphasise the importance of bacteria in human dental calculus - better proxy --> <!-- biofilm maturation and mineralisation --> <!-- dental calculus composition and distribution --> The different susceptibility of certain bacteria to mineralise may explain differences in bacterial profiles in plaque and dental calculus <span class="citation" data-cites="velskoMicrobialDifferences2019">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al., 2019</a>)</span></p>
<p>Attachment of dental calculus to enamel is further solidified by fusion of dental calculus with enamel rods <span class="citation" data-cites="rohanizadehUltrastructuralStudy2005 whiteDentalCalculus1997">(<a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref">Rohanizadeh &amp; LeGeros, 2005</a>; <a href="#ref-whiteDentalCalculus1997" role="doc-biblioref">White, 1997</a>)</span>.</p>
<p>The organic component of dental calculus consists of proteins and lipids, likely incorporated from bacteria, saliva, and food <span class="citation" data-cites="whiteDentalCalculus1997">(<a href="#ref-whiteDentalCalculus1997" role="doc-biblioref">White, 1997</a>)</span>. Calculus forming above the gumline (supragingival) has a lower inorganic content than calculus forming below the gumline (subgingival) <span class="citation" data-cites="jinSupragingivalCalculus2002">(<a href="#ref-jinSupragingivalCalculus2002" role="doc-biblioref">Jin &amp; Yip, 2002</a>)</span>.</p>
<!-- oral biofilm models. History, types, etc. -->
<p>Oral biofilm models are commonly used in dental research to assess the efficacy of certain treatments on dental pathogens <span class="citation" data-cites="filochePlaqueMicrocosm2007 extercateAAA2010">(<a href="#ref-extercateAAA2010" role="doc-biblioref">Exterkate et al., 2010</a>; <a href="#ref-filochePlaqueMicrocosm2007" role="doc-biblioref">Filoche et al., 2007</a>)</span>. These are often short-term models grown over a few days, but there also exist longer term models used to develop dental calculus <span class="citation" data-cites="middletonVitroCalculus1965 sissonsMultistationPlaque1991 wongCalciumPhosphate2002">(<a href="#ref-middletonVitroCalculus1965" role="doc-biblioref">Middleton, 1965</a>; <a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>; <a href="#ref-wongCalciumPhosphate2002" role="doc-biblioref">Wong et al., 2002</a>)</span>. There are multiple different types of models ranging from simplistic agar plate or multiwell-plate models <span class="citation" data-cites="ceriCalgaryBiofilm1999 extercateAAA2010">(<a href="#ref-ceriCalgaryBiofilm1999" role="doc-biblioref">Ceri et al., 1999</a>; <a href="#ref-extercateAAA2010" role="doc-biblioref">Exterkate et al., 2010</a>)</span>, to more complex setups like the constant depth film fermenter (CDFF) <span class="citation" data-cites="petersConstantDepth1988">(<a href="#ref-petersConstantDepth1988" role="doc-biblioref">Peters &amp; Wimpenny, 1988</a>)</span> and the multi-station artificial mouth (MAM) <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>)</span>. The more complex models have the benefit of a continuous flow of saliva or saliva-like medium and control over the environment, while the multiwell-plate models offer the advantage of generating more samples over the same amount of time <span class="citation" data-cites="mcbainBiofilmModels2009">(<a href="#ref-mcbainBiofilmModels2009" role="doc-biblioref">McBain, 2009</a>)</span>. Simplistic models restricted to a select subset of oral bacteria are often more reproducible than models using whole saliva, while the latter are more representative of the <em>in vivo</em> oral microbiome complexity <span class="citation" data-cites="roderStudyingBacterial2016 mcbainBiofilmModels2009">(<a href="#ref-mcbainBiofilmModels2009" role="doc-biblioref">McBain, 2009</a>; <a href="#ref-roderStudyingBacterial2016" role="doc-biblioref">Røder et al., 2016</a>)</span>.</p>
<!-- clearly stating the general problem being addressed by this particular study -->
<p>We present an oral biofilm model that can serve as a viable proxy for dental calculus, and provide a method for fundamental research on dental calculus in the past. The need for such a model is warranted by the different questions that are asked by archaeologists compared to clinical dentistry. We are interested in learning more about how dietary residues and microremains become trapped in calculus, and how the methods we use may inadvertently bias our interpretations; questions that are best addressed in a lab using a model. We used FTIR to verify the mineral composition, and metagenomic classification to characterise the bacterial composition, and compared our results against modern and archaeological human dental calculus. We found that the mineral and organic components mimic that of the modern reference calculus used for comparison, while the bacterial classification revealed a similar but distinct community structure.<!--The bacterial composition --> In addition to the benefit of increased control over parameters involved in calculus formation and dietary incorporation, our method also provides unlimited material for experimentation, rather than using the limited archaeological material currently available.</p>
</section>
<section id="materials-and-methods" class="level1">
<h1>Materials and Methods</h1>
<section id="biofilm-growth" class="level2">
<h2 class="anchored" data-anchor-id="biofilm-growth">Biofilm growth</h2>

<!-- Create table for all samples and indicate which samples underwent DNA, FTIR, etc. -->
<p>In this study we employ a multispecies oral biofilm model following a modified protocol from Sissons and colleagues <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">1991</a>)</span> and Shellis <span class="citation" data-cites="shellisSyntheticSaliva1978">(<a href="#ref-shellisSyntheticSaliva1978" role="doc-biblioref">1978</a>)</span>. The setup comprises a polypropylene 24 deepwell PCR plate (KingFisher 97003510) with a lid containing 24 pegs (substrata), which is autoclaved at 120°C, 1 bar overpressure, for 20 mins.</p>
<p>The artificial saliva (AS) is a modified version of the basal medium mucin (BMM) described by Sissons and colleagues <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">1991</a>)</span>. It is a complex medium containing 2.5 g/l partially purified mucin from porcine stomach (Type III, Sigma M1778), 5 g/l trypticase peptone (Roth 2363.1), 10 g/l proteose peptone (Oxoid LP0085), 5 g/l yeast extract (BD 211921), 2.5 g/l KCl, 0.35 g/l NaCl, 1.8 mmol/l CaCl<sub>2</sub>, 5.2 mmol/l Na<sub>2</sub>HPO<sub>4</sub> <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>)</span>, 6.4 mmol/l NaHCO<sub>3</sub> <span class="citation" data-cites="shellisSyntheticSaliva1978">(<a href="#ref-shellisSyntheticSaliva1978" role="doc-biblioref">Shellis, 1978</a>)</span>, 2.5 mg/l haemin. This is subsequently adjusted to pH 7 with NaOH pellets and stirring, autoclaved (15 min, 120°C, 1 bar overpressure), and supplemented with 5.8 (mu)mol/l menadione, 5 mmol/l urea, and 1 mmol/l arginine <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>)</span>.</p>
<!-- add to protocols.io -->
<p>Fresh whole saliva (WS) for inoculation was provided by a 31-year-old male donor with no history of caries, who abstained from oral hygiene for 24 hours, and no food was consumed two hours prior to donation. No antibiotics were taken up to six months prior to donation. The saliva was filtered through a sterilised (with bleach) nylon cloth to remove particulates. Substrata were inoculated with 1 ml/well of a two-fold dilution of WS in sterilised 20% glycerine for four hours at 36°C, to allow attachment of the salivary pellicle and plaque-forming bacteria. After initial inoculation, the substrata were transferred to a new plate containing 1 ml/well AS and incubated at 36°C, 30 rpm. The inoculation process was repeated on days 3 and 5. AS was partially refreshed once per day and fully refreshed every three days, throughout the experiment, by transferring the substrata to a new plate containing AS. To feed the bacteria, the substrata were transferred to a new plate, containing 5% (w/v) sucrose, for six minutes twice daily, except on inoculation days (days 0, 3, and 5), where the samples only received one sucrose treatment after inoculation.</p>
<p>Starch treatments were initiated on day 9 to avoid starch granule counts being affected by <span class="math inline">\(alpha\)</span>-amylase hydrolysis from inoculation saliva. An <span class="math inline">\(\alpha\)</span>-amylase assay confirmed the absence of any <span class="math inline">\(\alpha\)</span>-amylase activity in the system <span class="citation" data-cites="bartholdyInvestigatingBiases2022">(<a href="#ref-bartholdyInvestigatingBiases2022" role="doc-biblioref">Bartholdy &amp; Henry, 2022</a>)</span>. Starch treatments replaced sucrose treatments, occurring twice per day for six minutes. The starch treatments involved transferring the substrata to a new plate containing a 0.25% (w/v) starch from potato (Roth 9441.1) solution, a 0.25% (w/v) starch from wheat (Sigma S5127) solution, and a 0.5% (w/v) mixture of equal concentrations (w/v) wheat and potato. All starch solutions were created in a 5% (w/v) sucrose solution. Before transferring biofilm samples to the starch treatments, the starch plates were agitated to keep the starches in suspension in the solutions, and during treatments, the rpm was increased to 60.</p>
<p>After 15 days, mineralisation was encouraged with a calcium phosphate monofluorophosphate urea (CPMU) solution containing 20 mmol/l CaCl<sub>2</sub>, 12 mmol/l NaH<sub>2</sub>PO<sub>4</sub>, 5 mmol/l Na<sub>2</sub>PO<sub>3</sub>F, 500 mmol/l Urea <span class="citation" data-cites="pearceConcomitantDeposition1987 sissonsMultistationPlaque1991">(<a href="#ref-pearceConcomitantDeposition1987" role="doc-biblioref">Pearce &amp; Sissons, 1987</a>; <a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>)</span>, and (0.04 g/l MgCl). The substrata were submerged in 1 ml/well CPMU five times daily, every two hours, for six minutes. During the mineralisation period, starch treatments were reduced to once per day after the five CPMU treatments. This cycle was repeated for 10 days until the end of the experiment on day 24 (see @ref(fig:protocol-fig) for an overview of protocol). A more detailed protocol is available at<!-- protocols.io -->.</p>
<p>All laboratory work was conducted in sterile conditions under a laminar flow hood to prevent starch and bacterial contamination. Control samples were included to detect starch contamination.</p>
</section>
<section id="metagenomics" class="level2">
<h2 class="anchored" data-anchor-id="metagenomics">Metagenomics</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<table class="table table-sm table-striped">
<thead>
<tr class="header">
<th style="text-align: left;">Sample type</th>
<th style="text-align: right;">Sampling day</th>
<th style="text-align: right;">n</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">saliva</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">saliva</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">saliva</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">7</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">18</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">21</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">medium</td>
<td style="text-align: right;">24</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">byoc_calculus</td>
<td style="text-align: right;">24</td>
<td style="text-align: right;">16</td>
</tr>
</tbody>
</table>
</div>
</div>
<!-- Samples sent to Jena -->
<p>A total of 35 samples were taken during the experiment from the donated saliva, artificial saliva, and from the biofilm end-product on day 24. DNA extraction was performed at the archaeogenetic facility at the Max Planck Institute for the Science of Human History (Jena, Germany). Extractions were performed in duplicates<!-- ?? -->. A total of <!-- sample size --> DNA extracts.</p>
<!-- DNA extraction details -->
<p>DNeasy PowerSoil Kit from QIAGEN. C2 inhibitor removal step skipped, going directly to C3 step.</p>
<p>were paired-end sequenced on a NextSeq (2 color chemistry) to 150bp</p>
<!-- EAGER settings -->
<section id="preprocessing" class="level3">
<h3 class="anchored" data-anchor-id="preprocessing">Preprocessing</h3>
<p>Processing of the raw DNA reads was conducted using the nf-core/eager, v2.4.4 pipeline <span class="citation" data-cites="yatesEAGER2020">(<a href="#ref-yatesEAGER2020" role="doc-biblioref">Fellows Yates et al., 2020</a>)</span>. Adapter removal and read merging was performed using AdapterRemoval, v2.3.2 <span class="citation" data-cites="AdapterRemovalv2">(<a href="#ref-AdapterRemovalv2" role="doc-biblioref">Schubert et al., 2016</a>)</span>. Merged reads were mapped to the human reference genome (GRCh38 <!--link-->) using BWA, v0.7.17-r1188 <span class="citation" data-cites="BWA">(<a href="#ref-BWA" role="doc-biblioref">Li &amp; Durbin, 2009</a>)</span> with default settings (-n 0.01; -l 32), and unmapped reads were extracted using Samtools, v1.12.</p>
<p>Metagenomic classification was conducted in kraken, v2.1.2 using the Standard database <span class="citation" data-cites="kraken2">(<a href="#ref-kraken2" role="doc-biblioref">Wood et al., 2019</a>)</span>. <!-- cuperdec [@yatesOralMicrobiome2021]--></p>
<p>Environmental reference samples were downloaded from ENA and using SRA Toolkit.<!-- reference supp. mat.--> Oral reference samples were downloaded from the Human Metagenome Project (HMP), and modern calculus samples from Velsko et al. <span class="citation" data-cites="velskoDentalCalculus2017">(<a href="#ref-velskoDentalCalculus2017" role="doc-biblioref">2017</a>)</span> Only paired reads were processed, singletons were removed. Biofilm model samples from Edlund et al. <span class="citation" data-cites="edlundUncoveringComplex2018">(<a href="#ref-edlundUncoveringComplex2018" role="doc-biblioref">2018</a>)</span> were used as a reference. Links to the specific sequences, including human-filtered reads from this study, are included in the metadata<!-- link to metadata file-->.</p>
</section>
<section id="authentication" class="level3">
<h3 class="anchored" data-anchor-id="authentication">Authentication</h3>
<!-- sourcetracker -->
<p>Species with lower than 0.001% relative abundance were removed. SourceTracker <span class="citation" data-cites="knightsSourceTracker2011">(<a href="#ref-knightsSourceTracker2011" role="doc-biblioref">Knights et al., 2011</a>)</span> was used to estimate source composition of the oral biofilm model samples using a Bayesian framework. Sampes were compared with oral and environmental controls to detect potential external contamination. The R package <em>decontam</em> v1.16.0 <span class="citation" data-cites="Rdecontam">(<a href="#ref-Rdecontam" role="doc-biblioref">Davis et al., 2018</a>)</span> was used to identify potential contaminants using DNA concentrations with a probability threshold of 0.95 and negative controls with a probability threshold of 0.05. Putative contaminants were filtered out of the OTU tables for all downstream analyses. Authentication methods are described in more detail in the Supplementary material.<!--../06-reports/metagen-authentication.Rmd--></p>
</section>
<section id="community-composition" class="level3">
<h3 class="anchored" data-anchor-id="community-composition">Community composition</h3>
<p>Relative abundances of communities were calculated at the species- and genus-level, as recommended for compositional data <span class="citation" data-cites="gloorMicrobiomeDatasets2017">(<a href="#ref-gloorMicrobiomeDatasets2017" role="doc-biblioref">Gloor et al., 2017</a>)</span>. Shannon index calculated on species OTU tables of all artificial and oral reference samples using the vegan 2.6.2 R package <span class="citation" data-cites="Rvegan">(<a href="#ref-Rvegan" role="doc-biblioref">Oksanen et al., 2022</a>)</span>. Sparse principal components analysis (sPCA) was performed on model biofilm samples to … and a separate sPCA analysis was performed on biofilm model end-products and oral reference samples using the mixOmics R package 6.20.0 <span class="citation" data-cites="RmixOmics">(<a href="#ref-RmixOmics" role="doc-biblioref">Rohart et al., 2017</a>)</span>.</p>
</section>
<section id="differential-abundance" class="level3">
<h3 class="anchored" data-anchor-id="differential-abundance">Differential abundance</h3>
</section>
</section>
<section id="ftir" class="level2">
<h2 class="anchored" data-anchor-id="ftir">FTIR</h2>
<div class="cell" data-layout-align="center">

</div>
<p>To determine the mineral composition and level of crystallisation of the model dental calculus samples, we used Fourier Transform Infrared (FTIR) spectroscopy. We compared the spectra of model dental calculus with spectra archaeological dental calculus and used a built-in Omnic search library for mineral identification <span class="citation" data-cites="weinerInfraredSpectroscopy2010">(<a href="#ref-weinerInfraredSpectroscopy2010" role="doc-biblioref">Weiner, 2010a</a>)</span>. The archaeological sample was dental calculus from an isolated tooth from Middenbeemster, a rural, 19th century Dutch site. Samples were analysed at the Laboratory for Sedimentary Archaeology, Haifa University. The analysis was conducted with a Thermo Nicolet is5 spectrometer in transmission, at 4 cm<span class="math inline">\(^{-1}\)</span> resolution, with an average of 32 scans between wavenumbers 4000 and 400 cm<span class="math inline">\(^{-1}\)</span>.</p>
<p>Analysis was conducted on 28 biofilm samples from days 7, 12, 16, 20, and 24. Some samples from the same sampling day had to be combined to provide enough material for analysis. Samples analysed for FTIR originated from a different experiment than the metagenomic samples, following the same protocol (as described above). Samples were analysed following the method presented in <span class="citation" data-cites="asscherAtomicDisorder2011">Asscher, Regev, et al. (<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref">2011</a>)</span>. A few <span class="math inline">\(\mu\)</span>g of each sample were repeatedly ground together with KBr and pressed in a 7 mm die under two tons of pressure using a Specac mini-pellet press (Specac Ltd., GS01152). Repeated measurements of the splitting factor were taken after each grind, and a grind curve was produced following <span class="citation" data-cites="asscherAtomicDisorder2011">Asscher, Regev, et al. (<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref">2011</a>)</span>. Samples were ground and analysed up to six times (sample suffix a-f) for the grinding curve. Grinding curves were prepared for samples from days 16, 20, and 24. No grind curves were produced for samples from days 7 and 12. These were largely composed of organics and proteins, and did not form enough carbonated hydroxtapatite for analysis. The splitting factor of carbonate hydroxyapatite was calculated using a macro script, following <span class="citation" data-cites="weinerStatesPreservation1990">Weiner &amp; Bar-Yosef (<a href="#ref-weinerStatesPreservation1990" role="doc-biblioref">1990</a>)</span>, by dividing the sum of the height of the absorptions at 603 cm<span class="math inline">\(^{-1}\)</span> and 567 cm<span class="math inline">\(^{-1}\)</span> in the height of the valley between them. Following <span class="citation" data-cites="asscherAtomicDisorder2011">Asscher, Regev, et al. (<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref">2011</a>)</span> we plotted the IRSF against the full width at half maximum (FWHM) of the main absorption at 1035, and compared our grinding curves to the ones produced by <span class="citation" data-cites="asscherAtomicDisorder2011">Asscher, Regev, et al. (<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>Splitting factors of the doublet, and FWHM of the main PO<sub>4</sub> peak at 1040 were calculated from the grinding measurements, and plotted against each other to create grinding curves to explore crystallinity (crystal size) and the order and disorder. Disorder is a steep slope and large FWHM.</p>
</section>
<section id="statistics" class="level2">
<h2 class="anchored" data-anchor-id="statistics">Statistics</h2>
<!-- R packages used -->
<p>Statistical analysis was conducted in R Statistical Software R version 4.2.0 (2022-04-22) (Vigorous Calisthenics) <span class="citation" data-cites="Rbase">(<a href="#ref-Rbase" role="doc-biblioref">R Core Team, 2020</a>)</span>. Data cleaning and wrangling performed with packages from <strong>tidyverse</strong> <span class="citation" data-cites="tidyverse2019">(<a href="#ref-tidyverse2019" role="doc-biblioref">Hadley Wickham et al., 2019</a>)</span>. Plots were created using ggplot2 <span class="citation" data-cites="ggplot2">(<a href="#ref-ggplot2" role="doc-biblioref">H. Wickham, 2016</a>)</span>.</p>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="metagenomic-analysis" class="level2">
<h2 class="anchored" data-anchor-id="metagenomic-analysis">Metagenomic analysis</h2>
<section id="sample-authentication" class="level3">
<h3 class="anchored" data-anchor-id="sample-authentication">Sample authentication</h3>
<!-- SourceTracker -->
<p>The sources of taxa were estimated using SourceTracker2 <span class="citation" data-cites="knightsSourceTracker2011">(<a href="#ref-knightsSourceTracker2011" role="doc-biblioref">Knights et al., 2011</a>)</span>. The results suggest that the majority of taxa across samples have an oral microbial signature. SourceTracker2 results were compared to a database of oral taxa <span class="citation" data-cites="yatesOralMicrobiome2021">(<a href="#ref-yatesOralMicrobiome2021" role="doc-biblioref">Fellows Yates et al., 2021</a>)</span> to prevent removal of samples where oral taxa were assigned to a non-oral source, as some similar taxa with a signature from multiple sources are often classified as “Unknown” <span class="citation" data-cites="velskoMicrobialDifferences2019">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al., 2019</a>)</span>. A few samples suspected of containing a large proportion of contamination were removed (SYN015.F0101,SYN015.G0101,SYN015.H0101,SYN017.F0101,SYN017.G0101,SYN018.H0101,SYN013.I0101,SYN016.I0101). The removed samples were predominantly medium samples from later in the experiment, and a few model calculus samples (see Supp mat).</p>
</section>
<section id="decrease-in-community-diversity-across-experiment" class="level3">
<h3 class="anchored" data-anchor-id="decrease-in-community-diversity-across-experiment">Decrease in community diversity across experiment</h3>
<div class="cell" data-layout-align="center">

</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-shannon-byoc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-shannon-byoc-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 1: Plot of Shannon Index across experiment samples grouped by sampling time.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<!-- species diversity -->
<p>After contamination removal, samples consisted of 88 and 284 with a mean of 181.59. We used the Shannon Index to assess the species diversity and richness in our samples over the course of the experiment. Samples were grouped into sampling categories due to low sample sizes on sampling days (inoc = days 0, 3, 5; treatm = days 7, 9, 12, 15; final = day 24). There was a slight decrease in mean alpha diversity between inoculation (M = 2.1458561, sd = 0.8082455) and treatment and final samples (M = 1.7752494, 0.5678242; 1.8578819, 0.260961), as well as a decrease in variance within samples types (<a href="#fig-shannon-byoc">Figure&nbsp;1</a>). Based on the similarity of distribution between Pielou’s Species Evenness and the Shannon index, the main driver of differences is likely the species richness <!-- insert link to Supp Mat-->(Supp. Mat).</p>
</section>
<section id="medium-and-model-calculus-samples-are-distinct-from-the-inoculate" class="level3">
<h3 class="anchored" data-anchor-id="medium-and-model-calculus-samples-are-distinct-from-the-inoculate">Medium and model calculus samples are distinct from the inoculate</h3>
<div class="cell" data-layout-align="center">

</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/spca-byoc-plot-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">sPCA on species-level counts from experiment samples only.</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/spca-byoc-plot-2.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">sPCA on species-level counts from experiment samples only.</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/spca-byoc-plot-3.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">sPCA on species-level counts from experiment samples only.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The species profiles of the saliva inoculate used in our experiment were distinct from both medium and model calculus samples. Most of the separation is on PC1 of the sPCA, with minimal separation on PC2. Most of the model calculus samples also cluster separately from the medium samples on PC2, but maintaining some overlap with more mature medium samples. The largest negative loadings on PC1, separating inoculate from other samples, are an abundance of aerobes and facultative anaerobes in the inoculate, especially <em>Rothia</em> and <em>Neisseria</em> spp. Conversely, positive loadings consist mainly of anaerobes, especially <em>Selenomonas</em> spp.</p>
<div class="cell" data-layout-align="center">

</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-diffabund-byoc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-diffabund-byoc-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 2: Log-fold changes between sample types. Circles are species enriched in the model calculus, squares are enriched in saliva, and triangles in medium. Lines are standard error. Plot shows the top 20 absolute log-fold changes between model calculus and saliva.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Species enriched in saliva compared to model calculus are largely aerobic or facultative anaerobic, while species enriched in model calculus compared to saliva are mainly anaerobes. Compared to saliva, species differences between medium and model calculus are less pronounced <a href="#fig-diffabund-byoc">Figure&nbsp;2</a>.</p>
</section>
<section id="lower-diversity-in-artificial-samples-than-oral-references" class="level3">
<h3 class="anchored" data-anchor-id="lower-diversity-in-artificial-samples-than-oral-references">Lower diversity in artificial samples than oral references</h3>
<div class="cell" data-layout-align="center">

</div>
<!-- alpha diversity -->
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-shannon-compar" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-shannon-compar-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 3: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Again, the Shannon Index was used to evaluate alpha diversity, this time across model samples and oral reference samples. Model samples (M = 1.86, M = 1.74, M = 1.69) were consistently lower than oral reference samples (M = 2.74, M = 3.14, M = 3.02, M = 3.53, M = 3.04) in species richness and diversity. The Pielou Species Evenness has a similar distribution, although the comparative biofilm samples have a higher mean than biofilm samples from this study (Supp. Mat.)<!--link to supp mat-->.</p>
<!-- beta diversity -->
</section>
<section id="model-calculus-is-distinct-from-dental-calculus-and-other-oral-samples" class="level3">
<h3 class="anchored" data-anchor-id="model-calculus-is-distinct-from-dental-calculus-and-other-oral-samples">Model calculus is distinct from dental calculus and other oral samples</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-genus-env" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-genus-env-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 4: Core microbiome of the different types of samples represented as mean relative abundances at the genus level. NA = other genera.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The main overlap between the model calculus and oral comparative samples are the high relative abundance of <em>Streptococcus</em>. Model calculus consists mostly of <em>Enterococcus</em> and <em>Veillonella</em> spp., while oral comparative samples are more diverse (consistent with the Shannon Index).</p>
<div class="cell" data-layout-align="center">

</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/unnamed-chunk-14-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/unnamed-chunk-14-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p>Model calculus samples are distinct from both the oral reference samples and the biofilm model reference samples. They are separated from oral reference samples mainly on PC1, and from biofilm model reference samples (and, to some extent, oral samples) on PC2. The highest negative contributions are a mix of all types of aerotolerance, while the positive contributions are mostly (facultative) anaerobes, with <em>Enterococcus</em> spp. as the top three positive contributors to PC1. Top negative contributors are <em>Capnocytophaga</em> spp as well as the aerobes <em>Corynebacterium matruchotii</em> and <em>Rothia dentocariosa</em>. The top positive contributors to PC2 are all anaerobes, mainly from the genus <em>Selenomonas</em>. Top negative contributors to PC2 are a mix of aerotolerances, with many <em>Streptococcus</em> spp.</p>
<div class="cell" data-layout-align="center">

</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-diffabund-comp2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-diffabund-comp2-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 5: Log-fold changes between sample types. Circles are species enriched in the model calculus, triangles in modern calculus, diamonds are enriched in subgingival plaque, and squares in supragingival plaque. Plot shows the top 30 loadings (absolute value) in PC1 (A) and PC2 (B) between model calculus and other sample types, ordered by decreasing log-fold change. Bars represent standard error.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Based on the differential abundance analysis on the top loadings from PC1, the main differences between model calculus and oral reference samples, are that the oral reference samples are enriched with species with a diverse oxygen tolerance from a wide range of genera, while the model calculus is enriched with <em>Enterococcus</em> spp. The largest differences occur in <em>Corynebacterium matruchotii</em>, <em>Rothia dentocariosa</em>, and <em>Capnocytophaga gingivalis</em> (<a href="#fig-diffabund-comp2">Figure&nbsp;5</a>A). This is echoed in the differential abundance analysis on top loadings from PC2, where most of the species are enriched in model calculus, all of which are anaerobes, and the largest differences occurring in <em>Cryptobacterium curtum</em>, <em>Eggerthella lenta</em>, and <em>Mogibacterium diversum</em> (<a href="#fig-diffabund-comp2">Figure&nbsp;5</a>B).</p>
</section>
</section>
<section id="increasing-mineralisation-over-the-course-of-the-experiment" class="level2">
<h2 class="anchored" data-anchor-id="increasing-mineralisation-over-the-course-of-the-experiment">Increasing mineralisation over the course of the experiment</h2>
<div id="fig-experiment-spectra" class="cell quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-1" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-1.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(a) Spectra from day 7 samples.</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-2.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(b) Spectra from day 12 samples.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-3" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-3.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(c) Spectra from day 16 samples.</figcaption><p></p>
</figure>
</div>
</div>
<div class="cell-output-display quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-experiment-spectra-4" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-experiment-spectra-4.png" class="img-fluid figure-img" data-ref-parent="fig-experiment-spectra" width="672"></p>
<p></p><figcaption class="figure-caption">(d) Spectra from day 24 samples (final product).</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure 6: Select spectra from all experiment sampling days.</figcaption><p></p>
</figure>
</div>
<p>Day 7 spectra have large O–H and amide A absorbance bands in stretching mode around 3400 cm<span class="math inline">\(^{-1}\)</span>, as well as three marked CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. There is a clear amide I peak at 1650 and a less pronounced amide II peak at 1545 cm<span class="math inline">\(^{-1}\)</span>. In the ‘fingerprint’ region, C–O<span class="math inline">\(_3^{2-}\)</span> at 1450 and 1400 absorbance bands corresponding to the v<sub>3</sub> asymmetric stretching vibrations, P–O<sub>4</sub> absorbance band corresponding to the v<sub>3</sub> asymmetric stretching vibrations at 1080 cm<span class="math inline">\(^{-1}\)</span>, and minor phosphate absorption bands around 500 cm<span class="math inline">\(^{-1}\)</span> in sample F7.1A6, but absent in sample F7.2D1. The absorption bands at 1080 cm<span class="math inline">\(^{-1}\)</span>, and 1040-1047 cm<span class="math inline">\(^{-1}\)</span> and minor bending absorption bands of the phosphate doublet around 605 cm<span class="math inline">\(^{-1}\)</span> and 560 cm<span class="math inline">\(^{-1}\)</span> in sample F7.2D1, but absent in sample F7.1A6. The relative absorbance of O–H and Amide I and II bands are higher than the phosphate bands, representing a relatively higher content of lipids and proteins than inorganic content. Large variation between spectra.</p>
<p>Day 12, amide I and II continue to be the dominant peaks, and a higher ratio of both amide and O–H to PO<sub>4</sub> v<sub>3</sub> absorbance bands. Three marked CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. Reduced variation between two of the three spectra.</p>
<p>Day 16, the ratio of O–H and amides to PO<sub>4</sub> has shifted, with the main peak shifting to the PO<sub>4</sub> v<sub>3</sub> absorbance band at 1040 (except in sample F16.1A2). A well-defined PO<sub>4</sub> doublet at 600 and 560 is present. Small CO<span class="math inline">\(_3^{2-}\)</span> asymmetric stretching at 1450 cm<span class="math inline">\(^{-1}\)</span> and 1415 cm<span class="math inline">\(^{-1}\)</span>, and stretching vibrations at 875-870 cm<span class="math inline">\(^{-1}\)</span>. Decreased variability between the spectra, with most spectra exhibiting a higher phosphate-to-protein/lipid ratio.</p>
<p>Day 24, large O–H and amide A absorbance bands in stretching mode around 3400 cm<span class="math inline">\(^{-1}\)</span>, as well as three minor CH<sub>3</sub> and CH<span class="math inline">\(_2\)</span> stretching vibrations at 2960, 2920, and 2850 cm<span class="math inline">\(^{-1}\)</span>. Main peak of spectra is PO<sub>4</sub> v<sub>3</sub> at 1040 cm<span class="math inline">\(^{-1}\)</span>, well-defined PO<sub>4</sub> doublet at 600-550 cm<span class="math inline">\(^{-1}\)</span>. Amide I band, with small amide II and III bands. Carbonate peaks also present. Very little variation between all of the spectra.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="figures/compare-calc-spectra-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Spectra from archaeological calculus, modern reference calculus, and artificial calculus are compared. Absorbance was shifted to allow comparison of the three spectra, so the absorbance should be interpreted as relative, not absolute.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The archaeological and modern reference spectra are largely indistinguishable and consist of an O–H absorbance band (3400 cm<span class="math inline">\(^{-1}\)</span>), CH<sub>3</sub> bands (3000–2900 cm<span class="math inline">\(^{-1}\)</span>), carbonate (1420, 1458-1450, 875-870 cm<span class="math inline">\(^{-1}\)</span>), amide I band (1650 cm<span class="math inline">\(^{-1}\)</span>), and phosphates (1040, 604, 566 cm<span class="math inline">\(^{-1}\)</span>). The model calculus samples from the end of the experiment are similar to both reference samples. The main difference is a lower organic component in reference samples seen as a reduced amide I peak at around 1637 compared to the carbonate peak at around 1420, and an absence of amide II and III. Also a reduction in CH<sub>3</sub> bands at 3000-2900 cm<span class="math inline">\(^{-1}\)</span>.</p>
<section id="similar-crystallinity-and-order-compared-to-reference-calculus" class="level3">
<h3 class="anchored" data-anchor-id="similar-crystallinity-and-order-compared-to-reference-calculus">Similar crystallinity and order compared to reference calculus</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div id="fig-grind-curve" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="figures/fig-grind-curve-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 7: (A) Grinding curves of multiple materials; and (B) calculus-only materials, including biofilm samples from three days, and an archaeological calclulus sample.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<!-- FTIR spectra comparison of artificial and reference dental calculus -->
<p>Samples were compared to the results of <span class="citation" data-cites="asscherAtomicDisorder2011">Asscher, Regev, et al. (<a href="#ref-asscherAtomicDisorder2011" role="doc-biblioref">2011</a>)</span> and <span class="citation" data-cites="asscherVariationsAtomic2011">Asscher, Weiner, et al. (<a href="#ref-asscherVariationsAtomic2011" role="doc-biblioref">2011</a>)</span>, and the slopes of the trend lines for our model calculus are similar to those of fresh bone and dentin. No appreciable differences between days 16, 20, and 24. The archaeological dental calculus does show a slightly increased slope compared to model calculus from the three sampling days used in the grind curve.</p>
</section>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<!-- Big summary/important statement for the science -->
<p>In this study we present a [long-term] oral biofilm model with mineralisation to produce artificial dental calculus. Our proposed use of the model is to address a variety of research questions related to dietary information extracted from dental calculus, in both modern and ancient samples. For that to be feasible, the model needs to act as a viable proxy to dental calculus grown under natural conditions, i.e., in the human mouth. It needs, as much as possible, to mimic the diversity and complexity of the natural oral microbiome, while also offering control over factors such as dietary input, growth conditions, and replicability within and between experiments. Here, we assessed the viability of our model as a proxy for dental calculus using metagenomic classification and FTIR analysis to explore the bacterial and mineral composition, and compare with oral reference samples.</p>
<section id="microbiome" class="level2">
<h2 class="anchored" data-anchor-id="microbiome">Microbiome</h2>
<!-- diversity and complexity within experiment and compared to oral references -->
<p>There is a loss of diversity from donated saliva to model calculus, and when compared to oral reference samples, which is a common limitation in biofilm models <span class="citation" data-cites="edlundBiofilmModel2013 bjarnsholtVivoBiofilm2013">(<a href="#ref-bjarnsholtVivoBiofilm2013" role="doc-biblioref">Bjarnsholt et al., 2013</a>; <a href="#ref-edlundBiofilmModel2013" role="doc-biblioref">Edlund et al., 2013</a>)</span>. The donated saliva for the experiment had a lower diversity than the reference saliva samples, and may have contributed to a lower diversity in experiment samples. Consequently, there is also a lower diversity and richness when compared to other modern oral reference samples, including oral mucosa, saliva, plaque, and calculus<!--alpha-div figs-->. Model calculus samples primarily consist of species within the genera <em>Veillonella</em> and <em>Enterococcus</em>, with more limited contributions from <em>Streptococcus</em> and <em>Prevotella</em>. The core microbiomes of oral reference samples have similar levels of streptococci, but a much more even distribution of species from <em>Actinomyces</em>, <em>Capnocytophaga</em>, <em>Fusobacterium</em>, and more<!-- Figure ref to pie charts -->.</p>
<!-- community profiles compared to oral references -->
<p>Medium samples from early in the experiment have similar species profiles to the donated saliva, but gradually diverge over the course of the experiment, as seen in the increasing separation of samples on PC1 in Figure<!-- ref to sPCA figure -->. This may be caused by the growth conditions provided by the experimental setup, which may not sufficiently mimic the oral environment, allowing different oral species to thrive that do not normally thrive in the natural oral environment. Especially <em>Enterococcus</em> spp. seemed to thrive later in the experiment, at the expense of more fastidious species like <em>Capnocytophaga</em> and <em>Neisseria</em> spp., which require an atmosphere with at least 5% carbon dioxide to thrive, so the model may have a low carbon dioxide atmosphere <span class="citation" data-cites="tonjumNeisseria2017">(<a href="#ref-tonjumNeisseria2017" role="doc-biblioref">Tønjum &amp; van Putten, 2017</a>)</span>. Early colonisers, including the aforementioned <em>Capnocytophaga</em> and <em>Neisseria</em> spp., as well as <em>Actinomyces</em>, were dominant in oral reference samples and deficient in model calculus samples, as well as the comparative <em>in vitro</em> model. Not only is there a decreased species diversity between model and reference samples, but also a loss of diversity in aerotolerance. Of the dominant species in the model, most are either anaerobes or facultative anaerobes, predominantly Gram-negative <em>Veillonella</em> and <em>Prevotella</em>, and Gram-positive <em>Enterococcus</em>. Species within predominantly aerobic genera, such as <em>Neisseria</em>, <em>Rothia</em>, and <em>Ottowia</em>, are deficient in the model compared to the donated saliva and other oral samples<!--PC loading plots-->, suggesting a shift from a largely aerotolerant profile to an anaerobic profile during the experiment. While our model is not set up as an anaerobic system, the anaerobes seem to have outcompeted aerobes and, to some extent, facultative anaerobes. This is likely a result of communities of bacteria within the biofilm creating favourable microenvironments facilitated by the protective properties of the biofilm matrix <span class="citation" data-cites="flemmingBiofilmsEmergent2016 edlundUncoveringComplex2018">(<a href="#ref-edlundUncoveringComplex2018" role="doc-biblioref">Edlund et al., 2018</a>; <a href="#ref-flemmingBiofilmsEmergent2016" role="doc-biblioref">Flemming et al., 2016</a>)</span>. <em>Capnocytophaga</em> and <em>Actinomyces</em> spp. are predominantly (facultative) anaerobes, so their deficiency must be attributed to different reasons. The highest log-fold changes between model calculus and other samples were seen in <em>E. faecalis</em>, <em>E. durans</em>, and <em>E. casseliflavus</em>, and contributed most to the unique clustering of model calculus on PC1<!-- sPCA and DA figures-->. <em>E. faecalis</em> was also abundant in the comparative <em>in vitro</em> biofilm study<!-- large species plot-->. <em>Enterococcus</em> spp. out-competing other species may be a problem in model biofilm studies, but more comparative studies are needed to confirm (and there are limited WGS studies to use as a comparison).</p>
<p>Model calculus and the modern reference calculus clustered separately in the sPCA analysis. This may be due to human variation, as there can be large differences in the oral microbiome of two individuals, and modern reference samples were taken from the same geographical origin (as was the donor saliva for the model, which was taken from a different geographical origin). One of the main drivers of differences in species profiles has been previously shown to be individual variation, even more so than pathological condition <span class="citation" data-cites="velskoAncientDental2022a">(<a href="#ref-velskoAncientDental2022a" role="doc-biblioref">Velsko et al., 2022</a>)</span>. Whether or not distinct microbial profiles will affect the retention of dietary incorporation remains to be seen, and likely depends on the role bacteria play in the incorporation and retention of dietary compounds and microremains within a biofilm. Samples within the experiment were quite consistent allowing this to be explored further.</p>
<p>Species related to caries, such as <em>Steptococcus mutans</em> and members of the <em>Lactobacillus</em> genus, were deficient in both model calculus samples and oral reference samples. Both are integral in creating an acidic environment that favours demineralisation and inhibits mineralisation, so there absence is expected in a calculus model <span class="citation" data-cites="velskoMicrobialDifferences2019 edlundBiofilmModel2013">(<a href="#ref-edlundBiofilmModel2013" role="doc-biblioref">Edlund et al., 2013</a>; <a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al., 2019</a>)</span>. The abundance of <em>Veillonella</em> and <em>Prevotella</em> spp. could suggest a pathogenic potential within the model biofilm, as these are commonly reported as opportunistic or accessory pathogens in periodontal disease <span class="citation" data-cites="zhouVeillonellaeBridging2021 tettPrevotellaDiversity2021">(<a href="#ref-tettPrevotellaDiversity2021" role="doc-biblioref">Tett et al., 2021</a>; <a href="#ref-zhouVeillonellaeBridging2021" role="doc-biblioref">Zhou et al., 2021</a>)</span>.</p>
</section>
<section id="mineralisation" class="level2">
<h2 class="anchored" data-anchor-id="mineralisation">Mineralisation</h2>
<!-- mineralisation process - FTIR - gradual mineralisation -->
<p>FTIR analysis allowed us to address the mineralisation process of the model. The results showed an increasing mineral composition over the course of the experiment as the model biofilm matured. Samples taken early in the experiment, on day 7 and 12, had a high lipid and protein content consistent with the presence of microbial DNA and RNA within a matrix of extracellular polysaccharides <span class="citation" data-cites="jainIsolationCharacterization2013">(<a href="#ref-jainIsolationCharacterization2013" role="doc-biblioref">Jain et al., 2013</a>)</span>. The high water content, indicated by the O–H stretch, is also consistent with a biofilm, which is around 90% water <span class="citation" data-cites="bergerOralBiofilms2018">(<a href="#ref-bergerOralBiofilms2018" role="doc-biblioref">Berger et al., 2018</a>)</span>. As the samples mature over the course of the experiment, the ratio of proteins and lipids to phosphates shift from predominance of organic content to inorganic content in the form of carbonated hydroxyapatite<!--fig reference-->.</p>
<p>The artificial samples from day 24 are consistent with spectra of carbonated hydroxyapatite and resemble both the modern reference samples and the archaeological sample in mineral composition and crystallinity. The steeper slope in the grind curve plots of the archaeological sample may suggest that the crystals in archaeological samples are larger than in the model calculus <a href="#fig-grind-curve">Figure&nbsp;7</a>. A possible explanation is that the inorganic crystals within archaeological calculus have had more time to expand into the space left by degraded organic matter <span class="citation" data-cites="weinerBiologicalMaterials2010">(<a href="#ref-weinerBiologicalMaterials2010" role="doc-biblioref">Weiner, 2010b</a>)</span>. It may also have been an unusually crystallised sample; as we only analysed a single archaeological sample, and more modern and archaeological samples are needed to see if the high level of crystallinity is consistent in archaeological calculus, and if the level of crystallinity of the model calulus is more comparable to modern reference samples. The grinding curves for days 16, 20, and 24 were very similar, suggesting that the CPMU solution, which is introduced on day 16 to promote mineralisation, may not have much of an effect.</p>
<p>Although there were differences in species profiles between modern and model calculus, the mineral composition of the end results were similar, reinforcing the idea that, under the right circumstances, most bacteria will mineralise <span class="citation" data-cites="moorerCalcificationCariogenic1993">(<a href="#ref-moorerCalcificationCariogenic1993" role="doc-biblioref">Moorer et al., 1993</a>)</span>. Even occurring without one of the most well-known biomineraliser, <em>Corynebacterium matruchotii</em>, which was deficient in model, and was the strongest discriminator of oral samples from model samples<!--sPCA loadings fig-->. It may also suggest that, while bacteria are integral in the formation of dental calculus, and inevitably serve as part of the structure that dental calculus is built upon <span class="citation" data-cites="rohanizadehUltrastructuralStudy2005">(<a href="#ref-rohanizadehUltrastructuralStudy2005" role="doc-biblioref">Rohanizadeh &amp; LeGeros, 2005</a>)</span>, the species composition of the biofilm communities may be less important. And that mineralisation may largely be a chemical process (catalysed by biological factors) <span class="citation" data-cites="omelonReviewPhosphate2013">(<a href="#ref-omelonReviewPhosphate2013" role="doc-biblioref">Omelon et al., 2013</a>)</span>.</p>
</section>
<section id="replicability" class="level2">
<h2 class="anchored" data-anchor-id="replicability">Replicability</h2>
<!-- replicability within the experiment -->
<p>The model calculus displayed similar species diversity and richness indices across all samples, indicating a high level of replicability between samples in the experimental run, despite larger variation between medium samples<!-- figure reference -->. Both sPCA analyses conducted with experiment only samples and with oral reference samples also cluster model calculus closely together. It remains to be seen whether the replicability within the experiment also scales up to between-experiment replicability. The variation in mineral composition was initially high in early model biofilm samples from days 7 and 12, but starting day 16 the variation decreased, and continued to decrease until day 24. The model calculus samples from day 24 were largely similar in composition as observed in the FTIR spectra<!--fig ref-->. The use of a simple multiwell plate setup allows us to [submit] many samples to the same conditions which promotes replicability between samples<!--cite-->. While the experimental conditions, including the dietary input, are kept constant, and the mineral and bacterial profiles are similar between samples, there is still a relatively large variation in starch retention in the model calculus <span class="citation" data-cites="bartholdyInvestigatingBiases2022">(<a href="#ref-bartholdyInvestigatingBiases2022" role="doc-biblioref">Bartholdy &amp; Henry, 2022</a>)</span>.</p>
<!-- secondary findings -->
<!-- Scientific findings as extension of previous work -->
<!-- How this engineering will facilitate future research -->
<p>There are many biofilm models to choose from, so developing a new protocol may seem counter-productive; however, few are developed for long-term growth and even fewer for mineralisation. One of the exceptions involves a complex setup that is likely unfeasible to most archaeological research <span class="citation" data-cites="sissonsMultistationPlaque1991">(<a href="#ref-sissonsMultistationPlaque1991" role="doc-biblioref">Sissons et al., 1991</a>)</span>. Our model uses a simplified high-throughput setup more commonly seen in shorter-term biofilm models that mainly focus on dental plaque <span class="citation" data-cites="extercateAAA2010 tianUsingDGGE2010">(<a href="#ref-extercateAAA2010" role="doc-biblioref">Exterkate et al., 2010</a>; <a href="#ref-tianUsingDGGE2010" role="doc-biblioref">Tian et al., 2010</a>)</span>. As shown in a previous study, calculus and plaque have distinct microbial profiles <span class="citation" data-cites="velskoMicrobialDifferences2019">(<a href="#ref-velskoMicrobialDifferences2019" role="doc-biblioref">Velsko et al., 2019</a>)</span>, so their applicability to explore archaeological questions on dental calculus are limited. Clinical studies are also largely focused on short-term effects of anti-microbial treatments and inhibition of biofilm formation<!--citations-->, while archaeologists are more interested in questions related to … Questions, we believe, can to some extent be answered using an oral biofilm model. Our previous work has shown that, like dental calculus, the model calculus is able to trap and retain dietary starches, admittedly with some variability between samples <span class="citation" data-cites="bartholdyInvestigatingBiases2022">(<a href="#ref-bartholdyInvestigatingBiases2022" role="doc-biblioref">Bartholdy &amp; Henry, 2022</a>)</span>.</p>
</section>
<section id="limitations" class="level2">
<h2 class="anchored" data-anchor-id="limitations">Limitations</h2>
<p>A well-known limitation of biofilm models is the difficulty in capturing the diversity and complexity of the natural oral biome. Diversity and complexity may be represented as interspecies communities and complex metabolic dependencies between organisms within the communitues, or as an environmental complexity determined by nutrient availability, host immune-responses to biofilms, and fluctuating microenvironments across the biofilm in response to these factors <span class="citation" data-cites="edlundUncoveringComplex2018 bjarnsholtVivoBiofilm2013">(<a href="#ref-bjarnsholtVivoBiofilm2013" role="doc-biblioref">Bjarnsholt et al., 2013</a>; <a href="#ref-edlundUncoveringComplex2018" role="doc-biblioref">Edlund et al., 2018</a>)</span>. Our single-donor approach may have affected diversity of model. The donated saliva had a lower mean alpha-diversity than other saliva samples. Although benefit of single donor means maintaining the integrity of the original bacterial community. There are benefits and trade-offs for both pooled and single-donor saliva <span class="citation" data-cites="edlundBiofilmModel2013">(<a href="#ref-edlundBiofilmModel2013" role="doc-biblioref">Edlund et al., 2013</a>)</span>.</p>
<!-- Findings' scientific implications -->
<!-- Forward-looking statements about the field as a whole -->
<!-- a summary/importance of the engineering aspect of the work -->
<!-- How it facilitates scientific research -->
<!-- limitations of the platform show which questions would be unanswered -->
<p>Research relating to <span class="math inline">\(alpha\)</span>-amylase activity and its effect on the oral biome (whether dietary or otherwise) will require addition of amylase to the model during the experiment, as it is not present naturally in our current setup (as shown in <span class="citation" data-cites="bartholdyInvestigatingBiases2022">Bartholdy &amp; Henry (<a href="#ref-bartholdyInvestigatingBiases2022" role="doc-biblioref">2022</a>)</span>). Our model has no renewable source for <span class="math inline">\(alpha\)</span>-amylase once the inoculations have been completed. There are streptococci present in the model, which are known for their ability to bind amylase <span class="citation" data-cites="nikitkovaStarchBiofilms2013">(<a href="#ref-nikitkovaStarchBiofilms2013" role="doc-biblioref">Nikitkova et al., 2013</a>)</span>; however, it is unclear whether the correct strains are present in our model, and if they are, why they are not binding <span class="math inline">\(alpha\)</span>-amylase provided by the saliva inoculate. This could be due to the frequency of medium replacement potentially clearing out all of the unbound amylase every time medium is replaced. Our current model does not contain serum, which may be contributing to the deficiency of slow-growing fastidious organisms <span class="citation" data-cites="tianUsingDGGE2010">(<a href="#ref-tianUsingDGGE2010" role="doc-biblioref">Tian et al., 2010</a>)</span>.</p>
<p>Biofilms were grown in a standard shaking incubator, rather than a incubator specific to cell cultures. The lack of complex environmental control may cause the model to deviate from its natural growth over the 24 days that the experiment is run, due to a lack of precise control over physiological conditions.</p>
<p>There is also a possibility of contamination in the model. While the CPMU solution was prepared under sterile conditions, the solution itself was not autoclaved, and may have been a source of contamination. This is further … by the fact that none of the medium samples taken after the introduction of CPMU (day 14) made it past authentication in the metagenomic analysis, and that a large portion of the sources of many samples were estimated to be from indoor air and surface samples <!-- sourcetracker plot ref -->.</p>
<p>Another limitation involves the information on aerotolerance of the identified species. Species-level aerotolerance was collected from BacDive<!--reference, url, etc.--> and, at the time of writing<!--or rather, last time the manuscript was rendered-->, 55.9% species were missing aerotolerance values. This was mitigated by aggregating genus-level tolerances to missing species, and may have some errors (although unlikely to make any significant changes).</p>
<!-- limitations to biofilm models -->
<!-- Is it possible that the mixture of sucrose in the starch solutions eliminates the need to feed on starch? -->
<!-- modern calculus references samples were all from the same geographic origin -->
<!-- How it will facilitate the specific topic of research (e.g. archaeology) -->
</section>
<section id="future-work" class="level2">
<h2 class="anchored" data-anchor-id="future-work">Future work</h2>
<p>Our goals for additional validation measures involve functional profiles of bacterial, to see if metabolic behaviour of bacteria is consistent with <em>in vivo</em> conditions, and whether this is affected by the presence/absence of amylase. Further protocol optimisation will also be necessary to address some of the limitations of our current model. We may need to reduce the frequency of medium replacement (currently every three days) to help promote the growth of slower growing organisms and limit generalists such as enterococci. More infrequent medium replacement would facilitate slow-growing bacteria in establishing their metabolic relationships, allowing the byproducts of some species to become abundant enough for others that depend on these to grow <span class="citation" data-cites="marshDentalPlaque2005">(<a href="#ref-marshDentalPlaque2005" role="doc-biblioref">Marsh, 2005</a>)</span>. Finally, if the within-experiment replicability is also present between experiments. An important question to address is what role bacteria play in the incorporation and retention of dietary microremains, and whether differing bacterial profiles affects the dietary incorporation and retention within the biofilm. Regardless, the apparent replicability of our current setup is a promising result.</p>
<p>The calculus model we have presented has many potential benefits within archaeological research. It can be used to test many fundamental assumptions we have about the process of inclusion and subsequent extraction of various information about health and diet from archaeological dental calculus. How our current methods for exploring diet and health may be biased. How different methods of food processing in the past may reflect the picture we get from the samples we analyse. This method can be used for a wide range of methods-testing (e.g.&nbsp;DNA, proteomics, etc.) as well as training for various sampling methods, and testing decontamination protocols, without using up limited supplies of archaeological resources.</p>
<!-- ultimately we are left with more questions than answers, but isn't that the 
beauty of exploratory science? -->
</section>
</section>
<section id="references" class="level1 unnumbered">
<h1 class="unnumbered">References</h1>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-adlerSequencingAncient2013" class="csl-entry" role="doc-biblioentry">
Adler, C. J., Dobney, K., Weyrich, L. S., Kaidonis, J., Walker, A. W., Haak, W., Bradshaw, C. J., Townsend, G., Soltysiak, A., Alt, K. W., Parkhill, J., &amp; Cooper, A. (2013). Sequencing ancient calcified dental plaque shows changes in oral microbiota with dietary shifts of the <span>Neolithic</span> and <span>Industrial</span> revolutions. <em>Nature Genetics</em>, <em>45</em>(4), 450–455, 455e1. <a href="https://doi.org/10.1038/ng.2536">https://doi.org/10.1038/ng.2536</a>
</div>
<div id="ref-asscherAtomicDisorder2011" class="csl-entry" role="doc-biblioentry">
Asscher, Y., Regev, L., Weiner, S., &amp; Boaretto, E. (2011). Atomic <span>Disorder</span> in <span>Fossil Tooth</span> and <span>Bone Mineral</span>: <span>An FTIR Study Using</span> the <span>Grinding Curve Method</span>. <em>ArcheoSciences. Revue d’archéométrie</em>, <em>35, 35</em>, 135–141. <a href="https://doi.org/10.4000/archeosciences.3062">https://doi.org/10.4000/archeosciences.3062</a>
</div>
<div id="ref-asscherVariationsAtomic2011" class="csl-entry" role="doc-biblioentry">
Asscher, Y., Weiner, S., &amp; Boaretto, E. (2011). Variations in <span>Atomic Disorder</span> in <span>Biogenic Carbonate Hydroxyapatite Using</span> the <span>Infrared Spectrum Grinding Curve Method</span>. <em>Advanced Functional Materials</em>, <em>21</em>(17), 3308–3313. <a href="https://doi.org/10.1002/adfm.201100266">https://doi.org/10.1002/adfm.201100266</a>
</div>
<div id="ref-bartholdyInvestigatingBiases2022" class="csl-entry" role="doc-biblioentry">
Bartholdy, B. P., &amp; Henry, A. G. (2022). Investigating <span>Biases Associated With Dietary Starch Incorporation</span> and <span>Retention With</span> an <span>Oral Biofilm Model</span>. <em>Frontiers in Earth Science</em>, <em>10</em>. <a href="https://www.frontiersin.org/articles/10.3389/feart.2022.886512">https://www.frontiersin.org/articles/10.3389/feart.2022.886512</a>
</div>
<div id="ref-bergerOralBiofilms2018" class="csl-entry" role="doc-biblioentry">
Berger, D., Rakhamimova, A., Pollack, A., &amp; Loewy, Z. (2018). Oral <span>Biofilms</span>: <span>Development</span>, <span>Control</span>, and <span>Analysis</span>. <em>High-Throughput</em>, <em>7</em>(3), 24. <a href="https://doi.org/10.3390/ht7030024">https://doi.org/10.3390/ht7030024</a>
</div>
<div id="ref-bjarnsholtVivoBiofilm2013" class="csl-entry" role="doc-biblioentry">
Bjarnsholt, T., Alhede, M., Alhede, M., Eickhardt-Sørensen, S. R., Moser, C., Kühl, M., Jensen, P. Ø., &amp; Høiby, N. (2013). The in vivo biofilm. <em>Trends in Microbiology</em>, <em>21</em>(9), 466–474. <a href="https://doi.org/10.1016/j.tim.2013.06.002">https://doi.org/10.1016/j.tim.2013.06.002</a>
</div>
<div id="ref-buckleyDentalCalculus2014" class="csl-entry" role="doc-biblioentry">
Buckley, S., Usai, D., Jakob, T., Radini, A., &amp; Hardy, K. (2014). Dental <span>Calculus Reveals Unique Insights</span> into <span>Food Items</span>, <span>Cooking</span> and <span>Plant Processing</span> in <span>Prehistoric Central Sudan</span>. <em>PLOS ONE</em>, <em>9</em>(7), e100808. <a href="https://doi.org/10.1371/journal.pone.0100808">https://doi.org/10.1371/journal.pone.0100808</a>
</div>
<div id="ref-ceriCalgaryBiofilm1999" class="csl-entry" role="doc-biblioentry">
Ceri, H., Olson, M. E., Stremick, C., Read, R. R., Morck, D., &amp; Buret, A. (1999). The <span>Calgary Biofilm Device</span>: <span>New Technology</span> for <span>Rapid Determination</span> of <span>Antibiotic Susceptibilities</span> of <span>Bacterial Biofilms</span>. <em>Journal of Clinical Microbiology</em>, <em>37</em>(6), 1771–1776. <a href="https://doi.org/10.1128/JCM.37.6.1771-1776.1999">https://doi.org/10.1128/JCM.37.6.1771-1776.1999</a>
</div>
<div id="ref-Rdecontam" class="csl-entry" role="doc-biblioentry">
Davis, N. M., Proctor, D. M., Holmes, S. P., Relman, D. A., &amp; Callahan, B. J. (2018). Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data. <em>Microbiome</em>, <em>6</em>(1), 226. <a href="https://doi.org/10.1186/s40168-018-0605-2">https://doi.org/10.1186/s40168-018-0605-2</a>
</div>
<div id="ref-edlundBiofilmModel2013" class="csl-entry" role="doc-biblioentry">
Edlund, A., Yang, Y., Hall, A. P., Guo, L., Lux, R., He, X., Nelson, K. E., Nealson, K. H., Yooseph, S., Shi, W., &amp; McLean, J. S. (2013). An in vitrobiofilm model system maintaining a highly reproducible species and metabolic diversity approaching that of the human oral microbiome. <em>Microbiome</em>, <em>1</em>(1), 25. <a href="https://doi.org/10.1186/2049-2618-1-25">https://doi.org/10.1186/2049-2618-1-25</a>
</div>
<div id="ref-edlundUncoveringComplex2018" class="csl-entry" role="doc-biblioentry">
Edlund, A., Yang, Y., Yooseph, S., He, X., Shi, W., &amp; McLean, J. S. (2018). Uncovering complex microbiome activities via metatranscriptomics during 24 hours of oral biofilm assembly and maturation. <em>Microbiome</em>, <em>6</em>(1), 217. <a href="https://doi.org/10.1186/s40168-018-0591-4">https://doi.org/10.1186/s40168-018-0591-4</a>
</div>
<div id="ref-eerkensDentalCalculus2018" class="csl-entry" role="doc-biblioentry">
Eerkens, J. W., Tushingham, S., Brownstein, K. J., Garibay, R., Perez, K., Murga, E., Kaijankoski, P., Rosenthal, J. S., &amp; Gang, D. R. (2018). Dental calculus as a source of ancient alkaloids: <span>Detection</span> of nicotine by <span>LC-MS</span> in calculus samples from the <span>Americas</span>. <em>Journal of Archaeological Science: Reports</em>, <em>18</em>, 509–515. <a href="https://doi.org/10.1016/j.jasrep.2018.02.004">https://doi.org/10.1016/j.jasrep.2018.02.004</a>
</div>
<div id="ref-extercateAAA2010" class="csl-entry" role="doc-biblioentry">
Exterkate, R. A. M., Crielaard, W., &amp; Ten Cate, J. M. (2010). Different <span>Response</span> to <span>Amine Fluoride</span> by <span>Streptococcus</span> mutans and <span>Polymicrobial Biofilms</span> in a <span>Novel High-Throughput Active Attachment Model</span>. <em>Caries Research</em>, <em>44</em>(4), 372–379. <a href="https://doi.org/10.1159/000316541">https://doi.org/10.1159/000316541</a>
</div>
<div id="ref-yatesEAGER2020" class="csl-entry" role="doc-biblioentry">
Fellows Yates, J. A., Lamnidis, T. C., Borry, M., Valtueña, A. A., Fagernäs, Z., Clayton, S., Garcia, M. U., Neukamm, J., &amp; Peltzer, A. (2020). Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager. <em>bioRxiv</em>, 2020.06.11.145615. <a href="https://doi.org/10.1101/2020.06.11.145615">https://doi.org/10.1101/2020.06.11.145615</a>
</div>
<div id="ref-yatesOralMicrobiome2021" class="csl-entry" role="doc-biblioentry">
Fellows Yates, J. A., Velsko, I. M., Aron, F., Posth, C., Hofman, C. A., Austin, R. M., Parker, C. E., Mann, A. E., Nägele, K., Arthur, K. W., Arthur, J. W., Bauer, C. C., Crevecoeur, I., Cupillard, C., Curtis, M. C., Dalén, L., Bonilla, M. D.-Z., Fernández-Lomana, J. C. D., Drucker, D. G., … Warinner, C. (2021). The evolution and changing ecology of the <span>African</span> hominid oral microbiome. <em>Proceedings of the National Academy of Sciences</em>, <em>118</em>(20). <a href="https://doi.org/10.1073/pnas.2021655118">https://doi.org/10.1073/pnas.2021655118</a>
</div>
<div id="ref-filochePlaqueMicrocosm2007" class="csl-entry" role="doc-biblioentry">
Filoche, S. K., Soma, K. J., &amp; Sissons, C. H. (2007). Caries-related plaque microcosm biofilms developed in microplates. <em>Oral Microbiology and Immunology</em>, <em>22</em>(2), 73–79. <a href="https://doi.org/10.1111/j.1399-302X.2007.00323.x">https://doi.org/10.1111/j.1399-302X.2007.00323.x</a>
</div>
<div id="ref-flemmingBiofilmsEmergent2016" class="csl-entry" role="doc-biblioentry">
Flemming, H.-C., Wingender, J., Szewzyk, U., Steinberg, P., Rice, S. A., &amp; Kjelleberg, S. (2016). Biofilms: An emergent form of bacterial life. <em>Nature Reviews Microbiology</em>, <em>14</em>(9), 563–575. <a href="https://doi.org/10.1038/nrmicro.2016.94">https://doi.org/10.1038/nrmicro.2016.94</a>
</div>
<div id="ref-glasBiophysicalStudies1962" class="csl-entry" role="doc-biblioentry">
Glas, J.-E., &amp; Krasse, B. (1962). Biophysical <span>Studies</span> on <span>Dental Calculus</span> from <span>Germfree</span> and <span>Conventional Rats</span>. <em>Acta Odontologica Scandinavica</em>, <em>20</em>(2), 127–134. <a href="https://doi.org/10.3109/00016356209026100">https://doi.org/10.3109/00016356209026100</a>
</div>
<div id="ref-gloorMicrobiomeDatasets2017" class="csl-entry" role="doc-biblioentry">
Gloor, G. B., Macklaim, J. M., Pawlowsky-Glahn, V., &amp; Egozcue, J. J. (2017). Microbiome <span>Datasets Are Compositional</span>: <span>And This Is Not Optional</span>. <em>Frontiers in Microbiology</em>, <em>8</em>, 2224. <a href="https://doi.org/10.3389/fmicb.2017.02224">https://doi.org/10.3389/fmicb.2017.02224</a>
</div>
<div id="ref-hardyStarchGranulesDental2009" class="csl-entry" role="doc-biblioentry">
Hardy, K., Blakeney, T., Copeland, L., Kirkham, J., Wrangham, R., &amp; Collins, M. (2009). Starch granules, dental calculus and new perspectives on ancient diet. <em>Journal of Archaeological Science</em>, <em>36</em>(2), 248–255. <a href="https://doi.org/10.1016/j.jas.2008.09.015">https://doi.org/10.1016/j.jas.2008.09.015</a>
</div>
<div id="ref-hayashizakiSiteSpecific2008" class="csl-entry" role="doc-biblioentry">
Hayashizaki, J., Ban, S., Nakagaki, H., Okumura, A., Yoshii, S., &amp; Robinson, C. (2008). Site specific mineral composition and microstructure of human supra-gingival dental calculus. <em>Archives of Oral Biology</em>, <em>53</em>(2), 168–174. <a href="https://doi.org/10.1016/j.archoralbio.2007.09.003">https://doi.org/10.1016/j.archoralbio.2007.09.003</a>
</div>
<div id="ref-hendyProteomicCalculus2018" class="csl-entry" role="doc-biblioentry">
Hendy, J., Warinner, C., Bouwman, A., Collins, M. J., Fiddyment, S., Fischer, R., Hagan, R., Hofman, C. A., Holst, M., Chaves, E., Klaus, L., Larson, G., Mackie, M., McGrath, K., Mundorff, A. Z., Radini, A., Rao, H., Trachsel, C., Velsko, I. M., &amp; Speller, C. F. (2018). Proteomic evidence of dietary sources in ancient dental calculus. <em>Proceedings. Biological Sciences</em>, <em>285</em>(1883), 20180977. <a href="https://doi.org/10.1098/rspb.2018.0977">https://doi.org/10.1098/rspb.2018.0977</a>
</div>
<div id="ref-henryCalculusSyria2008" class="csl-entry" role="doc-biblioentry">
Henry, A. G., &amp; Piperno, D. R. (2008). Using plant microfossils from dental calculus to recover human diet: A case study from <span>Tell</span> al-<span>Raqā</span>’i, <span>Syria</span>. <em>Journal of Archaeological Science</em>, <em>35</em>(7), 1943–1950. <a href="https://doi.org/10.1016/j.jas.2007.12.005">https://doi.org/10.1016/j.jas.2007.12.005</a>
</div>
<div id="ref-jainIsolationCharacterization2013" class="csl-entry" role="doc-biblioentry">
Jain, K., Parida, S., Mangwani, N., Dash, H. R., &amp; Das, S. (2013). Isolation and characterization of biofilm-forming bacteria and associated extracellular polymeric substances from oral cavity. <em>Annals of Microbiology</em>, <em>63</em>(4), 1553–1562. <a href="https://doi.org/10.1007/s13213-013-0618-9">https://doi.org/10.1007/s13213-013-0618-9</a>
</div>
<div id="ref-jiFluorideMagnesium2000" class="csl-entry" role="doc-biblioentry">
Ji, H., Nakagaki, H., Hayashizaki, J., Tsuboi, S., Kato, K., Toyama, A., Arai, K., Thuy, T. T., Ha, N. T. T., Kameyama, Y., Kirkham, J., &amp; Robinson, C. (2000). Fluoride and magnesium concentrations in human dental calculus obtained from <span>Japanese</span> and <span>Chinese</span> patients. <em>Archives of Oral Biology</em>, <em>45</em>(7), 611–615. <a href="https://doi.org/10.1016/S0003-9969(00)00021-2">https://doi.org/10.1016/S0003-9969(00)00021-2</a>
</div>
<div id="ref-jinSupragingivalCalculus2002" class="csl-entry" role="doc-biblioentry">
Jin, Y., &amp; Yip, H.-K. (2002). Supragingival <span>Calculus</span>: <span>Formation</span> and <span>Control</span>. <em>Critical Reviews in Oral Biology &amp; Medicine</em>. <a href="https://doi.org/10.1177/154411130201300506">https://doi.org/10.1177/154411130201300506</a>
</div>
<div id="ref-kazarinaPostmedievalMicrobial2021" class="csl-entry" role="doc-biblioentry">
Kazarina, A., Petersone-Gordina, E., Kimsis, J., Kuzmicka, J., Zayakin, P., Griškjans, Ž., Gerhards, G., &amp; Ranka, R. (2021). The <span>Postmedieval Latvian Oral Microbiome</span> in the <span>Context</span> of <span>Modern Dental Calculus</span> and <span>Modern Dental Plaque Microbial Profiles</span>. <em>Genes</em>, <em>12</em>(2), 309. <a href="https://doi.org/10.3390/genes12020309">https://doi.org/10.3390/genes12020309</a>
</div>
<div id="ref-knightsSourceTracker2011" class="csl-entry" role="doc-biblioentry">
Knights, D., Kuczynski, J., Charlson, E. S., Zaneveld, J., Mozer, M. C., Collman, R. G., Bushman, F. D., Knight, R., &amp; Kelley, S. T. (2011). Bayesian community-wide culture-independent microbial source tracking. <em>Nature Methods</em>, <em>8</em>(9), 761–763. <a href="https://doi.org/10.1038/nmeth.1650">https://doi.org/10.1038/nmeth.1650</a>
</div>
<div id="ref-BWA" class="csl-entry" role="doc-biblioentry">
Li, H., &amp; Durbin, R. (2009). Fast and accurate short read alignment with <span>Burrows</span>–<span>Wheeler</span> transform. <em>Bioinformatics</em>, <em>25</em>(14), 1754–1760. <a href="https://doi.org/10.1093/bioinformatics/btp324">https://doi.org/10.1093/bioinformatics/btp324</a>
</div>
<div id="ref-maHumanDiet2022" class="csl-entry" role="doc-biblioentry">
Ma, Z., Liu, S., Li, Z., Ye, M., &amp; Huan, X. (2022). Human <span>Diet Patterns During</span> the <span>Qijia Cultural Period</span>: <span>Integrated Evidence</span> of <span>Stable Isotopes</span> and <span class="nocase">Plant Micro-remains From</span> the <span>Lajia Site</span>, <span>Northwest China</span>. <em>Frontiers in Earth Science</em>, <em>10</em>. <a href="https://www.frontiersin.org/articles/10.3389/feart.2022.884856">https://www.frontiersin.org/articles/10.3389/feart.2022.884856</a>
</div>
<div id="ref-marshDentalPlaque2006" class="csl-entry" role="doc-biblioentry">
Marsh, P. D. (2006). Dental plaque as a biofilm and a microbial community – implications for health and disease. <em>BMC Oral Health</em>, <em>6</em>(S1), S14. <a href="https://doi.org/10.1186/1472-6831-6-S1-S14">https://doi.org/10.1186/1472-6831-6-S1-S14</a>
</div>
<div id="ref-marshDentalPlaque2005" class="csl-entry" role="doc-biblioentry">
Marsh, P. D. (2005). Dental plaque: Biological significance of a biofilm and community life-style. <em>Journal of Clinical Periodontology</em>, <em>32</em>(s6), 7–15. <a href="https://doi.org/10.1111/j.1600-051X.2005.00790.x">https://doi.org/10.1111/j.1600-051X.2005.00790.x</a>
</div>
<div id="ref-mcbainBiofilmModels2009" class="csl-entry" role="doc-biblioentry">
McBain, A. J. (2009). In <span>Vitro Biofilm Models</span>: <span>An Overview</span>. In <em>Advances in <span>Applied Microbiology</span></em> (Vol. 69, pp. 99–132). <span>Academic Press</span>. <a href="https://doi.org/10.1016/S0065-2164(09)69004-3">https://doi.org/10.1016/S0065-2164(09)69004-3</a>
</div>
<div id="ref-mickleburghNewInsights2012" class="csl-entry" role="doc-biblioentry">
Mickleburgh, H. L., &amp; Pagán-Jiménez, J. R. (2012). New insights into the consumption of maize and other food plants in the pre-<span>Columbian Caribbean</span> from starch grains trapped in human dental calculus. <em>Journal of Archaeological Science</em>, <em>39</em>(7), 2468–2478. <a href="https://doi.org/10.1016/j.jas.2012.02.020">https://doi.org/10.1016/j.jas.2012.02.020</a>
</div>
<div id="ref-middletonVitroCalculus1965" class="csl-entry" role="doc-biblioentry">
Middleton, J. D. (1965). Human salivary proteins and artificial calculus formation in vitro. <em>Archives of Oral Biology</em>, <em>10</em>(2), 227–235. <a href="https://doi.org/10.1016/0003-9969(65)90024-5">https://doi.org/10.1016/0003-9969(65)90024-5</a>
</div>
<div id="ref-moorerCalcificationCariogenic1993" class="csl-entry" role="doc-biblioentry">
Moorer, W. R., Ten Cate, J. M., &amp; Buijs, J. F. (1993). Calcification of a <span>Cariogenic Streptococcus</span> and of <span>Corynebacterium</span> (<span>Bacterionema</span>) matruchotii. <em>Journal of Dental Research</em>, <em>72</em>(6), 1021–1026. <a href="https://doi.org/10.1177/00220345930720060501">https://doi.org/10.1177/00220345930720060501</a>
</div>
<div id="ref-nikitkovaStarchBiofilms2013" class="csl-entry" role="doc-biblioentry">
Nikitkova, A. E., Haase, E. M., &amp; Scannapieco, F. A. (2013). Taking the <span>Starch</span> out of <span>Oral Biofilm Formation</span>: <span>Molecular Basis</span> and <span>Functional Significance</span> of <span>Salivary</span> α-<span>Amylase Binding</span> to <span>Oral Streptococci</span>. <em>Applied and Environmental Microbiology</em>, <em>79</em>(2), 416–423. <a href="https://doi.org/10.1128/AEM.02581-12">https://doi.org/10.1128/AEM.02581-12</a>
</div>
<div id="ref-Rvegan" class="csl-entry" role="doc-biblioentry">
Oksanen, J., Simpson, G. L., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Solymos, P., Stevens, M. H. H., Szoecs, E., Wagner, H., Barbour, M., Bedward, M., Bolker, B., Borcard, D., Carvalho, G., Chirico, M., De Caceres, M., Durand, S., … Weedon, J. (2022). <em>Vegan: <span>Community</span> ecology package</em> [Manual]. <a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>
</div>
<div id="ref-omelonReviewPhosphate2013" class="csl-entry" role="doc-biblioentry">
Omelon, S., Ariganello, M., Bonucci, E., Grynpas, M., &amp; Nanci, A. (2013). A <span>Review</span> of <span>Phosphate Mineral Nucleation</span> in <span>Biology</span> and <span>Geobiology</span>. <em>Calcified Tissue International</em>, <em>93</em>(4), 382–396. <a href="https://doi.org/10.1007/s00223-013-9784-9">https://doi.org/10.1007/s00223-013-9784-9</a>
</div>
<div id="ref-pearceConcomitantDeposition1987" class="csl-entry" role="doc-biblioentry">
Pearce, E. I. F., &amp; Sissons, C. H. (1987). The <span>Concomitant Deposition</span> of <span>Strontium</span> and <span>Fluoride</span> in <span>Dental Plaque</span>. <em>Journal of Dental Research</em>, <em>66</em>(10), 1518–1522. <a href="https://doi.org/10.1177/00220345870660100101">https://doi.org/10.1177/00220345870660100101</a>
</div>
<div id="ref-petersConstantDepth1988" class="csl-entry" role="doc-biblioentry">
Peters, A., &amp; Wimpenny, J. W. T. (1988). A <span>Constant-Depth Laboratory Model Film Fermenter</span>. In <em><span>CRC Handbook</span> of <span>Laboratory Model Systems</span> for <span>Microbial Ecosystems</span></em>. <span>CRC Press</span>.
</div>
<div id="ref-petersonViscoelasticityBiofilms2015" class="csl-entry" role="doc-biblioentry">
Peterson, B. W., He, Y., Ren, Y., Zerdoum, A., Libera, M. R., Sharma, P. K., van Winkelhoff, A.-J., Neut, D., Stoodley, P., van der Mei, H. C., &amp; Busscher, H. J. (2015). Viscoelasticity of biofilms and their recalcitrance to mechanical and chemical challenges. <em>FEMS Microbiology Reviews</em>, <em>39</em>(2), 234–245. <a href="https://doi.org/10.1093/femsre/fuu008">https://doi.org/10.1093/femsre/fuu008</a>
</div>
<div id="ref-Rbase" class="csl-entry" role="doc-biblioentry">
R Core Team. (2020). <em>R: <span>A</span> language and environment for statistical computing</em> [Manual]. <span>R Foundation for Statistical Computing</span>; <span>R Foundation for Statistical Computing</span>. <a href="https://www.R-project.org/">https://www.R-project.org/</a>
</div>
<div id="ref-roderStudyingBacterial2016" class="csl-entry" role="doc-biblioentry">
Røder, H. L., Sørensen, S. J., &amp; Burmølle, M. (2016). Studying <span>Bacterial Multispecies Biofilms</span>: <span>Where</span> to <span>Start</span>? <em>Trends in Microbiology</em>, <em>24</em>(6), 503–513. <a href="https://doi.org/10.1016/j.tim.2016.02.019">https://doi.org/10.1016/j.tim.2016.02.019</a>
</div>
<div id="ref-rohanizadehUltrastructuralStudy2005" class="csl-entry" role="doc-biblioentry">
Rohanizadeh, R., &amp; LeGeros, R. Z. (2005). Ultrastructural study of calculus–enamel and calculus–root interfaces. <em>Archives of Oral Biology</em>, <em>50</em>(1), 89–96. <a href="https://doi.org/10.1016/j.archoralbio.2004.07.001">https://doi.org/10.1016/j.archoralbio.2004.07.001</a>
</div>
<div id="ref-RmixOmics" class="csl-entry" role="doc-biblioentry">
Rohart, F., Gautier, B., Singh, A., &amp; Le Cao, K.-A. (2017). <span class="nocase">mixOmics</span>: <span>An R</span> package for ’omics feature selection and multiple data integration. <em>PLoS Computational Biology</em>, <em>13</em>(11), e1005752. <a href="http://www.mixOmics.org">http://www.mixOmics.org</a>
</div>
<div id="ref-AdapterRemovalv2" class="csl-entry" role="doc-biblioentry">
Schubert, M., Lindgreen, S., &amp; Orlando, L. (2016). <span>AdapterRemoval</span> v2: Rapid adapter trimming, identification, and read merging. <em>BMC Research Notes</em>, <em>9</em>, 88. <a href="https://doi.org/10.1186/s13104-016-1900-2">https://doi.org/10.1186/s13104-016-1900-2</a>
</div>
<div id="ref-shellisSyntheticSaliva1978" class="csl-entry" role="doc-biblioentry">
Shellis, R. P. (1978). A synthetic saliva for cultural studies of dental plaque. <em>Archives of Oral Biology</em>, <em>23</em>(6), 485–489. <a href="https://doi.org/10.1016/0003-9969(78)90081-X">https://doi.org/10.1016/0003-9969(78)90081-X</a>
</div>
<div id="ref-sissonsMultistationPlaque1991" class="csl-entry" role="doc-biblioentry">
Sissons, C. H., Cutress, T. W., Hoffman, M. P., &amp; Wakefield, J. S. J. (1991). A <span class="nocase">Multi-station Dental Plaque Microcosm</span> (<span>Artificial Mouth</span>) for the <span>Study</span> of <span>Plaque Growth</span>, <span>Metabolism</span>, <span class="nocase">pH</span>, and <span>Mineralization</span>: <em>Journal of Dental Research</em>. <a href="https://doi.org/10.1177/00220345910700110301">https://doi.org/10.1177/00220345910700110301</a>
</div>
<div id="ref-tettPrevotellaDiversity2021" class="csl-entry" role="doc-biblioentry">
Tett, A., Pasolli, E., Masetti, G., Ercolini, D., &amp; Segata, N. (2021). Prevotella diversity, niches and interactions with the human host. <em>Nature Reviews Microbiology</em>, <em>19</em>(9, 9), 585–599. <a href="https://doi.org/10.1038/s41579-021-00559-y">https://doi.org/10.1038/s41579-021-00559-y</a>
</div>
<div id="ref-tianUsingDGGE2010" class="csl-entry" role="doc-biblioentry">
Tian, Y., He, X., Torralba, M., Yooseph, S., Nelson, K. e., Lux, R., McLean, J. s., Yu, G., &amp; Shi, W. (2010). Using <span>DGGE</span> profiling to develop a novel culture medium suitable for oral microbial communities. <em>Molecular Oral Microbiology</em>, <em>25</em>(5), 357–367. <a href="https://doi.org/10.1111/j.2041-1014.2010.00585.x">https://doi.org/10.1111/j.2041-1014.2010.00585.x</a>
</div>
<div id="ref-tonjumNeisseria2017" class="csl-entry" role="doc-biblioentry">
Tønjum, T., &amp; van Putten, J. (2017). 179 - <span>Neisseria</span>. In J. Cohen, W. G. Powderly, &amp; S. M. Opal (Eds.), <em>Infectious <span>Diseases</span> (<span>Fourth Edition</span>)</em> (pp. 1553–1564.e1). <span>Elsevier</span>. <a href="https://doi.org/10.1016/B978-0-7020-6285-8.00179-9">https://doi.org/10.1016/B978-0-7020-6285-8.00179-9</a>
</div>
<div id="ref-velskoMicrobialDifferences2019" class="csl-entry" role="doc-biblioentry">
Velsko, I. M., Fellows Yates, J. A., Aron, F., Hagan, R. W., Frantz, L. A. F., Loe, L., Martinez, J. B. R., Chaves, E., Gosden, C., Larson, G., &amp; Warinner, C. (2019). Microbial differences between dental plaque and historic dental calculus are related to oral biofilm maturation stage. <em>Microbiome</em>, <em>7</em>(1), 102. <a href="https://doi.org/10.1186/s40168-019-0717-3">https://doi.org/10.1186/s40168-019-0717-3</a>
</div>
<div id="ref-velskoDentalCalculus2017" class="csl-entry" role="doc-biblioentry">
Velsko, I. M., Overmyer, K. A., Speller, C., Klaus, L., Collins, M. J., Loe, L., Frantz, L. A. F., Sankaranarayanan, K., Lewis, C. M., Martinez, J. B. R., Chaves, E., Coon, J. J., Larson, G., &amp; Warinner, C. (2017). The dental calculus metabolome in modern and historic samples. <em>Metabolomics</em>, <em>13</em>(11), 134. <a href="https://doi.org/10.1007/s11306-017-1270-3">https://doi.org/10.1007/s11306-017-1270-3</a>
</div>
<div id="ref-velskoAncientDental2022a" class="csl-entry" role="doc-biblioentry">
Velsko, I. M., Semerau, L., Inskip, S. A., García-Collado, M. I., Ziesemer, K., Ruber, M. S., Benítez de Lugo Enrich, L., Molero García, J. M., Valle, D. G., Peña Ruiz, A. C., Salazar-García, D. C., Hoogland, M. L. P., &amp; Warinner, C. (2022). Ancient dental calculus preserves signatures of biofilm succession and interindividual variation independent of dental pathology. <em>PNAS Nexus</em>, <em>1</em>(4), pgac148. <a href="https://doi.org/10.1093/pnasnexus/pgac148">https://doi.org/10.1093/pnasnexus/pgac148</a>
</div>
<div id="ref-warinnerPathogensHost2014" class="csl-entry" role="doc-biblioentry">
Warinner, C., Rodrigues, J. F., Vyas, R., Trachsel, C., Shved, N., Grossmann, J., Radini, A., Hancock, Y., Tito, R. Y., Fiddyment, S., Speller, C., Hendy, J., Charlton, S., Luder, H. U., Salazar-Garcia, D. C., Eppler, E., Seiler, R., Hansen, L. H., Castruita, J. A., … Cappellini, E. (2014). Pathogens and host immunity in the ancient human oral cavity. <em>Nature Genetics</em>, <em>46</em>(4), 336–344. <a href="https://doi.org/10.1038/ng.2906">https://doi.org/10.1038/ng.2906</a>
</div>
<div id="ref-weinerInfraredSpectroscopy2010" class="csl-entry" role="doc-biblioentry">
Weiner, S. (2010a). Infrared <span>Spectroscopy</span> in <span>Archaeology</span>. In <em>Microarchaeology: <span>Beyond</span> the <span>Visible Archaeological Record</span></em> (1st ed., pp. 275–316). <span>Cambridge University Press</span>. <a href="https://doi.org/10.1017/CBO9780511811210">https://doi.org/10.1017/CBO9780511811210</a>
</div>
<div id="ref-weinerBiologicalMaterials2010" class="csl-entry" role="doc-biblioentry">
Weiner, S. (2010b). Biological <span>Materials</span>: <span>Bones</span> and <span>Teeth</span>. In <em>Microarchaeology: <span>Beyond</span> the <span>Visible Archaeological Record</span></em> (pp. 99–134). <span>Cambridge University Press</span>.
</div>
<div id="ref-weinerStatesPreservation1990" class="csl-entry" role="doc-biblioentry">
Weiner, S., &amp; Bar-Yosef, O. (1990). States of preservation of bones from prehistoric sites in the <span>Near East</span>: <span>A</span> survey. <em>Journal of Archaeological Science</em>, <em>17</em>(2), 187–196. <a href="https://doi.org/10.1016/0305-4403(90)90058-D">https://doi.org/10.1016/0305-4403(90)90058-D</a>
</div>
<div id="ref-whiteDentalCalculus1997" class="csl-entry" role="doc-biblioentry">
White, D. J. (1997). Dental calculus: Recent insights into occurrence, formation, prevention, removal and oral health effects of supragingival and subgingival deposits. <em>European Journal of Oral Sciences</em>, <em>105</em>(5), 508–522. <a href="https://doi.org/10.1111/j.1600-0722.1997.tb00238.x">https://doi.org/10.1111/j.1600-0722.1997.tb00238.x</a>
</div>
<div id="ref-ggplot2" class="csl-entry" role="doc-biblioentry">
Wickham, H. (2016). <em>Ggplot2: <span>Elegant Graphics</span> for <span>Data Analysis</span></em>. <span>Springer-Verlag</span>. <a href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a>
</div>
<div id="ref-tidyverse2019" class="csl-entry" role="doc-biblioentry">
Wickham, Hadley, Averick, M., Bryan, J., Chang, W., McGowan, L. D., François, R., Grolemund, G., Hayes, A., Henry, L., Hester, J., Kuhn, M., Pedersen, T. L., Miller, E., Bache, S. M., Müller, K., Ooms, J., Robinson, D., Seidel, D. P., Spinu, V., … Yutani, H. (2019). Welcome to the <span class="nocase">tidyverse</span>. <em>Journal of Open Source Software</em>, <em>4</em>(43), 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>
</div>
<div id="ref-wongCalciumPhosphate2002" class="csl-entry" role="doc-biblioentry">
Wong, L., Sissons, C. H., Pearce, E. I. F., &amp; Cutress, T. W. (2002). Calcium phosphate deposition in human dental plaque microcosm biofilms induced by a ureolytic <span class="nocase">pH-rise</span> procedure. <em>Archives of Oral Biology</em>, <em>47</em>(11), 779–790. <a href="https://doi.org/10.1016/S0003-9969(02)00114-0">https://doi.org/10.1016/S0003-9969(02)00114-0</a>
</div>
<div id="ref-kraken2" class="csl-entry" role="doc-biblioentry">
Wood, D. E., Lu, J., &amp; Langmead, B. (2019). Improved metagenomic analysis with <span>Kraken</span> 2. <em>Genome Biology</em>, <em>20</em>(1), 257. <a href="https://doi.org/10.1186/s13059-019-1891-0">https://doi.org/10.1186/s13059-019-1891-0</a>
</div>
<div id="ref-yaoIdentificationProtein2003" class="csl-entry" role="doc-biblioentry">
Yao, Y., Berg, E. A., Costello, C. E., Troxler, R. F., &amp; Oppenheim, F. G. (2003). Identification of protein components in human acquired enamel pellicle and whole saliva using novel proteomics approaches. <em>J Biol Chem</em>, <em>278</em>(7), 5300–5308. <a href="https://doi.org/10.1074/jbc.M206333200">https://doi.org/10.1074/jbc.M206333200</a>
</div>
<div id="ref-zhouVeillonellaeBridging2021" class="csl-entry" role="doc-biblioentry">
Zhou, P., Manoil, D., Belibasakis, G. N., &amp; Kotsakis, G. A. (2021). Veillonellae: <span>Beyond Bridging Species</span> in <span>Oral Biofilm Ecology</span>. <em>Frontiers in Oral Health</em>, <em>2</em>, 774115. <a href="https://doi.org/10.3389/froh.2021.774115">https://doi.org/10.3389/froh.2021.774115</a>
</div>
</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    toggleBodyColorMode(window.document.querySelector("link#quarto-bootstrap"));
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>