Our biofilm setup consists of whole saliva as the inoculate to better approach
natural microbial communities within the human oral cavity, and a 24-well plate
to generate multiple replicated conditions in a single experimental run
(see @fig-protocol for an overview of the protocol).
The biofilm
is grown for 25 days to allow time for growth of larger deposits and mineralisation.

![Overview of the protocol for biofilm growth.](figures/ExpL_protocol.png){#fig-protocol}

To determine the composition of microbial communities, we sampled the medium
from the biofilm wells over the course of the experiment and sequenced the DNA
to identify species that are present in the model, and whether these mimic
natural oral communities. During a separate experimental run, under the same
conditions, we directly sampled the biofilms on multiple days and determined the
mineral composition using FTIR, and compared the spectra to those of natural
dental calculus, both modern and archaeological. Samples were taken on both controls
and starch treatments, but differences between these samples were not explored
in this study.

## Biofilm growth

<!-- Create table for all samples and indicate which samples underwent DNA, FTIR, etc. -->
```{r}
#| label: matmet-setup
#| include: false
library(dplyr)
library(stringr)
library(magrittr)
# env_summ <- metadata %>%
#   group_by(Env) %>%
#   count()
```

We employ a multispecies oral biofilm model following a modified
protocol from Sissons and colleagues [-@sissonsMultistationPlaque1991] and
Shellis [-@shellisSyntheticSaliva1978]. The setup comprises a polypropylene
24 deepwell PCR plate (KingFisher 97003510) with a lid containing 24 pegs (substrata),
which are autoclaved at 120&deg;C, 1 bar overpressure, for 20 mins.

The artificial saliva (hereafter referred to as medium) is a modified version of
the basal medium mucin (BMM)
described by Sissons and colleagues [-@sissonsMultistationPlaque1991].
It is a complex medium containing 2.5 g/l partially purified mucin from porcine
stomach (Type III, Sigma M1778),
5 g/l trypticase peptone (Roth 2363.1), 10 g/l proteose peptone (Oxoid LP0085),
5 g/l yeast extract (BD 211921), 2.5 g/l KCl, 0.35 g/l NaCl, 1.8 mmol/l CaCl~2~,
5.2 mmol/l Na~2~HPO~4~ [@sissonsMultistationPlaque1991], 6.4 mmol/l NaHCO~3~
[@shellisSyntheticSaliva1978], 2.5 mg/l haemin. This is subsequently
adjusted to pH 7 with NaOH pellets and stirring, autoclaved (15 min, 120&deg;C,
1 bar overpressure), and supplemented with 5.8 \(mu\)mol/l menadione, 5 mmol/l urea,
and 1 mmol/l arginine [@sissonsMultistationPlaque1991].

Fresh whole saliva (WS) for inoculation was provided by a 31-year-old male donor
with no history of caries, who abstained from oral hygiene for 24 hours, and no
food was consumed two hours prior to donation. No antibiotics were taken up to six
months prior to donation.
Saliva was stimulated by chewing on parafilm, then filtered through a bleach-sterilised
nylon cloth to remove particulates.
Substrata were inoculated with 1 ml/well of a two-fold dilution of WS in sterilised
20% glycerine for four hours at 36&deg;C, to allow attachment of the
salivary pellicle and plaque-forming bacteria. After initial inoculation, the
substrata were transferred to a new plate containing 1 ml/well medium and incubated
at 36&deg;C, with gentle motion at 30 rpm. The inoculation process was repeated
on days 3 and 5 by transferring the samples to a new plate with inoculate.
medium was partially refreshed once per day, by topping up the wells to the original
volume with more medium, and fully refreshed every three days,
throughout the experiment, by transferring the substrata to a new plate containing
medium. To feed the bacteria, the substrata were transferred to a new plate, containing
5% (w/v) sucrose, for six minutes twice daily, except on inoculation days
(days 0, 3, and 5), where the samples only received one sucrose treatment after
inoculation.

On day 9, starch treatments were introduced, replacing sucrose treatments (except
for control sample). As with the sucrose treatments, starch treatments occurred
twice per day for six minutes, and involved transferring the substrata to a
new plate containing a 0.25% (w/v) starch from potato (Roth 9441.1) solution, a
0.25% (w/v) starch from wheat (Sigma S5127) solution, and a 0.5% (w/v) mixture of
equal concentrations (w/v) wheat and potato.
All starch solutions were created in a 5%
(w/v) sucrose solution. Before transferring biofilm samples to the starch treatments,
the starch plates were agitated to keep the starches in suspension in the solutions,
and during treatments, the rpm was increased to 60.
The purpose of starch treatments is to explore the incorporation
of starch granules into the model calculus.
Starch treatments were initiated on day 9 (@fig-protocol) to avoid starch granule
counts being affected by $\alpha$-amylase hydrolysis from the inoculation saliva.
An $\alpha$-amylase
assay conducted on samples from days 3, 6, 8, 9, 10, 12, and 14 also showed that
there is no host salivary $\alpha$-amylase activity in the system.
The results of the starch incorporation and $\alpha$-amylase activity assay have
been reported in a separate article [@bartholdyInvestigatingBiases2022].

After 15 days, mineralisation was encouraged with a
calcium phosphate monofluorophosphate urea (CPMU) solution containing
20 mmol/l CaCl~2~, 12 mmol/l NaH~2~PO~4~, 5 mmol/l Na~2~PO~3~F, 500 mmol/l Urea
[@pearceConcomitantDeposition1987; @sissonsMultistationPlaque1991],
and (0.04 g/l MgCl). The substrata were submerged in 1 ml/well CPMU five times daily,
every two hours, for six minutes, at 30 rpm. During the mineralisation period,
starch treatments
were reduced to once per day, two hours after the last CPMU treatment. This cycle
was repeated
for 10 days until the end of the experiment on day 24
(@fig-protocol). A more detailed
protocol is available at                                                        <!-- protocols.io -->.

All laboratory work was conducted in sterile conditions under a laminar flow hood
to prevent starch and bacterial contamination. Starch-free control samples that
were only fed sucrose were included to detect starch contamination.

## Metagenomics

```{r}
#| label: tbl-dna-samples
#| tbl-cap: "Number of samples taken during the experiment, separated by sampling day and sample type."
dna_samples <- dna_experiment_metadata %>%
  #filter(Study == "this_study") %>%
  dplyr::select(`#SampleID`, Env, day) #%>%
# dna_samples %>%
#   knitr::kable(col.names = c("Sample ID", "Sample type", "Sampling day"))

dna_experiment_metadata %>%
  mutate(Env = factor(Env, levels = c("saliva", "medium", "model_calculus"))) %>% 
  group_by(Env, day) %>%
  count() %>%
  arrange(day, Env) %>% 
  knitr::kable(col.names = c("Sample type", "Sampling day", "n"))
```


<!-- Samples sent to Jena -->
A total of `r nrow(dna_samples)` samples were taken during the experiment from
the donated saliva, artificial saliva, and from the
biofilm end-product on day 24 (@tbl-dna-samples).
DNA extraction was performed at the archaeogenetic facility at the Max Planck
Institute for the Science of Human History (Jena, Germany), using the DNeasy
PowerSoil Kit from QIAGEN. C2 inhibitor removal step skipped, going directly
to C3 step.

The DNA was sheared to 500bp through sonication with a Covaris M220
Focused-ultrasonicator. Double-stranded libraries were prepared [@aronHalfUDG2020]
and dual indexed [@stahlDoublestrandedIndexing2019], with the indexing protocol
being adapted for longer DNA fragments. Briefly, the modifications consisted of
adding 3 μl of DMSO was added to the indexing reaction, and extending the
amplification cycles to 95°C for 60 s, 58°C for 60 s, and 72°C for 90 s. The
libraries were paired-end sequenced on a NextSeq 500 to 150bp.

<!-- EAGER settings -->
### Preprocessing

Processing of the raw DNA reads was conducted using the
`r paste(filter(software_versions, str_detect(software, "eager")))`
pipeline [@yatesEAGER2020]. The pipeline included adapter removal and read merging
using
`r paste(filter(software_versions, str_detect(software, regex("adapterremoval", ignore_case = T))))`
[@AdapterRemovalv2].
Merged reads were mapped to the human reference genome (GRCh38 <!--link-->)
using
`r paste(filter(software_versions, str_detect(software, regex("bwa", ignore_case = T))))`
[@BWA] with ancient DNA settings (-n 0.01; -l 32), and
unmapped reads were extracted using
`r paste(filter(software_versions, str_detect(software, regex("samtools", ignore_case = T))))`.
The final step of the pipeline, metagenomic classification, was conducted in
`r paste(filter(software_versions, str_detect(software, regex("kraken", ignore_case = T))))`
[@kraken2]
using the Standard 60GB database (https://genome-idx.s3.amazonaws.com/kraken/k2_standard_20220926.tar.gz).

Environmental reference samples were downloaded directly from ENA and from NCBI
using the SRA Toolkit.                                                            <!-- reference supp. mat.-->
Oral reference samples were downloaded from the Human Metagenome Project (HMP),
and modern calculus samples from Velsko et al. [-@velskoDentalCalculus2017].
From the HMP data, only paired reads were processed, singletons were removed.
*In vitro* biofilm model samples
from @edlundUncoveringComplex2018 were used as a reference.
Links to the specific
sequences, including human-filtered reads from this study, are included in the
metadata.                                                                       <!-- link to metadata file-->

### Authentication

Species with lower than 0.001% relative abundance across all samples were removed
from the species table.
**SourceTracker2** [@knightsSourceTracker2011] was used to estimate source composition
of the abundance-filtered oral biofilm model samples using a Bayesian framework.
Abundance-filtered
samples were compared with oral and environmental controls to detect potential
external contamination<!--(see Table S?)-->.
The R package **decontam** v`r packageVersion("decontam")` [@Rdecontam] was used
to identify potential contaminants in the abundance-filtered table
using DNA concentrations with a probability threshold of 0.95
and negative controls with a probability threshold of 0.05. Putative contaminants
were filtered out of the OTU tables for all downstream analyses.
Authentication methods are described in more
detail in the Supplementary material.<!--../06-reports/metagen-authentication.Rmd-->

### Community composition

Relative abundances of communities were calculated at the species- and genus-level,
as recommended for compositional data [@gloorMicrobiomeDatasets2017].
Shannon index and Pileou's evenness index was calculated on species OTU tables of all
model and oral reference
samples using the **vegan** v`r packageVersion("vegan")` R package [@Rvegan]. Shannon index
was calculated on all experimental samples to see if there is an overall loss or gain in
diversity and richness across the experiment.
Sparse principal component analysis (sPCA) was performed on model biofilm
samples to assess microbial composition between samples within the experiment,
and a separate sPCA analysis was performed on model calculus and oral
reference samples. The sPCA analysis was conducted using the **mixOmics**
v`r packageVersion("mixOmics")`
R package [@RmixOmics].  
The core microbiome was calculated by taking the mean genus-level relative abundance
within each sample type for model calculus, modern reference calculus,
sub- and supragingival plaque. Genera present at lower than 5% relative
abundance were grouped into the category 'other'.
Information on the oxygen tolerance of bacterial species was downloaded from 
BacDive [@reimerBacDive2022] and all variations of the major categories anaerobe,
facultative anaerobe, and aerobe were combined into the appropriate major category.
In species with no available information on oxygen
tolerance, the most common oxygen tolerance at the genus level was applied.

### Differential abundance

Differential abundance of species is calculated using the Analysis of Compositions
of Microbiomes with Bias Correction (ANCOM-BC) method from the
**ANCOMBC** R package v`r packageVersion("ANCOMBC")` [@linANCOMBC2020], with species-level OTU as input. Results
are presented as the log fold change of species between paired sample types with
95% confidence intervals. P-values are adjusted using the false
discovery rate (FDR) method. Samples are grouped by sample type (i.e. saliva,
plaque, modern calculus, model calculus). To supplement the sPCA analyses, we
plot the log fold change of the top 30 species in each of principal components 1
and 2, allowing us to see which species are enriched in the different samples and
causing clustering in the sPCA.

## FTIR

```{r}
byoc_ftir <- ftir_metadata %>%
  filter(source == "Artificial")
```

To determine the mineral composition and level of crystallisation of the model
dental calculus samples, we used Fourier Transform Infrared (FTIR) spectroscopy.
We compared the spectra of model dental calculus with spectra of archaeological
and modern dental calculus and used a built-in Omnic search library for
mineral identification
[@weinerInfraredSpectroscopy2010;@mentzerDistributionAuthigenic2014]. 
The archaeological dental calculus was sampled from an isolated permanent tooth<!--lower incisor???-->
from Middenbeemster, a rural, 19th century Dutch site [@lemmersMiddenbeemster2013].
Samples were analysed at the Laboratory for Sedimentary Archaeology, Haifa University.
The analysis was conducted with a Thermo Scientific Nicolet is5 spectrometer in
transmission, at 4 cm$^{-1}$ resolution, with an average of 32 scans between 4000
and 400 cm$^{-1}$ wavenumbers.

Analysis was conducted on `r nrow(byoc_ftir)` model calculus samples from days
7, 12, 16, 20, and 24. Some samples from the same sampling day had to be
combined to provide enough material for analysis. Samples analysed for FTIR
were grown during a separate experimental run conducted at a different time from the
experimental run growing the metagenomic samples, but following the same setup
and protocol (as described above). Samples were analysed following the method
presented in @asscherAtomicDisorder2011 and @asscherVariationsAtomic2011.
A few $\mu$g of each sample were repeatedly ground together with KBr and pressed
in a 7 mm die under two tons of pressure using a Specac mini-pellet press
(Specac Ltd., GS01152).
Repeated measurements of the splitting factor (SF) of the absorbance bands at 605
and 567 cm−1 wavenumbers were taken after each grind, and a
grind curve was produced following @asscherAtomicDisorder2011 to try and detect
changes in the hydroxyapatite crystallinity over time. Samples were
ground and analysed up to six times (sample suffix a-f) for the grinding curve.
Grinding curves were prepared for samples from days 16, 20, and 24. No grind
curves were produced for samples from days 7 and 12. These were largely composed
of organics and proteins, and did not form enough mineral (hydroxyapatite) for
analysis. The splitting factor of carbonate hydroxyapatite was calculated using
a macro script, following @weinerStatesPreservation1990. The calculation involves
dividing the sum of
the height of the absorptions at 603 cm$^{-1}$ and 567 cm$^{-1}$ by the height
of the valley between them. Following @asscherAtomicDisorder2011 and
@asscherVariationsAtomic2011, we plotted the
splitting factor against the full width at half maximum (FWHM) of the main absorption
at 1035-1043 cm$^{-1}$ to explore crystallinity (crystal size) and the order and
disorder of hydroxyapatite. We then compared our grinding curve slopes and FWHM
to the ones produced by @asscherVariationsAtomic2011. @asscherVariationsAtomic2011
and @asscherAtomicDisorder2011
demonstrated that while the decrease in FWHM of each grinding in the curve reflects
a decrease in particle size due to grinding, the location of the curves within a
plot of the FWHM against the splitting factor expresses the disorder effect.
Thus the curves with steeper slopes, higher splitting factor, and lower FWHM
represent lower levels of disorder in the mineral
[Figure 2 in @asscherVariationsAtomic2011].

## Statistics

<!-- R packages used -->
Statistical analysis was conducted in R Statistical Software `r version$version.string`
(`r version$nickname`) [@Rbase]. Data cleaning and wrangling performed with packages from
**tidyverse** [@tidyverse2019]. Plots were created using **ggplot2** [@ggplot2].
