## Metagenomic analysis

```{r}
#| label: results-metagen-setup
#| include: false

removed_samples <- dna_metadata %>%
  mutate(rm = if_else(`#SampleID` %in% dna_analysis_metadata$`#SampleID`, F, T),
         col = case_when(rm == T ~ "red",
                         rm == F ~ "black")) %>% # colour coded names if needed
  mutate(sample = `#SampleID`)

# summary of species counts
byoc_summ <- species_counts_long %>%
  filter(str_detect(sample, "SYN"),
         count > 0) %>%
  group_by(sample) %>%
  count(species) %>%
  summarise(count = sum(n))

# summary stats for alpha diversity in experiment
alpha_summ_byoc <- alpha_div_long %>%
  filter(
    str_detect(sample, "SYN") # isolate samples from this study
    ) %>%
  group_by(index, day_grouped) %>% # group by sample type
  summarise(mean = mean(value),
            sd = sd(value))


# summary stats for grouped samples by Env
alpha_summ_all <- alpha_div_long %>%
  filter(
    Env != "skin",
    Env != "sediment",
    Env != "stool",
    Env != "indoor_air") %>%
  group_by(index, Env) %>%
  summarise(mean = mean(value),
            sd = sd(value))



# byoc-spca-plot-setup
# set common ggplot theme for loading plots
loading_theme_pc1 <-  
  theme_minimal() +
  theme(
      panel.grid.major.x = element_blank(),
      panel.grid.minor.x = element_blank(),
      panel.grid.major.y = element_blank(),
      axis.ticks.x.bottom = element_line(size = 1),
      axis.text.y.left = element_text(size = 8, vjust = 0, face = "italic"),
      #axis.text.x = element_text(size = 8, vjust = 0),
      axis.line.x = element_line(),
      legend.position = "right"
          )

loading_theme_pc2 <-  
  theme_minimal() +
  theme(
      panel.grid.major.y = element_blank(),
      panel.grid.minor.y = element_blank(),
      panel.grid.major.x = element_blank(),
      axis.ticks.x.bottom = element_line(size = 1),
      #axis.text.x = element_text(face = "italic"),
      axis.text.x.top = element_text(size = 8, angle = 90, vjust = 0, hjust = 0, face = "italic"),
      axis.line.y = element_line(),
      legend.position = "none"
      )

aerotol_fill_pal <- scale_fill_manual(
  na.value = "grey50",
  drop = FALSE,
  values = viridisLite::inferno(
    n = length(levels(as.factor(bac_properties$`Oxygen tolerance`))),
    begin = 0.2,
    end = 0.8
    )
  )

aerotol_col_pal <- scale_colour_manual(
  na.value = "grey50",
  drop = FALSE,
  values = viridisLite::inferno(
    n = length(levels(as.factor(bac_properties$`Oxygen tolerance`))),
    begin = 0.2,
    end = 0.8
  )
)

options(ggplot2.discrete.colour = function() scale_colour_viridis_d(),
        ggplot2.discrete.fill = function() scale_fill_viridis_d())
# differential abundance
```

### Sample authentication

<!-- SourceTracker -->
To determine the extent of contamination in our samples, we performed a
source-tracking analysis using SourceTracker2 [@knightsSourceTracker2011]. <!-- supp. mat. figure -->
Results suggest that the majority of taxa across samples have an oral microbial
signature, and therefore our samples are minimally affected by external contamination (Figure S1).
We compared SourceTracker2 results to a database of oral taxa from
the cuperdec v`r packageVersion("cuperdec")` R package [@yatesOralMicrobiome2021]
to prevent removal of samples
where oral taxa were assigned to a non-oral source (Figure S2), as some taxa with a
signature from multiple sources are often classified as "Unknown"
[@velskoMicrobialDifferences2019]. We included several oral sources, which may
increase the risk of this occurring. Samples containing a large
proportion (>70%) of environmental
contamination were removed. The removed
samples were predominantly medium samples from later in the experiment, and a few
model calculus samples. After contaminated samples were removed,
suspected contaminant-species were removed from the remaining samples using the
decontam R package [@Rdecontam].
After contamination removal, samples consisted of between
`r paste(range(byoc_summ$count), collapse = " and ")`
species with a mean of
`r paste(round(mean(byoc_summ$count)))`.

### Decrease in community diversity across experiment

```{r}
shan_byoc <- filter(alpha_summ_byoc, index == "shannon") # isolate shannon index values
pielou_byoc <- filter(alpha_summ_byoc, index == "pilou_even")
richness_byoc <- filter(alpha_summ_byoc, index == "richness")
```


```{r}
#| label: fig-diversity-byoc
#| fig-cap: "Plot of Pielou Evenness Index, number of species, and Shannon Index across experiment samples grouped by sampling time. inoc = samples from days 0-5; treatm = samples from days 6-23; model = model calculus samples from day 24."

div_byoc_fig +
    theme_bw() +
    theme(
      axis.ticks.x = element_blank(),
      legend.position = "none"
    ) +
    labs(
      y = "Value",
      x = "Sampling time"
    )
```

<!-- species diversity -->
To monitor the development of microbial communities over the
course of the experiment, we used the Shannon Index to assess the species
diversity and richness at various stages of our protocol. Samples were grouped
into sampling categories due to low sample sizes on sampling days
(inoc = days 0, 3, 5; treatm = days 7, 9, 12, 15; model = day 24).
There was a slight decrease in mean Shannon Index between inoculation
and treatment samples, followed by a slight increase to model calculus samples,
as well as a decrease in variance within samples types.
The Pielou Evenness Index showed a similar pattern while the number of species
increased between the treatment period and the final model calculus (@fig-diversity-byoc).

### Medium and model calculus samples are distinct from the inoculate

```{r}
#| label: byoc-spca-fig-setup
byoc_spca_plot <- byoc_spca_base +
    labs(
      x = paste(
        "PC1", 
        scales::percent(byoc_explain_var[[1]], accuracy = 0.1)),
      y = paste(
        "PC2", 
        scales::percent(byoc_explain_var[[2]], accuracy = 0.1)),
      col = "Sampling day",
      shape = "Source"
      ) +
    theme_bw() +
    theme(
      legend.position = "left"
    ) +
    scale_x_continuous(position = "top") +
    #scale_y_continuous(position = "right") +
    scale_shape_discrete(solid = F, labels = byoc_source_labels) #+
    #scale_colour_viridis_d()

byoc_loadings_pc1 <- byoc_comp_1 +
    labs(x = "", y = "") +
    #scale_x_continuous(limits = c(min_byoc_PC1,max_byoc_PC1)) +
  #scale_y_discrete(position = "right") +
    aerotol_fill_pal +
    loading_theme_pc1

byoc_loadings_pc2 <- byoc_comp_2 +
    scale_y_continuous(limits = c(min_byoc_PC2,max_byoc_PC2), position = "right") +
    scale_x_discrete(position = "top") +
    labs(x = "", y = "") +
    aerotol_fill_pal +
    loading_theme_pc2
```

```{r}
#| label: fig-spca-byoc
#| fig-cap: "sPCA on species-level counts and oxygen tolerance in samples from this study only. Figure shows the main sPCA plot (A), species loadings on PC2 (B), and species loadings on PC1 (C)."
#| fig-width: 11
#| fig-height: 10
# align PC1 and PC2 loading plots with (0,0) on the PC1 and PC2 of the sPCA plot
ggdraw() +
    draw_plot(byoc_spca_plot, width = 0.56, height = 0.44, x = 0.01, y = 0.46) +
    draw_plot(byoc_loadings_pc2, width = 0.46, height = 0.48, x = 0.56, y = 0.48) +
    draw_plot(byoc_loadings_pc1, width = 0.69, height = 0.46, x = 0.06, y = 0.01) +
    draw_figure_label("A") +
    draw_figure_label("B", position = "top") +
    draw_figure_label("C", position = "bottom.left")

# ggdraw(byoc_spca_plot, xlim = c(1,1), ylim = c(2,2)) +
#     draw_plot(byoc_comp_1, width = 1, height = 1, x = 0.4, y = 0.1) +
#     draw_plot(byoc_comp_2, width = 0.5, height = 0.5, x = 0.1, y = 0.4)

#(byoc_comp_2 + byoc_spca_plot + byoc_comp_1) + 
#  plot_layout(design = plot_design, guides = "auto", widths = c(1,0.2,1), heights = c(1,0.2,1)) + plot_annotation(tag_levels = "A")
```

We next examined whether there is a change in the species composition over
time in our samples by assessing the beta-diversity in a PCA.
The species profiles of the saliva inoculate used in our experiment were
distinct from both medium and model calculus samples. Most of the separation
of saliva from model calculus is on PC1 of the sPCA, where
most of the positive sample loadings are driven by anaerobic species (model calculus),
especially *Selenomonas* spp, and
negative loadings are predominantly facultative anaerobes and some aerobes,
such as *Rothia* and *Neisseria* spp (saliva).
Medium and saliva are separated mostly on PC2, with medium samples located
between saliva and model calculus samples.
Model
calculus samples also cluster separately from the medium samples on PC2,
with some overlap between
the more mature medium samples and model calculus.
Most of the negative loadings separating saliva and model calculus
from medium samples are dominated by *Actinomyces* spp., while positive species
loadings are more diverse, and seemingly unrelated to aerotolerance (@fig-spca-byoc).

```{r}
byoc_da_pca <- byoc_logf_full %>%
  inner_join(byoc_pca_loadings, by = "species")
```


```{r}
#| label: fig-diffabund-byoc
#| fig-cap: "Log-fold changes between sample types. Circles are species enriched in the model calculus, squares are enriched in saliva, and triangles in medium. Lines are standard error. Plot shows the top 30 absolute log-fold changes between model calculus and saliva."
#| fig-width: 6
#| fig-height: 4
byoc_logf_base +
    theme_bw() +
    scale_shape_manual(values = c(1,2,5), labels = byoc_source_labels) +
    #scale_colour_viridis_d(option = "H", na.value = "grey50") +
    aerotol_col_pal +
    theme(
      axis.text.y = element_text(size = 8, face = "italic")
    ) +
    labs(x = "Log fold change", y = "", shape = "Sample type")

```

We determined whether there are species that are differentially abundant between
our sample types using the ANCOMBC R package [@linANCOMBC2020], giving
us an idea of how the biofilm develops under our experimental conditions.
Species enriched in saliva compared to model calculus are largely aerobic or
facultatively anaerobic, while
species enriched in model calculus compared to saliva are mainly anaerobes.
The differences between saliva and calculus are more pronounced than between
medium and model calculus, which is expected (@fig-diffabund-byoc).

### Lower diversity in artificial samples than oral references

```{r}
shan_compar <- alpha_summ_all %>% 
  filter(index == "shannon") # isolate shannon index values

shan_model <- shan_compar %>%
  filter(Env == "model_calculus" |
           Env == "medium" |
           Env == "vitro_biofilm")
  
shan_oral <- shan_compar %>%
  filter(Env != "model_calculus",
           Env != "medium",
           Env != "vitro_biofilm")

shan_saliva <- alpha_div_long %>%
  filter(Env == "saliva",
         index == "shannon") %>%
  group_by(Study) %>%
  summarise(mean = mean(value, na.rm = T),
            sd = sd(value, na.rm = T))
```

<!-- alpha diversity -->
<!-- convert to raincloud plot and combine figure with pie chart -->
```{r}
#| label: fig-shannon-compar
#| fig-cap: "Shannon Index for model calculus and medium samples, as well as oral reference samples and comparative *in vitro* study."
#| fig-height: 6
#| fig-width: 6
div_compar_fig +
    #facet_wrap(~ index, scales = "free_y") +
    theme_bw() +
    theme(
      axis.title.x = element_blank(),
      axis.title.y = element_blank(),
      axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1),
      axis.ticks.x = element_blank(),
      panel.grid.major.x = element_blank(),
      legend.position = "right"
    ) +
    guides(fill = "none", col = "none") +
  scale_x_discrete(labels = oral_source_labels) +
  scale_shape_discrete(
    #values = 1:4,
    labels = c("Edlund et al. 2018", "HMP", "this study", "Velsko et al. 2019")
    )
```

We used the Shannon Index to compare alpha-diversity in our model to oral
reference samples. The mean Shannon Index of model samples---medium, model calculus,
reference *in vitro* biofilm were consistently lower than the means of oral reference
samples---mucosa, modern reference dental calculus, saliva, and subgingival and
subgingival plaque. The Pielou species evenness index has a similar
distribution, although the comparative biofilm samples have a higher mean
than biofilm samples from this study. Saliva inoculate samples from this study
have a lower mean Shannon index than reference samples, which may have
contributed to the lower alpha-diversity in model samples compared to reference
samples. The number of species follows the same trend.

### Model calculus is distinct from dental calculus and other oral samples

```{r}
#| label: fig-core-genera
#| fig-cap: "Core genera within the different types of samples represented as mean relative abundances at the genus level. Other = other genera present in lower than 5% relative abundance."
#| fig-height: 5
# Pie chart of core genera
core_genera_fig +
    theme_void() +
    theme(
      legend.text = element_text(face = "italic")
    ) +
    scale_fill_viridis_d(na.value = "grey50", labels = c(genus_names, "Other")) + # janky solution...
    labs(fill = "Genus")
```

We calculated the mean relative abundances of the genera in each sample to compare
the core genera of model calculus with oral reference samples. The most
common genera (>5% relative abundance) are shown in @fig-core-genera.
The main overlap between the model calculus and oral reference
samples is the high relative abundance of *Streptococcus*.
Model calculus consists mostly of *Enterococcus* and
*Veillonella* spp., despite both having low abundance in donor saliva<!--supp mat-->.
*Enterococcus* are also known environmental contaminants, and we cannot exclude
environmental contamination as a possible source for these species in our model.
Oral reference samples have a more balanced composition, as they are also represented
by fastidious early-coloniser species like *Capnocytophaga* and *Neisseria*
spp., which require an environment with at least 5% carbon dioxide to thrive
[@tonjumNeisseria2017].

```{r}
#| label: compar-spca-fig-setup
# sPCA plot of PCs 1 and 2
compar_spca_plot <- compar_spca_base +
    scale_x_continuous(position = "top") +
    #scale_y_continuous(position = "right") +
    theme_bw() +
    theme(
      legend.position = "left"
    ) +
    labs(
      x = paste(
        "PC1", scales::percent(compar_explain_var[[1]], accuracy = 0.1)
      ),
      y = paste(
        "PC2", scales::percent(compar_explain_var[[2]], accuracy = 0.1)
      ),
      shape = "Source"
    ) +
    scale_colour_viridis_d(
      option = "C",
      begin = 0.3,
      end = 0.9,  
      labels = c(
        "other studies",
        "this study"
      )) +
    scale_shape_manual(values = 1:length(levels(as.factor(compar_princomp$Env))), 
                       labels = oral_source_labels
                       )

# loading plot PC1
compar_loadings_pc1 <- compar_comp_1 +
    labs(x = "", y = "") +
    aerotol_fill_pal +
    loading_theme_pc1

# loading plot PC2
compar_loadings_pc2 <- compar_comp_2 +
    labs(x = "", y = "") +
    aerotol_fill_pal +
    loading_theme_pc2
```

```{r}
#| label: fig-spca-compar
#| fig-cap: "sPCA on species-level counts from model calculus and reference samples. Figure shows (A) the main sPCA plot, (B) the species loadings from PC2, and (C) species loadings on PC1."
#| fig-width: 9
#| fig-height: 8
# align PC1 and PC2 loading plots with (0,0) on the PC1 and PC2 of the sPCA plot
ggdraw() +
    draw_plot(compar_spca_plot, width = 0.57, height = 0.44, x = 0.01, y = 0.46) +
    draw_plot(compar_loadings_pc2, width = 0.45, height = 0.48, x = 0.57, y = 0.52) +
    draw_plot(compar_loadings_pc1, width = 0.69, height = 0.45, x = 0.04, y = 0.01) +
    draw_figure_label("A") +
    draw_figure_label("B", position = "top") +
    draw_figure_label("C", position = "bottom.left")
```

To directly compare the beta-diversity of our model calculus with oral reference
samples, including modern dental calculus, we used an sPCA including only
our model calculus and reference samples.
Model calculus samples are distinct from both the oral reference samples and
the biofilm model reference samples. They are separated from oral reference samples
mainly on PC1, and from biofilm model reference samples (and, to some extent, oral
samples) on PC2. The highest
negative contributions are a mix of all types of aerotolerance, while the
positive contributions are mostly (facultative) anaerobes, with *Enterococcus* spp.
as the top three positive contributors to PC1. Top negative contributors are
*Capnocytophaga* spp as well as the aerobes *Corynebacterium matruchotii* and
*Rothia dentocariosa*. The top positive contributors to PC2 are all anaerobes,
mainly from the genus *Selenomonas*. Top negative contributors to PC2 are a mix of
aerotolerances, with many *Streptococcus* spp.

```{r}
#| label: comp-diffabund-fig-setup

# compar_logf_plot <- compar_logf_base +  
#     theme_bw() +
#     theme(axis.title = element_blank(),
#           axis.text.x = element_blank()
#           ) +
#     scale_shape_manual(values = c(1,2,5,7)) +
#     scale_colour_viridis_d(option = "H", na.value = "grey50")

compar_logf_pc1_plot <- compar_logf_pc1 +  
    theme_bw() +
    theme(
      axis.title = element_blank(),
      axis.text.x = element_blank(),
      axis.text.y = element_text(size = 8, face = "italic"),
      #legend.position = "none"
    ) +
    scale_shape_manual(values = c(1,2,5,7), labels = oral_source_labels) +
    #scale_colour_viridis_d(option = "H", na.value = "grey50", drop = FALSE) +
    aerotol_col_pal +
    labs(x = "Log fold change", y = "Species", shape = "Sample type")


compar_logf_pc2_plot <- compar_logf_pc2 +
    theme(
      axis.title.y = element_blank(),
      axis.text.y = element_text(size = 8, face = "italic"),
      strip.text = element_text(size = 8),
      legend.position = "none"
      ) +
    scale_shape_manual(values = c(1,2,5,7), labels = oral_source_labels) +
    scale_x_continuous(limits = layer_scales(compar_logf_pc1)$x$get_limits()) +
    #scale_colour_viridis_d(option = "H", na.value = "grey50", drop = FALSE) +
    aerotol_col_pal +
    labs(x = "Log fold change", y = "Species", shape = "Sample type")
```

```{r}
#| label: fig-diffabund-comp
#| fig-cap: "Log-fold changes between sample types. Circles are species enriched in the model calculus, triangles in modern calculus, diamonds are enriched in subgingival plaque, and squares in supragingival plaque. Plot shows the top 30 loadings (absolute value) in PC1 (A) and PC2 (B) between model calculus and other sample types, ordered by decreasing log-fold change. Bars represent standard error."
#| fig-width: 7
#| fig-height: 7

compar_logf_pc1_plot / compar_logf_pc2_plot + plot_annotation(tag_levels = "A") + plot_layout(guides = "collect")
```
<!-- you should set up that there are known differences in the species composition of saliva and calculus, so you want to see if your calculus samples are enriched in species that are enriched in natural calculus-->

To investigate which species are enriched in different sample types, and compare
the final product of our model with naturally occurring plaque and calculus samples,
we performed differential abundance analysis on our model calculus samples,
modern dental calculus, and sub- and supragingival plaque.
Based on the differential abundance analysis the main differences between model
calculus and oral reference samples, when looking at the top 30 contributors to
PC1, are that the oral reference samples are
enriched with species with a diverse oxygen tolerance from a wide range of genera,
while the model calculus is enriched with *Enterococcus* spp. The largest
differences occur in *Corynebacterium matruchotii*, *Rothia dentocariosa*, and
*Capnocytophaga gingivalis* ([@fig-diffabund-comp]A).
This is echoed when looking at the top 30 contributors to PC2,
where most of the species are enriched in model calculus, all of which are
anaerobes, and the largest differences occurring in *Cryptobacterium curtum*,
*Eggerthella lenta*, and *Mogibacterium diversum* ([@fig-diffabund-comp]B).

## Samples show an increased mineralisation over the course of the experiment

```{r}
#| label: results-ftir-setup
#| include: false

ftir_day7 <- ftir_day7 +
  theme(
    axis.title.x = element_blank()
  ) +
  labs(col = "Analysis ID")

ftir_day12 <- ftir_day12 +
  theme(
    axis.title.x = element_blank()
  ) +
  labs(col = "Analysis ID")

ftir_day16 <- ftir_day16 +
  theme(
    axis.title.x = element_blank()
  ) +
  labs(col = "Analysis ID")

ftir_day24 <- ftir_day24 +
  theme(
    axis.title.x = element_blank()
  ) +
  labs(col = "Analysis ID")

calc_compar <- calc_compar +
  labs(
    x = expression("wavenumber ("*cm^{-1}*")"),
    col = "Analysis ID"
  )

#ftir_data <- left_join(ftir_metadata, ftir_data_id, by = c("sample_id" = "sample"))

#ftir_data_long <- inner_join(ftir_metadata, ftir_data, by = c("analysis_id" = "sample"))
```

Comparing the development of the model calculus to the reference samples, it is
evident that between days 7 and 24 there is a decrease of the protein components
and increase of the inorganic mineral carbonate hydroxyapatite. The model calculus
samples from the end of the experiment are similar to both the modern and
archaeological reference samples. The main difference is a lower organic component
in reference samples seen as a reduced amide I peak at around 1637 compared to
the carbonate peak at around 1420, and an absence of amide II and III. Further,
there is a reduction in CH3 bands at 3000-2900 cm$^{-1}$ ([@fig-ftir-spectra]A-D).

Sample spectra from days 7 and 12 are characterised by a high content of proteins as evident
by the strong amide I absorbance band at 1650, a less pronounced amide II band
at 1545 cm$^{-1}$, and the small amide III band at 1237 cm$^{-1}$. Related
to the organic component of the samples are also the three marked CH~3~ and CH~2~
stretching vibrations at 2960, 2920, and 2850 cm$^{-1}$ wavenumbers. The presence
of mineral component is evident from the presence of C--O$^{2-}_3$ absorbance
bands at 1450 and 1400 cm$^{-1}$ wavenumbers typical of carbonates, and
P--O$^{3-}_4$ absorbance band at 1080 and 1056 cm$^{-1}$ which are related to
phosphate minerals. There is a large variation between the spectra, possibly
indicating different formation rates of the different components in the samples
([@fig-ftir-spectra]A and B).

In spectra from days 16 to 24, the ratio of amides to PO~4~ has shifted, with
the main peak shifting to the PO~4~ v~3~ absorbance band at 1039--1040 cm$^{-1}$,
indicating that the main component of
the samples is carbonate hydroxyapatite. A well-defined PO~4~ doublet at 600 and
560 is present. Small CO$_3^{2-}$ asymmetric stretching at 1450 cm$^{-1}$ and
1415 cm$^{-1}$, and stretching vibrations at 875-870 cm$^{-1}$ indicate that the
carbonate minerals component is also becoming more crystallised. There is a
decreased variability between the spectra, with most spectra exhibiting a higher
phosphate-to-protein/lipid ratio ([@fig-ftir-spectra]C and D).

## Model calculus has a similar mineral composition to natural calculus

To determine whether the model dental calculus is comparable to natural dental calculus,
both modern and archaeological dental calculus were analysed with FTIR spectroscopy to
ascertain their composition.
The archaeological and modern reference spectra are largely indistinguishable and
consist of a broad O–H absorbance band (3400 cm$^{-1}$) related to amid a and hydroxyl
group, weak CH3 bands (3000–2900 cm$^{-1}$), amide I band (1650 cm$^{-1}$) which is
related to the protein content, carbonate (1420, 1458-1450, 875-870 cm$^{-1}$),
and phosphates (1036-1040, 602-4, 563-566 cm$^{-1}$) ([@fig-ftir-spectra]E) which,
together with the hydroxyl and the carbonate, can be identified as derived from
carbonate hydroxyapatite, the main mineral found in mature dental calculus
[@hayashizakiSiteSpecific2008; @jinSupragingivalCalculus2002].

```{r}
#| label: fig-ftir-spectra
#| fig-cap: "Select spectra from all experiment sampling days; (A) day 7, (B) day 12, (C) day 16, and (D) day 24. Absorbance bands in stretching mode around 3400 cm−1 typical of the hydroxyl group. Analysis ID for model samples is constructed as: F[day sampled].[well sampled]_[grind sample]."
#| fig-width: 12
#| fig-height: 8

(ftir_day7 + (ftir_day12) + theme(axis.title.y = element_blank())) / (ftir_day16 + ftir_day24 + theme(axis.title.y = element_blank())) / calc_compar +
  plot_layout(nrow = 3) + plot_annotation(tag_levels = "A")
```

## Samples show similar crystallinity and order to reference calculus

<!-- FTIR spectra comparison of artificial and reference dental calculus -->
We determined the level of crystallinity and order of the carbonate hydroxyapatite
in our samples as an indication for its maturity by using the grinding curves method
presented by @asscherAtomicDisorder2011 and @asscherVariationsAtomic2011.  
Samples were compared to published trendlines for archaeological and modern enamel
[@asscherAtomicDisorder2011]. We see no appreciable differences between days 16,
20, and 24. The archaeological dental calculus shows a slightly increased slope
compared to model calculus from the three sampling days used in the grind curve
(@fig-grind-curve), possibly indicating larger crystal size due to more complete
crystalisation. The steeper slope of enamel samples is consistent with a more
ordered structure in enamel compared to dental calculus.


```{r message=FALSE, warning=FALSE}
#| label: fig-grind-curve
#| fig-cap: "Grinding curves of our biofilm and model calculus compared to published trendlines (dashed light grey lines) for archaeological (dotted line) and modern (dashed line) enamel."
# grind_all_plot + inset_element(grind_calc_plot, left = 0.4, bottom = 0.48, right = 1, top = 1.03) + 
#   plot_layout(guides = "collect") + plot_annotation(tag_levels = "A")
grind_all_plot + 
  scale_colour_manual(
    values = c(viridisLite::viridis(4), "grey60"),
    labels = c(
      "Archaeological calculus", 
      "Artificial calculus day 16", 
      "Artificial calculus day 20",
      "Artificial calculus day 24", 
      "Enamel (Asscher et al. 2011)",
      "And another one"
    )
  ) +
  guides(shape = "none", linetype = "none") +
  # scale_shape_manual(
  #   values = c(1:4, NA),
  #   labels = c(
  #     "Archaeological calculus", 
  #     "Artificial calculus day 16", 
  #     "Artificial calculus day 20",
  #     "Artificial calculus day 24", 
  #     "Enamel (Asscher et al. 2011)"
  #   )
  # ) +
  scale_linetype_manual(
    values = c("dashed", "dotted"),
    # limits = c(
    #   "Archaeological calculus",
    #   "Artificial calculus",
    #   "Enamel (Asscher et al. 2011)",
    #   "And another one"
    # ),
    labels = c(
      "Arch",
      "Model",
      "Modern enamel",
      "Arch enamel"
    )
  ) +
  #scale_x_continuous(limits = c(75,250)) +
  #scale_y_continuous(limits = c(2,6)) +
  labs(
    col = "Sample", shape = "Sample",
    #linetype = "Type",
    x = "FWHM of the 1035 peak",
    y = "Splitting factor"
  )
```

