---
title: "Supplementary Material: Assessing the validity of a calcifying oral biofilm model as a suitable proxy for dental calculus"
#author: "B.P.Bartholdy, I.M.Velsko, S.Gur-Arieh, Z.Fagernäs, C.Warinner, A.G.Henry" # should use _quarto metadata?

format:
  html: 
    toc: true
    code-fold: true
    output-file: "supplementary-material.html"
    df-print: paged
  pdf: 
    toc: true
    output-file: "supplementary-material.pdf"
    fig-pos: "H"
    knitr: 
      opts_chunk: 
        echo: false
execute: 
  warning: false
knitr: 
  opts_chunk: 
    collapse: true
    warning: false
    fig-path: "figures/supp-"
crossref:
  fig-prefix: "Supplementary Figure"
  tbl-prefix: "Supplementary Table"
number-sections: true
prefer-html: true
editor_options: 
  chunk_output_type: console
bibliography: references.bib
---

```{r}
#| label: setup
#| include: false
library(here)
library(dplyr)
library(tidyr)
library(forcats)
library(readr)
library(stringi)
library(ggplot2)
library(plotly)
library(cuperdec)
library(patchwork)
library(mixOmics)
#knitr::opts_knit$set(root.dir = here())

# upload ftir data

ftir_metadata <- read_tsv(here("01-documentation/ftir-metadata.tsv"))
ftir_data <- read_csv(here("05-results/FTIR/ftir_full-data.csv"))
ftir_grind_data <- read_csv(here("05-results/FTIR/ftir_grind-data.csv"))

# upload dna data

dna_metadata <- read_tsv(here("01-documentation/dna-metadata.tsv"))
dna_analysis_metadata <- read_tsv(here("01-documentation/dna-analysis-metadata.tsv"))
dna_experiment_metadata <- read_tsv(here("01-documentation/dna-experiment-metadata.tsv"))
lib_conc <- read_tsv(here("04-analysis/SYN_DNA_concentrations.tsv"))
software <- read_tsv(here("01-documentation/software_versions.csv"), col_names = c("software", "version"))
bac_properties <- read_tsv(here("04-analysis/bacdive/species-properties.tsv"))
genus_oxytol <- read_tsv(here("04-analysis/bacdive/genus-O2tolerance.tsv"))
list_of_contaminants <- read_tsv(here("05-results/metagenomics/list-of-contaminants.txt"), col_names = F)
otu_decontam <- read_tsv(here("05-results/metagenomics/post-decontam_taxatable.tsv"))
kraken_otu <- read_tsv(here("04-analysis/OTUfilter_table.tsv"))
load(here("05-results/metagenomics/spca_byoc.rda"))
load(here("05-results/metagenomics/spca_species.rda"))
alpha_div <- read_tsv(here("05-results/metagenomics/alpha-diversity.tsv"))
byoc_logf_full <- read_tsv(here("05-results/metagenomics/byoc_logf-full.tsv"))
plaque_logf_full <- read_tsv(here("05-results/metagenomics/plaque_logf-full.tsv"))
sourcetracker2 <- read_tsv(here("04-analysis/sourcetracker/sourcetracker2_output/mixing_proportions.txt"))
all_data_long <- read_tsv(here("04-analysis/sourcetracker/source-comb_long.tsv"))
clr_compar_long <- readr::read_tsv(here("05-results/metagenomics/clr-compar.tsv")) %>%
  pivot_longer(-sample, values_to = "clr_count", names_to = "species")
clr_byoc_long <- readr::read_tsv(here("04-analysis/beta-diversity/clr-byoc.tsv")) %>%
  pivot_longer(-sample, values_to = "clr_count", names_to = "species")
pca_loadings <- readr::read_tsv(here("05-results/metagenomics/all-pca-loadings.tsv"))

source(here("02-scripts/functions.R"))
source(here("02-scripts/OUT_setup-qmd.R"))
source(here("02-scripts/OUT_ftir-figures.R"))
source(here("02-scripts/OUT_metagen-figures.R"))
```

```{r}
#| label: extra-setup-computations

# helper object to organise byoc samples by day and whether or not they were removed during authentication
day_order <- dna_experiment_metadata %>%
  mutate(Env = factor(
    Env, 
    levels = c("saliva", "medium", "byoc_calculus") # force level order so it doesn't order alphabetically
    ) 
  ) %>% 
  group_by(Env) %>%
  arrange(day, .by_group = T) %>%
  filter(`#SampleID` %in% colnames(sourcetracker2)) %>%
  mutate(
    rm = if_else(`#SampleID` %in% dna_analysis_metadata$`#SampleID`, F, T),
    col = case_when(rm == T ~ "red", # excluded samples coloured red
                    rm == F ~ "black"))

```

## Introduction

Metagenomic analysis was conducted on our oral biofilm model to assess the
viability of the model as a proxy for dental calculus.
To determine the mineral composition and level of crystallisation of the model
dental calculus samples, we used Fourier Transform Infrared (FTIR) spectroscopy.

## Materials

Samples for DNA were collected
from a seperate experimental run than samples for DNA, but experimental conditions
were the same in each. Samples for DNA were medium collected from the bottom of
each well after three days of growth (before medium was refreshed). Samples for
FTIR were taken directly from the biofilm and dried prior to analysis.

```{r}
#| label: tbl-byoc-samples
#| tbl-cap: "Table of biofilm samples taken for metagenomic analysis. Includes which day in the experiment the sample was taken, and sample type (Env)."
dna_experiment_metadata %>% 
  #filter(str_detect(`#SampleID`, "LIB|SYN", negate = F)) %>% 
  dplyr::select(c(`#SampleID`, day, Env)) %>% 
  arrange(day, Env) %>% 
  knitr::kable()
```


```{r}
#| label: tbl-ftir-samples
#| tbl-cap: "Table of samples taken for FTIR analysis, including the day the sample was taken and what was sampled."
ftir_metadata %>% #| 
  filter(
    !is.na(analysis_id),
    source != "Modern"
  ) %>%
  dplyr::select(c(sample_id, day, source)) %>% 
  arrange(day) %>% 
  knitr::kable()
```


## Metagenomic analysis

### Reference database and sequences

The reference database used in the EAGER pipeline was the Standard Kraken 2 database,
downloaded from <https://genome-idx.s3.amazonaws.com/kraken/k2_standard_20210517.tar.gz>

```sh
wget https://genome-idx.s3.amazonaws.com/kraken/k2_standard_20210517.tar.gz -P 03-data/reference_genomes/
```

The human reference genome GRCh38 was downloaded on 2022-04-22.

```sh
wget ftp://ftp.ncbi.nlm.nih.gov/refseq/H_sapiens/annotation/GRCh38_latest/refseq_identifiers/GRCh38_latest_genomic.fna.gz -P 03-data/reference-genomes/
```

```{r}
#| label: tbl-oral-samples
#| tbl-cap: "Table of oral reference samples. Includes sample type (Env), and associated project and study with DOI."
dna_metadata %>% 
  filter(
    Study != "this_study",
    !Env %in% env_controls
  ) %>% 
  dplyr::select(
    !c(SourceSink, Study)
  ) %>% 
  arrange(Env) %>% 
  knitr::kable()
```

```{r}
#| label: tbl-env-samples
#| tbl-cap: "Table of environmental reference samples (contamination testing). Includes sample type (Env), and associated project and study with DOI."
dna_metadata %>% 
  filter(Env %in% env_controls) %>%
  dplyr::select(!c(SourceSink, Study)) %>% 
  arrange(Env) %>% 
  knitr::kable()
```


### Pre-processing

```{r}
#| label: tbl-lib-conc
knitr::kable(lib_conc)
```

Processing of the raw DNA reads was conducted using the
`r paste(filter(software, stri_detect(software, fixed = "eager")))`
pipeline [@yatesEAGER2020], run on the [ALICE HPC cluster](https://pubappslu.atlassian.net/wiki/spaces/HPCWIKI/overview?homepageId=10748192) in Leiden .

Separate batch scripts were created for artificial samples (*02-scripts/PRE_byoc-kraken.slurm*);
saliva, plaque, stool, and sediment samples (*01-documentation/source_samples_key.csv*, *02-scripts/PRE_source-kraken.slurm*);
indoor air and buccal mucosa samples (*02-scripts/PRE_indoor-mucosa-kraken.slurm*);
and *in vitro* biofilm samples from Edlund et al. [-@edlundUncoveringComplex2018] (*02-scripts/PRE_vitro-biofilms-kraken.slurm*).

Adapter removal and read merging was performed using
`r paste(filter(software, stri_detect(software, fixed = "adapterremoval", opts_fixed = list("case_insensitive" = TRUE))))`
[@AdapterRemovalv2]. Merged reads were mapped to the human reference genome,
GRCh38, using
`r paste(filter(software, stri_detect(software, fixed = "bwa", opts_fixed = list("case_insensitive" = TRUE))))`
[@BWA] with default settings (-n 0.01; -l 32), and unmapped reads were extracted
using
`r paste(filter(software, stri_detect(software, fixed = "samtools", opts_fixed = list("case_insensitive" = TRUE))))`.

Metagenomic classification was conducted in
`r paste(filter(software, stri_detect(software, fixed = "kraken", opts_fixed = list("case_insensitive" = TRUE))))`
using the Standard database (https://benlangmead.github.io/aws-indexes/k2).

Kraken output reports from the EAGER output were combined and converted to OTU tables, and only
species-level assignments were selected for downstream analysis
(*02-scripts/PRE_01_comb-kraken-reports.R*).
The OTU tables were further filtered by removing species with relative abundance
lower than 0.001% (*02-scripts/PRE_02-dataprep.R*).

### Authentication

#### SourceTracker2

SourceTracker2 [@knightsSourceTracker2011] was used to estimate source composition
of the oral biofilm model samples using a Bayesian framework. Samples were compared
with oral and environmental controls to detect potential external contamination.

The following steps were taken to authenticate sequences:

{{< include ../02-scripts/AUTH_01_ST2.md >}}

*02-scripts/AUT_02_decontam.R*

Plots were created of estimated contributions of various sources to the saliva,
model calculus and medium samples. Samples are arranged from left to right by how
late in the experiment they were sampled, with left being the earliest samples.
The output from SourceTracker2 was compared to a dataset
of known oral species (@fig-oral-sources).

Many of the later medium samples were assigned to the 'Unknown' and 'indoor_air'
categories. To see whether this was the result of external
contamination or the presence of oral taxa with an unknown origin (which could be
due to the taxa matching multiple sources), the taxa from 'Unknown' were compared
to the oral reference database from the **cuperdec** R package. Only samples with
a large proportion of oral taxa and a low proportion of indoor_air
(`indoor_air + Unknown < oral` and `oral > 70%`) were included.

{{< embed ../06-reports/metagen-authentication.qmd#fig-oral-sources >}}

A large proportion of the species assigned to the various sources are known to
be oral taxa. The samples do also seem to contain some external contamination.

Based on the results from SourceTracker, samples
`r paste(filter(day_order, rm == T)$'#SampleID', sep = ",")`
were removed from the analysis. The removed samples were all sampled late in
the experiment (day 18+).

Results from SourceTracker indicated a large portion of species from most samples
are from an oral origin (plaque, saliva, or calculus). Some samples contained a
large proportion of species from potential contaminants (indoor air) and of
unknown origin. Potential contaminants were compared to a database of oral
bacteria to see the proportion of known oral species were present in the samples.
Many of the species assigned to `indoor_air` are known oral species
(@fig-oral-sources).


#### decontam

Potential contaminants were identified using the frequency and prevalence method
in the **decontam** v`r packageVersion("decontam")` [@Rdecontam] R package. Samples
from *indoor_air*, *skin*, and *sediment* were used as negative controls for
the prevalence method with a probability threshold of 0.01. 
DNA concentrations (*04-documentation/SYN_DNA_concentrations.tsv*) were used
for the 'frequency' method with a probability threshold of 0.99
and negative controls.

Putative contaminants were filtered out of the
OTU tables (*05-results/post-decontam_taxatable.tsv*) for all downstream analyses.

See *06-reports/metagen-authentication.qmd* for more details.

```{r}
byoc_counts_long <- species_counts_long %>% 
  filter(
    stri_detect(sample, fixed = "SYN"),
    #sample %in% dna_analysis_metadata$`#SampleID`, # does not change dimensions
    count > 0
  )

byoc_species_summ <- byoc_counts_long %>%
  group_by(sample) %>%
  count(sample) %>%
  summarise(count = sum(n))
```

After running `isContaminant()` and filtering out (non-oral) potential contaminants,
the samples contained between
`r paste(range(byoc_species_summ$count), collapse = " and ")`
species with a mean of
`r paste(round(mean(byoc_species_summ$count), 2))` (@fig-species-counts).

{{< embed ../06-reports/metagen-authentication.qmd#fig-species-counts >}}


### Community composition

Genus- and species-level OTU tables were prepared from the decontaminated OTU
table, and relative abundance of species in a sample was calculated as recommended
for compositional data [@gloorMicrobiomeDatasets2017].

*02-scripts/DIV_01_alpha-beta.R*

```{r}
#| label: species-profiles-setup
#| include: false
# which species had no match on BacDive?

no_match <- unique(
  filter(species_properties, is.na(`Oxygen tolerance`))$species
)
```

#### Alpha-diversity

<!-- community composition: alpha diversity -->
Alpha-diversity, specifically the Shannon Index, was calculated to compare species
richness and diversity across experimental and comparative oral samples. Shannon
Index was calculated using the R package **vegan** v`r packageVersion("vegan")`
[@Rvegan].

##### Within experiment

Days are grouped to increase sample sizes:

- Inoculation (`inoc`) = days 0,3,5
- Treatment (`treatm`) = days 7,9,12,15
- End-product (`final`) = day 24


```{r}
#| label: alpha-div-setup
#| include: false

alpha_div_byoc <- filter(alpha_div_long, stri_detect(sample, fixed = "SYN"))

shan_byoc <- filter(alpha_div_byoc, index == "shannon")  # isolate shannon index values
pielou_byoc <- filter(alpha_div_byoc, index == "pilou_even")
richness_byoc <- filter(alpha_div_byoc, index == "richness")

shan_compar <- alpha_div_long %>% 
  filter(index == "shannon") # isolate shannon index values

shan_model <- shan_compar %>%
  filter(Env == "model_calculus" |
           Env == "medium" |
           Env == "vitro_biofilm")
  
shan_oral <- shan_compar %>%
  filter(Env != "model_calculus",
           Env != "medium",
           Env != "vitro_biofilm")

shan_saliva <- alpha_div_long %>%
  filter(Env == "saliva",
         index == "shannon") %>%
  group_by(Study) %>%
  summarise(mean = mean(value, na.rm = T),
            sd = sd(value, na.rm = T))

```

```{r}
#| label: tbl-alpha-experiment
#| tbl-cap: "Mean alpha diversity values of experiment samples, with standard deviation, grouped by index and days."
knitr::kable(alpha_summ_byoc)
```

There was a slight decrease in mean Shannon Index between inoculation
and treatment samples,
followed by a slight increase to model calculus samples,
as well as a decrease in variance within samples types.
The Pielou Evenness Index showed a similar pattern
while number of species increased between the treatment period and the final model
calculus (@fig-alpha-experiment and @tbl-alpha-experiment).
The model calculus is less variable than the initial and middle biofilm samples.

{{< embed ../06-reports/metagen-diversity.qmd#fig-alpha-experiment >}}

##### Compared to oral reference samples

We used the Shannon Index to compare alpha-diversity in our model to oral
reference samples. The mean Shannon Index of model
samples---medium, model calculus, reference *in vitro* biofilm 
were consistently lower than the means of oral reference samples---mucosa,
modern reference dental calculus, saliva, and subgingival and subgingival plaque.
The Pielou species evenness index has a similar
distribution, although the comparative biofilm samples have a higher mean
than biofilm samples from this study.
Saliva inoculate samples from this study
have a lower mean Shannon index than reference samples
which may have
contributed to the lower alpha-diversity in model samples compared to reference
samples (@fig-alpha-compar and @tbl-alpha-all).

Compared to other oral samples, there is lower species diversity and richness
in the medium and model calculus.


```{r}
#| label: fig-alpha-compar
alpha_div_long %>%
  filter(!Env %in% env_controls) %>%
  mutate(Env = case_when(stri_detect(Env, fixed = "plaque") ~ "plaque",
                         TRUE ~ Env)) %>%
  ggplot(aes(x = Env, y = value)) +
    geom_violin(aes(col = Env, fill = Env), alpha = 0.5) +
    geom_boxplot(width = 0.1) +
    facet_wrap(~ index, scales = "free_y") +
    theme_bw() +
    theme(
      axis.title = element_blank(),
      axis.text.x = element_blank(),
      axis.ticks.x = element_blank()
    )
```

```{r}
#| label: tbl-alpha-all
#| tbl-cap: "Mean alpha diversity values of experiment and comparative samples, with standard deviation, grouped by index and type (Env)."
alpha_div_long %>%
  filter(!Env %in% env_controls) %>%
  group_by(index, Env) %>%
  summarise(
    mean = mean(value),
    sd = sd(value)
  ) %>%
  arrange(desc(mean)) %>%
  knitr::kable()
```

#### Species profiles

```{r}
#| label: species-properties-plots

# Oxygen tolerance plots
oxy_tol <- species_properties %>%
  distinct(species, .keep_all = T) %>% 
  count(`Oxygen tolerance`) %>%
  mutate(perc = scales::percent(n / sum(n), accuracy = 0.1)) %>%
  ggplot(aes(x = "", y = n, fill = `Oxygen tolerance`)) +
    geom_col(colour = "white") + 
    coord_polar(theta = "y") +
    geom_text(aes(label = perc), position = position_stack(vjust = 0.5), col = "white") +
    theme_void() +
    scale_fill_manual(
      values = c(viridisLite::turbo(n = length(levels(as.factor(species_properties$`Oxygen tolerance`)))))
    )

byoc_oxytol <- species_properties %>% 
  filter(stri_detect(sample, fixed = "SYN")) %>% 
  ggplot(aes(y = sample, x = rel_abund, fill = `Oxygen tolerance`)) +
    geom_col() +
    theme_minimal() +
    scale_y_discrete(limits = rev(day_order$`#SampleID`)) +
    scale_fill_manual(
      values = viridisLite::turbo(n = length(levels(as.factor(species_properties$`Oxygen tolerance`))))
      ) +
  theme(
    axis.text.y = element_text(colour = rev(experiment_colours$env_col))
  )

# ABS plots
byoc_abs <- species_properties %>% 
  filter(stri_detect(sample, fixed = "SYN")) %>%
  ggplot(aes(y = sample, x = rel_abund, fill = abs)) +
    geom_col() +
    theme_minimal() +
    scale_y_discrete(limits = rev(day_order$`#SampleID`)) +
    scale_fill_viridis_d(option = "G") +
    theme(
      axis.text.y = element_blank(),
      axis.title = element_blank()
      )
```

<!-- species profiles -->

Information on the oxygen tolerance of bacterial species was downloaded from 
[BacDive](https://bacdive.dsmz.de) on
`r stri_extract(list.files("../03-data/", pattern = "bacdive"), regex = "\\d{4}-\\d{2}-\\d{2}")[length(list.files("../03-data/", pattern = "bacdive"))]`.
A total of **`r length(no_match)`** out of
**`r length(unique(species_counts_long$species))`**
(`r scales::percent(length(no_match) / length(unique(species_counts_long$species)), accuracy = 0.1)`)
species did not have a match on BacDive. These were assigned oxygen tolerance
based on the most common occurrence within the genus they belong.
A list of amylase-binding streptococci (ABS) was created based on
[@nikitkovaStarchBiofilms2013].

The most notable differences between model calculus and oral samples are the
lower proportion of ABS in and higher proportion of anaerobes in model calculus,
as well as the absence of aerobes (@fig-byoc-properties).

```{r}
#| label: fig-byoc-properties
#| fig-cap: "The distribution of oxygen tolerance (left) and ABS (right) in experimental samples."
byoc_oxytol + byoc_abs + plot_layout(guides = "collect")
```

#### Species composition

```{r}
#| label: spca-setup
#| include: false
# explained variance
byoc_explain_var <- spca_byoc$prop_expl_var$X # experiment samples
compar_explain_var <- spca_species$prop_expl_var$X # comparative samples

# loadings of samples
byoc_princomp <- spca_byoc$x %>% # experiment samples
  as_tibble(rownames = "sample") %>%
  left_join(dna_experiment_metadata, by = c("sample" = "#SampleID"))
spca_compar <- spca_species$x %>% # comparative samples
  as_tibble(rownames = "sample") %>%
  left_join(dna_metadata, by = c("sample" = "#SampleID"))

# loadings of species
exp_pca_loadings <- spca_byoc$loadings$X %>% # experiment samples
  as_tibble(rownames = "species") %>%
  #left_join(bac_properties, by = "species") %>% 
  left_join(species_properties, by = "species") %>% 
  dplyr::select(species,PC1,PC2, `Oxygen tolerance`) %>%
  arrange(desc(abs(PC1))) %>%
  distinct(species, .keep_all = T)

comp_pca_loadings <- spca_species$rotation %>% # comparative samples
  as_tibble(rownames = "species") %>%
  left_join(species_properties, by = "species") %>% 
  dplyr::select(species,PC1,PC2,PC3,`Oxygen tolerance`) %>%
  arrange(desc(abs(PC1))) %>% 
  distinct(species, .keep_all = T)
#exp_pca_loadings
```


Sparse principal components analysis (sPCA) was conducted on centered-log-ratio
(CLR) transformed species-level counts using the R package **mixOmics**
v`r packageVersion("mixOmics")` [@RmixOmics]. The **mixOmics** implementation of
sPCA uses a LASSO penalisation to eliminate unimportant variables.
Two separate analyses were conducted: 1) on experiment samples to assess the
difference in sample types and biofilm age; and 2) on oral comparative samples
and biofilm model end-products to explore community differences between *in vitro*
and *in vivo* biofilms.

```{r}
#| label: fig-scree
#| fig-cap: "Scree plot of principal components from the sPCA on experiment samples and comparative samples and model calculus."
#| fig-subcap: 
#|   - "Experiment samples"
#|   - "Comparative samples and model calculus"
plot(spca_byoc$prop_expl_var$X, type = "l", ylab = "Proportion explained variance")
plot(spca_species$prop_expl_var$X, type = "l", ylab = "Proportion explained variance")
```

The main overlap between the model calculus and oral comparative
samples are the high relative abundance of *Streptococcus*.
Model calculus consists mostly of *Enterococcus*
and *Veillonella* spp., while oral comparative samples are more
diverse (consistent with results from alpha diversity).

PC1 separates *in vivo* (negative) from *in vitro* (positive) oral samples.
PC2 separates the model calculus (positive) from the comparative *in vitro* biofilm
(negative). The positive PC1 loadings are dominated by *Enterococcus* spp.,
*Lactobacillus* spp., and aerobes. Negative PC1 loadings are mixed oxygen tolerant
and dominated by *Capnocytophaga* and *Neisseria* spp. Model calculus has a
unique signature on PC1 and PC2, driven largely by *Enterococcus* spp (@fig-spca-comp).


```{r}
#| label: fig-spca-comp
#| fig-cap: "sPCA of model calculus samples and oral reference samples. Contributions of species to each dimension are shown as directed arrows, where the length of the arrow is the weight of the contribution. Only shows correlations of 0.9 or higher."
#| fig-height: 7
#| fig-width: 7

biplot(
  spca_species,
  group = spca_compar$Env,
  comp = 1:2,
  cutoff = 0.90,
  ind.names = F, 
  var.names.size = 3
) +
  theme(legend.position = "bottom")
```

Loadings obtained from the sPCA analyses were used in combination with
CLR-transformed counts to create heatmaps of species- and genus-level counts
within the experiment and across comparative samples.

Counts from model biofilm and oral reference samples were transformed with a
centered ratio log-transform and ordered by PC1 loading.
*Enterococcus faecalis*, *Enterococcus casseliflavus*, and *Enterococcus durans*
are more abundant in the *in vitro* biofilm samples than
*in vivo* samples of plaque and calculus. Conversely, *Neisseria*, *Actinomyces*,
and *Capnocytophaga* spp. are deficient in the *in vitro* samples
(@fig-species-pos1 & @fig-species-neg1).


```{r}
#| label: fig-species-pos1
#| fig-cap: "Heat map of centered ratio log-transform abundances of the top 200 species with the highest positive loading on PC1. Light is more abundant."
#| fig-height: 12
#| fig-width: 9
# top positive loadings on PC1
species_pos_pc1 +
    scale_fill_viridis_c() +
    theme(
      axis.title.y = element_blank(),
      axis.title.x = element_blank(),
      axis.text.x = element_text(
        angle = 90, size = 8, vjust = 0.5, hjust = 1
        ),
      axis.text.y = element_text(face = "italic")
      )
```

```{r}
#| label: fig-species-neg1
#| fig-cap: "Heat map of centered ratio log-transform abundances of the top 200 species with the highest negative loading on PC1. Light is more abundant."
#| fig-height: 12
#| fig-width: 9
# Top negative loadings on PC1
species_neg_pc1 +
  scale_fill_viridis_c() +
  theme(
    axis.title.y = element_blank(),
    axis.title.x = element_blank(),
    axis.text.x = element_text(
      angle = 90, size = 8, vjust = 0.5, hjust = 1
    ),
    axis.text.y = element_text(face = "italic")
  )
```


### Differential abundance

*02-scripts/DIFF_01_lfc.R*

#### Within experiment

There seems to be a gradual decline in species counts rather than a sharp dropoff, supporting that
differences btw medium and calculus are related to maturation stage of biofilm [@velskoMicrobialDifferences2019; @fig-trend-clr; @fig-trend-clr2; @fig-trend-clr3].

```{r}
#| label: fig-trend-clr
#| fig-cap: "CLR-transformed counts of species over the duration of the experiment. Select species with highest 'drop-off' between medium and calculus"

top_da_species <- byoc_logf_full %>%
  filter(abn == "medium") %>%
  arrange(desc(lfc)) %>%
  distinct(species) %>%
  slice_head(n = 15) %>%
  .$species

clr_byoc_long %>%
  right_join(dna_experiment_metadata, by = c("sample" = "#SampleID")) %>%
  filter(species %in% top_da_species) %>%
  ggplot(aes(x = day, y = clr_count, col = species)) +
    geom_jitter(width = 0.2, show.legend = FALSE) +
    facet_wrap(~ species, ncol = 2) +
    theme_minimal()
```


```{r}
#| label: fig-trend-clr2
#| fig-cap: "CLR-transformed counts of species over the duration of the experiment. Select species with highest increase in the model calculus."

top_da_species2 <- byoc_logf_full %>%
  filter(abn == "byoc_calculus" |
           stri_detect(species, fixed = "Enterococcus")) %>%
  arrange(desc(lfc)) %>%
  distinct(species) %>%
  slice_head(n = 15) %>%
  .$species

top_species_counts <- clr_byoc_long %>%
  arrange(desc(clr_count)) %>%
  distinct(species) %>%
  slice_head(n = 15) %>%
  .$species

clr_byoc_long %>%
  right_join(dna_experiment_metadata, by = c("sample" = "#SampleID")) %>%
  filter(species %in% top_species_counts) %>%
  ggplot(aes(x = day, y = clr_count, col = species)) +
  geom_jitter(width = 0.2, show.legend = FALSE) +
  facet_wrap(~ species, ncol = 2) +
  theme_minimal()
```

```{r}
#| label: fig-trend-clr3
#| fig-cap: "CLR-transformed counts of species over the duration of the experiment. Select Enterococcus species."
clr_byoc_long %>%
  right_join(dna_experiment_metadata, by = c("sample" = "#SampleID")) %>%
  filter(species %in% top_da_species2) %>%
  ggplot(aes(x = day, y = clr_count, col = species, shape = Env)) +
  geom_jitter(width = 0.2, show.legend = FALSE) +
  facet_wrap(~ species) +
  theme_minimal()
```

Species enriched in saliva compared to model calculus are largely aerobic, while
species enriched in model calculus compared to saliva are mainly anaerobes [@fig-byoc-lfc].

{{< embed ../06-reports/metagen-diffabund.qmd#fig-byoc-lfc >}}

#### Comparative samples


{{< embed ../06-reports/metagen-diffabund.qmd#fig-plaque-lfc >}}

None of the species enriched in model calculus are aerobes. The top two species
(**Corynebacterium matruchotii** and **Rothia dentocariosa**)
enriched in plaque and calculus comparative samples are aerobes, while the rest
are more balanced between the various types of oxygen tolerance. The abundance
of anaerobes in model calculus compared to oral comparative samples is consistent
with the sPCA analysis [@fig-plaque-lfc].

```{r}
plaque_logf_pc1 <- plaque_logf_full %>%
  filter(species %in% comp_pca_loadings$species[1:30]) %>% 
  left_join(bac_properties, by = "species") %>% 
  #slice_head(n = 60) %>% 
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = `Oxygen tolerance`, shape = abn)) +
    geom_point() +
    geom_linerange(aes(xmin = lower, xmax = upper)) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +  
    theme_bw() +
    theme(axis.title = element_blank(),
          axis.text.x = element_blank()
          ) +
    scale_shape_manual(values = c(1,2,5,7))

plaque_logf_pc2 <- plaque_logf_full %>%
  filter(species %in% arrange(comp_pca_loadings, abs(PC2))$species[1:40]) %>% # not sure why I have to select 40 to get 30... I should look into that
  left_join(bac_properties, by = "species") %>% 
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = `Oxygen tolerance`, shape = abn)) +
    geom_point() +
    geom_linerange(aes(xmin = lower, xmax = upper)) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    theme_bw() +
    facet_wrap(~ env) +
    theme(axis.title.y = element_blank()) +
    scale_shape_manual(values = c(1,2,5,7)) +
    scale_x_continuous(limits = layer_scales(plaque_logf_pc1)$x$get_limits())
```

```{r}
#| label: fig-diffabund-plaque2
#| fig-cap: "Log-fold changes between sample types. Circles are species enriched in the model calculus, triangles in modern calculus, diamonds are enriched in subgingival plaque, and squares in supragingival plaque. Plot shows the top 30 species in PC1 (A) and PC2 (B) between model calculus and other sample types. Bars represent standard error."
#| fig-height: 8
#| fig-width: 7

plaque_logf_pc1 / plaque_logf_pc2 + plot_annotation(tag_levels = "A") + plot_layout(guides = "collect") & labs(shape = "Type") & theme(axis.text.y = element_text(size = 7))
```


### Discussion

<!-- from metagen-diversity -->
Main takeaway is the loss of diversity from donated saliva to model calculus, and
when compared to oral reference samples. The donated saliva for the experiment
had a lower diversity than the reference saliva samples, and may have contributed
to a lower diversity in experiment samples. Community profile of model calculus
differs from
the oral reference samples, while modern calculus mostly resembles oral reference
samples. The main difference between model and reference samples seems to be
a lack of aerobes in model samples and a dominance by *Enterococcus* spp.

<!-- from metagen-diffabund -->
*Enterococcus faecalis* has the highest log-fold change, with a higher abundance
in the model calculus samples compared to the reference samples and
may represent one of the main differences between model calculus and other
reference samples, especially modern calculus, consistent with the results of the
sPCA analysis. The high relative abundance of *E. faecalis* may represent
contamination despite it being commonly found in the oral cavity. It was also
abundant in the comparative *in vitro* biofilm study.
*Enterococcus* spp. out-competing other species may be a problem in
model biofilm studies, but more comparative studies are needed to confirm
(and there are not many WGS studies out there).
*Capnocytophaga*, *Actinomyces*, and *Neisseria* spp. deficient in model samples
(both this study and comparative).
Very few aerobes made it into the model. Apart from the donated saliva, very few
aerobes were detected in any of the experimental samples. *Rothia* spp. disappeared
between saliva and medium samples. We may need to reduce the frequency of
medium replacement (currently every three days) to help promote the
growth of slower growing organisms.

*Capnocytophaga* and *Actinomyces* spp. are
predominantly
(facultative) anaerobes, so their deficiency must be attributed to different
reasons. Both *Capnocytophaga* and *Neisseria* spp. are fastidious and
require an atmosphere with at least 5% carbon dioxide to thrive, so the model
may have a low carbon dioxide atmosphere [@tonjumNeisseria2017].

ABS are also underrepresented in the experimental samples
compared to the modern calculus and plaque reference samples. There was a
relatively low frequency of ABS in the model samples compared to the reference
samples, which may be attributed to the presence of sucrose in the treatment
solutions, potentially eliminating the niche for ABS.


## FTIR analysis

Select FTIR spectra not shown in the main manuscript (@fig-ftir-spectra). These can
also be explored in an interactive dashboard (<https://websaur.shinyapps.io/byocDashboard/>).


```{r}
#| label: fig-ftir-spectra
#| layout-ncol: 2
#| fig-height: 2.3
#| fig-cap: "FTIR spectra"
#| fig-subcap: 
#|   - "Day 7"
#|   - "Day 7"
#|   - "Day 12"
#|   - "Day 12"
#|   - "Day 16"
#|   - "Day 16"
#|   - "Day 20"
#|   - "Day 20"
#|   - "Day 24"
#|   - "Day 24"
ftir_spect_plot(ftir_data, "F7.2D1")
ftir_spect_plot(ftir_data, "F7.1A6")
ftir_spect_plot(ftir_data, "F12.1A5+F12.B1")
ftir_spect_plot(ftir_data, "F12.1D1+F12.1D2")
ftir_spect_plot(ftir_data, "F16.1B2")
ftir_spect_plot(ftir_data, "F16.1D6")
ftir_spect_plot(ftir_data, "F20.1A1")
ftir_spect_plot(ftir_data, "F20.1B5")
ftir_spect_plot(ftir_data, "F24.1B3")
ftir_spect_plot(ftir_data, "F24.2A4")
```


### Discussion

<!-- from 06-reports/FTIR-analysis.qmd -->

Experimental samples are indicative of biofilm growth and maturation, including
mineralisation, represented by a reduction in proteins and lipids (1460, 1420, 870)
and the increasing intensity of peaks related to carbonated hydroxyapatite (1040,
600-550 cm$^{-1}$) over the course of the experiment.

The presence of collagen in dental calculus (and saliva) is still not clear. It
may be a result of external contamination [@mackiePreservationMetaproteome2017]
or end-products
of collagen degradation (e.g. carboxyterminal telopeptide of type I collagen, ICTP)
which may or may not be related to periodontitis
[@gursoySalivaryCollagen2013]. It is more likely that the proline-rich proteins
(secondary amine) present in the samples are attributed to the biofilm
matrix.

The artificial samples from day 24 resemble both the modern reference samples and
the archaeological sample in mineral composition and crystallinity. The steeper
slope of the archaeological sample may suggest larger crystals, which is to be
expected, as the inorganic crystals expand into the space left by degrading
organic matter over time [@weinerBiologicalMaterials2010]. 
Overall, the spectra of the artificial calculus show an increase in the
inorganic component and a decrease in the organic component over time.
The grinding curves for days 16, 20, and 24 were very similar, suggesting
that the CPMU solution (introduced on day 16) may not have much of an effect.

The archaeological and modern reference spectra are largely indistinguishable and
consist of a broad O–H absorbance band (3400 cm$^{-1}$) related to hydroxyl group,
weak CH3 bands (3000–2900 cm$^{-1}$) , carbonate (1420, 1458-1450, 875-870 cm$^{-1}$),
amide I band (1650 cm$^{-1}$) which is related to the protein content, and phosphates
(1036-1040, 602-4, 563-566 cm$^{-1}$) which together withthe hydroxyl and the carbonate
can be identified as derived from carbonate hydroxyapatite, the main mineral found
in mature dental calculus (Hayashizaki et al., 2008; [@jinSupragingivalCalculus2002]).
The model
calculus samples from the end of the experiment are similar to both the modern and
archaeological reference samples. The main difference is a lower organic component
in reference samples seen as a reduced amide I peak at around 1637 compared to the
carbonate peak at around 1420, and an absence of amide II and III. Also a reduction
in CH3 bands at 3000-2900 cm$^{-1}$.


## Software versions

### EAGER

Software versions:

```{r}
software %>%
  knitr::kable(col.names = c("Software", "Version"))
```

### R

#### Session

```{r}
print(sessionInfo(), locale = F)
```

#### Packages

```{r}
#| label: tbl-list-packages
#| tbl-cap: "List of R packages and dependencies."
renv_packages_list <- jsonlite::read_json(here("renv.lock"))
do.call(rbind, renv_packages_list$Packages) %>% 
  as.data.frame() %>%
  dplyr::select(Package, Version, Source) %>%
  knitr::kable(row.names = F)
```


## References cited