<!-- Big summary/important statement for the science -->
In this study we present a [long-term] oral biofilm model with mineralisation
to produce artificial dental calculus. Our proposed use of the model is to address
a variety of research questions related to dietary information extracted from
dental calculus, in both modern and ancient samples.
For that to be feasible, the model needs to act as a viable proxy to
dental calculus grown under natural conditions, i.e., in the human mouth.
It needs, as much as possible, to mimic the diversity and complexity of the
natural oral microbiome, while also offering control over factors such as dietary
input, growth conditions, and replicability within and between experiments.
Here, we assessed the viability of our model as a proxy for dental calculus
using metagenomic classification and FTIR analysis to explore the bacterial
and mineral composition, and compare with oral reference samples.

## Microbiome

<!-- diversity and complexity within experiment and compared to oral references -->
There is a loss of diversity from donated saliva to model calculus, and
when compared to oral reference samples, which is a common limitation in
biofilm models [@edlundBiofilmModel2013; @bjarnsholtVivoBiofilm2013].
The donated saliva for the experiment
had a lower diversity than the reference saliva samples, and may have contributed
to a lower diversity in experiment samples. Consequently, there is also
a lower diversity and richness when compared to other modern oral reference samples,
including oral mucosa, saliva, plaque, and calculus<!--alpha-div figs-->.
Model calculus samples
primarily consist of species within the genera *Veillonella* and *Enterococcus*,
with more limited contributions from *Streptococcus* and *Prevotella*. The core
microbiomes of oral reference samples have similar levels of streptococci,
but a much more even distribution of species from *Actinomyces*, *Capnocytophaga*,
*Fusobacterium*, and more<!-- Figure ref to pie charts -->.

<!-- community profiles compared to oral references -->
Medium samples from early in the experiment have similar species profiles
to the donated saliva, but gradually diverge over the course of the experiment,
as seen in the increasing separation of samples on PC1 in Figure<!-- ref to sPCA figure -->.
This may be caused by the growth conditions provided by the experimental setup,
which may not sufficiently mimic the oral environment, allowing different oral
species to thrive that do not normally thrive in the natural oral environment.
Especially *Enterococcus* spp. seemed to thrive later in the experiment, at the
expense of more fastidious species like *Capnocytophaga* and *Neisseria* spp.,
which require an atmosphere with at least 5% carbon dioxide to thrive, so the model
may have a low carbon dioxide atmosphere [@tonjumNeisseria2017]. Early colonisers,
including the aforementioned *Capnocytophaga* and *Neisseria* spp., as well as
*Actinomyces*, were dominant in oral reference samples and deficient in model
calculus samples, as well as the comparative *in vitro* model.
Not only is there a decreased species diversity between model and reference samples,
but also a loss of diversity in aerotolerance. Of the dominant species in the model,
most are either anaerobes or facultative anaerobes, predominantly Gram-negative
*Veillonella* and *Prevotella*, and Gram-positive *Enterococcus*.
Species within predominantly aerobic genera, such as *Neisseria*, *Rothia*, and
*Ottowia*, are deficient in the model compared to the donated saliva
and other oral samples<!--PC loading plots-->, suggesting a shift from a largely
aerotolerant profile to an anaerobic profile during the experiment.
While our model is not set up as an anaerobic system, the anaerobes seem to
have outcompeted aerobes and, to some extent, facultative anaerobes.
This is likely a result of communities of bacteria within the biofilm creating
favourable microenvironments facilitated by the protective properties of the
biofilm matrix [@flemmingBiofilmsEmergent2016; @edlundUncoveringComplex2018].
*Capnocytophaga* and *Actinomyces* spp. are predominantly (facultative) anaerobes,
so their deficiency must be attributed to different reasons.
The highest log-fold changes
between model calculus and other samples were seen in *E. faecalis*, *E. durans*,
and *E. casseliflavus*, and contributed
most to the unique clustering of model calculus on PC1<!-- sPCA and DA figures-->.
*E. faecalis* was also
abundant in the comparative *in vitro* biofilm study<!-- large species plot-->.
*Enterococcus* spp. out-competing other species may be a problem in
model biofilm studies, but more comparative studies are needed to confirm
(and there are limited WGS studies to use as a comparison).

Model calculus and the modern reference calculus clustered separately in the sPCA
analysis. This may be due to human variation, as there can be large differences
in the oral microbiome of two individuals, and modern reference samples were taken
from the same geographical origin (as was the donor saliva for the model, which
was taken from a different geographical origin). One of the main drivers of
differences in species profiles has been previously shown to be individual
variation, even more so than pathological condition [@velskoAncientDental2022a].
Whether or not distinct microbial profiles will affect the retention of
dietary incorporation remains to be seen, and likely depends on the role bacteria
play in the incorporation and retention of dietary compounds and microremains
within a biofilm. Samples within the experiment were quite consistent allowing
this to be explored further.


Species related to caries, such as *Steptococcus mutans* and members of
the *Lactobacillus* genus, were deficient in both model calculus samples
and oral reference samples. Both are integral in creating an acidic environment
that favours demineralisation and inhibits mineralisation, so there absence is
expected in a calculus model
[@velskoMicrobialDifferences2019; @edlundBiofilmModel2013].
The abundance of *Veillonella* and *Prevotella* spp. could suggest a pathogenic
potential within the model biofilm, as these are commonly reported as opportunistic
or accessory pathogens in periodontal disease
[@zhouVeillonellaeBridging2021; @tettPrevotellaDiversity2021].

## Mineralisation

<!-- mineralisation process - FTIR - gradual mineralisation -->
FTIR analysis allowed us to address the mineralisation process of the model.
The results showed an increasing mineral composition over the course of the
experiment as the model biofilm matured.
Samples taken early in the experiment, on day 7 and 12, had a high lipid and
protein content consistent with the presence of microbial DNA and RNA within
a matrix of extracellular polysaccharides [@jainIsolationCharacterization2013].
The high water content, indicated by
the O--H stretch, is also consistent with a biofilm, which is around 90%
water [@bergerOralBiofilms2018]. As the samples mature over the course of the
experiment, the ratio of proteins
and lipids to phosphates shift from predominance of organic content to
inorganic content in the form of carbonated hydroxyapatite<!--fig reference-->.

The artificial samples from day 24 are consistent with spectra of carbonated
hydroxyapatite and resemble both the modern reference samples and
the archaeological sample in mineral composition and crystallinity. The steeper
slope in the grind curve plots of the archaeological sample may suggest that the
crystals in archaeological samples are larger than in the model calculus
[@fig-grind-curve]. A possible explanation is that
the inorganic crystals within archaeological calculus have had more time to expand
into the space left by degraded organic matter [@weinerBiologicalMaterials2010].
It may also have been an unusually crystallised sample; as we only analysed a
single archaeological sample, and more modern and archaeological samples are
needed to see if the high level of crystallinity is consistent in archaeological
calculus, and if the level of crystallinity of the model calulus is more comparable
to modern reference samples.
The grinding curves for days 16, 20, and 24 were very similar, suggesting
that the CPMU solution, which is introduced on day 16 to promote mineralisation,
may not have much of an effect.

Although there were differences in species profiles between modern and model
calculus, the mineral composition of the end results were similar, reinforcing
the idea that, under the right circumstances, most bacteria will mineralise
[@moorerCalcificationCariogenic1993]. Even occurring without one of the most
well-known biomineraliser, *Corynebacterium matruchotii*, which was deficient in model,
and was the strongest discriminator of oral samples from model samples<!--sPCA loadings fig-->.
It may also suggest that, while bacteria
are integral in the formation of dental calculus, and inevitably serve as part
of the structure that dental calculus is built upon [@rohanizadehUltrastructuralStudy2005],
the species composition of the biofilm communities may be less important. And that
mineralisation may largely be a chemical process (catalysed by biological factors)
[@omelonReviewPhosphate2013].

## Replicability
<!-- replicability within the experiment -->
The model calculus displayed similar species diversity and richness indices across
all samples, indicating a high level of replicability between samples in the
experimental run, despite larger variation between medium samples<!-- figure reference -->.
Both sPCA analyses conducted with experiment only samples and with oral
reference samples also cluster model calculus closely together.
It remains to be seen whether the replicability within the experiment
also scales up to between-experiment replicability.
The variation in mineral composition was initially high in early model biofilm
samples from days 7 and 12, but starting day 16 the variation decreased, and
continued to decrease until day 24. The model calculus samples from day 24
were largely similar in composition as observed in the FTIR spectra<!--fig ref-->.
The use of a simple multiwell plate setup allows us to [submit] many samples
to the same conditions which promotes replicability between samples<!--cite-->.
While the experimental conditions, including the dietary input, are kept constant,
and the mineral and bacterial profiles are similar between samples, there is
still a relatively large variation in starch retention in the model calculus
[@bartholdyInvestigatingBiases2022].

<!-- secondary findings -->

<!-- Scientific findings as extension of previous work -->
<!-- How this engineering will facilitate future research -->
There are many biofilm models to choose from, so developing a new protocol may
seem counter-productive; however, few are developed for long-term
growth and even fewer for mineralisation. One of the exceptions involves a complex setup
that is likely unfeasible to most archaeological research [@sissonsMultistationPlaque1991].
Our model uses a simplified high-throughput setup more commonly seen in shorter-term
biofilm models that mainly focus on dental plaque [@extercateAAA2010; @tianUsingDGGE2010].
As shown in a previous study, calculus and plaque have distinct microbial profiles
[@velskoMicrobialDifferences2019], so their applicability to explore archaeological
questions on dental calculus are limited. Clinical studies are also largely focused
on short-term effects of anti-microbial treatments and inhibition of biofilm
formation<!--citations-->, while archaeologists are more interested in questions
related to ... Questions, we believe, can to some extent be answered using an
oral biofilm model.
Our previous work has shown that, like dental calculus, the model calculus is
able to trap and retain dietary starches, admittedly with some variability
between samples [@bartholdyInvestigatingBiases2022].

## Limitations

A well-known limitation of biofilm models is the difficulty in capturing the
diversity and complexity of the natural oral biome. Diversity and complexity may
be represented as interspecies communities and complex metabolic dependencies between
organisms within the communitues, or as an environmental complexity determined by
nutrient availability, host immune-responses to biofilms, and fluctuating
microenvironments across the biofilm in response to these factors
[@edlundUncoveringComplex2018; @bjarnsholtVivoBiofilm2013].
Our single-donor approach may have affected diversity of model. The donated saliva
had a lower mean alpha-diversity than other saliva samples. Although benefit of
single donor means maintaining the integrity of the original bacterial community.
There are benefits and trade-offs for both pooled and single-donor saliva
[@edlundBiofilmModel2013].


<!-- Findings' scientific implications -->

<!-- Forward-looking statements about the field as a whole -->

<!-- a summary/importance of the engineering aspect of the work -->

<!-- How it facilitates scientific research -->

<!-- limitations of the platform show which questions would be unanswered -->

Research relating to $alpha$-amylase activity and its effect on the oral biome
(whether dietary or otherwise) will require addition of amylase to the model during
the experiment, as it is not present naturally in our current setup
(as shown in @bartholdyInvestigatingBiases2022).
Our model has no renewable source for $alpha$-amylase once the inoculations
have been completed. There are streptococci present in the model, which are known
for their ability to bind amylase [@nikitkovaStarchBiofilms2013]; however,
it is unclear whether the correct strains are present in our model, and if they are,
why they are not binding $alpha$-amylase provided by the saliva inoculate.
This could
be due to the frequency of medium replacement potentially clearing out all of the
unbound amylase every time medium is replaced.
Our current model does not contain serum, which may be
contributing to the deficiency of slow-growing fastidious organisms [@tianUsingDGGE2010].

Biofilms were grown in a standard shaking incubator, rather than a incubator specific
to cell cultures. The lack of complex environmental control may cause the model
to deviate from its natural growth over the 24 days that the experiment
is run, due to a lack of precise control over physiological conditions.

There is also a possibility of contamination in the model. While the CPMU solution was
prepared under sterile conditions, the solution itself was not autoclaved, and may
have been a source of contamination. This is further ... by the fact that none of
the medium samples taken after the introduction of CPMU (day 14) made it past
authentication in the metagenomic analysis, and that a large portion of the
sources of many samples were estimated to be from indoor air and surface samples
<!-- sourcetracker plot ref -->.

Another limitation involves the information on aerotolerance of the identified
species. Species-level aerotolerance was collected from BacDive<!--reference, url, etc.--> and,
at the time of writing<!--or rather, last time the manuscript was rendered-->,
`r scales::percent(length(no_match) / length(unique(species_counts_long$species)), accuracy = 0.1)`
species were missing aerotolerance values. This was mitigated by aggregating genus-level
tolerances to missing species, and may have some errors (although unlikely to make
any significant changes).

<!-- limitations to biofilm models -->

<!-- Is it possible that the mixture of sucrose in the starch solutions eliminates the need to feed on starch? -->

<!-- modern calculus references samples were all from the same geographic origin -->

<!-- How it will facilitate the specific topic of research (e.g. archaeology) -->

## Future work

Our goals for additional validation measures involve functional profiles of bacterial,
to see if metabolic behaviour
of bacteria is consistent with *in vivo* conditions, and whether this is
affected by the presence/absence of amylase.
Further protocol optimisation will also be necessary to address some of the
limitations of our current model. We may need to reduce the frequency of
medium replacement (currently every three days) to help promote the
growth of slower growing organisms and limit generalists such as enterococci.
More infrequent medium replacement would facilitate slow-growing bacteria in
establishing their metabolic relationships, allowing the byproducts of some species
to become abundant enough for others that depend on these to grow [@marshDentalPlaque2005].
Finally, if the within-experiment replicability is also present between
experiments.
An important question to address is what role bacteria play in the incorporation
and retention of dietary microremains, and whether differing bacterial profiles
affects the dietary incorporation and retention within the biofilm. Regardless,
the apparent replicability of our current setup is a promising result.

The calculus model we have presented has many potential benefits within archaeological
research. It can be used to test many fundamental assumptions we have about
the process of inclusion and subsequent extraction of various information about
health and diet from archaeological dental calculus.
How our current methods for exploring diet and health may be biased.
How different methods of food processing in the past may reflect the picture
we get from the samples we analyse.
This method can be used for a wide range of methods-testing (e.g. DNA, proteomics, etc.) 
as well as training for various sampling methods, and testing decontamination protocols,
without using up limited supplies of archaeological resources.

<!-- ultimately we are left with more questions than answers, but isn't that the 
beauty of exploratory science? -->
