# Materials and Methods

<!-- Create table for all samples and indicate which samples underwent DNA, FTIR, etc. -->

## Biofilm growth

In this study we employ a multispecies oral biofilm model following a modified
protocol from Sissons and colleagues [-@sissonsMultistationDentalPlaque1991] and
Shellis [-@shellisSyntheticSalivaCultural1978]. The setup comprises a polypropylene 
24 deepwell PCR plate (KingFisher 97003510) with a lid containing 24 pegs (substrata), which 
is autoclaved at 120&deg;C, 1 bar overpressure, for 20 mins.
 
The artificial saliva (AS) is a modified version of the basal medium mucin (BMM)
described by Sissons and colleagues [-@sissonsMultistationDentalPlaque1991]. 
It contains 2.5 g/l partially purified mucin from porcine stomach (Type III, Sigma M1778), 
5 g/l trypticase peptone (Roth 2363.1), 10 g/l proteose peptone (Oxoid LP0085),
5 g/l yeast extract (BD 211921), 2.5 g/l KCl, 0.35 g/l NaCl, 1.8 mmol/l CaCl~2~, 
5.2 mmol/l Na~2~HPO~4~ [@sissonsMultistationDentalPlaque1991], 6.4 mmol/l NaHCO~3~ 
[@shellisSyntheticSalivaCultural1978], 2.5 mg/l haemin. This is subsequently 
adjusted to pH 7 with NaOH pellets and stirring, autoclaved (15 min, 120&deg;C, 
1 bar overpressure), and supplemented with 5.8 \(mu\)mol/l menadione, 5 mmol/l urea, 
and 1 mmol/l arginine [@sissonsMultistationDentalPlaque1991].

<!-- add to protocols.io -->
Fresh whole saliva (WS) for inoculation was provided by a 31-year-old male donor 
with no history of caries, who abstained from oral hygiene for 24 hours, and no 
food was consumed two hours prior to donation. No antibiotics were taken up to six 
months prior to donation. 
The saliva was filtered through a sterilised (with bleach) nylon cloth to remove
particulates.
Substrata were inoculated with 1 ml/well of a two-fold dilution of WS in sterilised 
20% glycerine for four hours at 36&deg;C, to allow attachment of the
salivary pellicle and plaque-forming bacteria. After initial inoculation, the 
substrata were transferred to a new plate containing 1 ml/well AS and incubated 
at 36&deg;C, 30 rpm. The inoculation process was repeated on days 3 and 5. 
AS was partially refreshed once per day and fully refreshed every three days, 
throughout the experiment, by transferring the substrata to a new plate containing 
AS. To feed the bacteria, the substrata were transferred to a new plate, containing 
5% (w/v) sucrose, for six minutes twice daily, except on inoculation days 
(days 0, 3, and 5), where the samples only received one sucrose treatment after
inoculation. 

Starch treatments were initiated on day 9 to avoid starch granule counts being 
affected by $alpha$-amylase hydrolysis from inoculation saliva. 
Starch treatments replaced sucrose treatments, occurring twice per day
for six minutes. The starch treatments involved transferring the substrata to a 
new plate containing a 0.25% (w/v) starch from potato (Roth 9441.1) solution, a
0.25% (w/v) starch from wheat (Sigma S5127) solution, and a 0.5% (w/v) mixture of equal
concentrations (w/v) wheat and potato. All starch solutions were created in a 5%
(w/v) sucrose solution. Before transferring biofilm samples to the starch treatments, 
the starch plates were agitated to keep the starches in suspension in the solutions, 
and during treatments, the rpm was increased to 60.

After 15 days, mineralisation was encouraged with a 
calcium phosphate monofluorophosphate urea (CPMU) solution containing 
20 mmol/l CaCl~2~, 12 mmol/l NaH~2~PO~4~, 5 mmol/l Na~2~PO~3~F, 500 mmol/l Urea
[@pearceConcomitantDepositionStrontium1987; @sissonsMultistationDentalPlaque1991], 
and (0.04 g/l MgCl). The substrata were submerged in 1 ml/well CPMU five times daily, 
every two hours, for six minutes. During the mineralisation period, starch treatments
were reduced to once per day after the five CPMU treatments. This cycle was repeated 
for 10 days until the end of the experiment on day 24 
(see \@ref(fig:protocol-fig) for an overview of protocol). A more detailed
protocol is available at<!-- protocols.io -->.

All laboratory work was conducted in sterile conditions under a laminar flow hood 
to prevent starch and bacterial contamination. Control samples were included to 
detect starch contamination.

## Metagenomics

<!-- Samples sent to Jena -->
DNA extraction was performed at the archaeogenetic facility at the Max Planck 
Institute for the Science of Human History (Jena, Germany). Extractions were 
performed in duplicates <!-- ?? -->. A total of <!-- sample size --> DNA extracts.

<!-- DNA extraction details -->

were paired-end sequenced on a NextSeq (2 color chemistry) to 150bp

<!-- EAGER settings -->
### Preprocessing

Processing of the raw DNA reads was conducted in EAGER v. 2.4.1 [@yatesEAGER2020].
Adapter removal and read merging was performed using AdapterRemoval v. 2.3.2 [@AdapterRemovalv2]. 
Merged reads were mapped to the human reference genome (GRCh38 <!--link-->)
using BWA v. 0.7.17-r1188 [@BWA] with default settings (-n 0.01; -l 32).
<!-- SAMTools-->
<!-- endorS.py-->
<!-- Kraken -->
<!-- cuperdec [@yatesOralMicrobiome2021]-->
The R package *decontam* [@R-decontam] was used to identify potential contaminants
using library extraction blanks with a probability threshold of 0.1.

Metagenomic classification in Kraken 2 using the Standard database [@kraken2].

## FTIR

<!-- FTIR specs -->

Spectra were obtained using KBr and interpreted using <!-- insert reference library --> and modern calculus samples.

## Statistics

<!-- R packages used -->