---
title: "Differential abundance"
---

```{r}
#| label: setup
#| include: false
library(here)
library(readr)
library(dplyr)
library(ggplot2)
library(phyloseq)
library(microbiome)
library(ANCOMBC)
taxatable <- read_tsv(here("05-results/metagenomics/post-decontam_taxatable.tsv"))
analysis_metadata <- read_tsv(here("01-documentation/dna-analysis-metadata.tsv"))
experiment_metadata <- read_tsv(here("01-documentation/dna-experiment-metadata.tsv")) %>%
  filter(`#SampleID` %in% analysis_metadata$`#SampleID`)
pca_loadings <- read_tsv(here("05-results/metagenomics/all-pca-loadings.tsv"))
exp_pca_loadings <- read_tsv(here("05-results/metagenomics/experiment-pca-loadings.tsv"))
bac_properties <- read_tsv(here("04-analysis/bacdive/species-properties.tsv"))
byoc_logf_full <- read_tsv(here("05-results/metagenomics/byoc_logf-full.tsv"))
plaque_logf_full <- read_tsv(here("05-results/metagenomics/plaque_logf-full.tsv"))
```

Data are converted to `phyloseq` format. Experiment samples are first run on
their own, and then artificial calculus samples are run together with calculus
and plaque reference samples
(see *02-scripts/DIFF_01_abundance.R*).

## ANCOM-BC

Differential abundance is calculated using the ANCOM-BC method from the `ANCOMBC`
R package [@linANCOMBC2020]. P-values adjusted using the false discovery rate (FDR) method. Samples
grouped by sample type (i.e. saliva, plaque, modern calculus, artificial calculus).

First, differential abundance of experimental samples were calculated,
comparing abundance between donated saliva, medium, and the model calculus.
Then, differential abundance of oral reference samples and model calculus
was calculated.

### Across experiment

```{r}
#| label: tbl-byoc-diff
#| tbl-cap: "Top 20 largest absolute log-fold changes in artificial samples."
byoc_logf_full %>%
  arrange(desc(abs(lfc))) %>%
  slice_head(n = 20) %>%
  dplyr::select(species, lfc)#, `Oxygen tolerance`, abs)
```

```{r}
#| label: fig-byoc-lfc
#| fig-cap: "Top 20 largest absolute log-fold changes in artificial samples."
byoc_logf_full %>% 
  group_by(env) %>% 
  arrange(desc(abs(lfc))) %>%
  slice_head(n = 20) %>% # causes loss of some env values
  ungroup() %>%
  arrange(desc(abs(lfc))) %>%
  slice_head(n = 20) %>% # extract top 20 lfc from modern_calculus
  bind_rows(filter( # recombine with other env values so all are included in plot
    byoc_logf_full, 
    env != "saliva",
    species %in% .$species
    )
  ) %>%
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = abn)) +
    geom_point() +
    geom_linerange(aes(xmin = lower, xmax = upper)) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +
    theme_bw()
```

```{r}
#| label: fig-byoc-lfcPC1
#| fig-cap: "Log-fold changes for the 20 species in artificial samples with largest loading on PC1."
byoc_logf_full %>%
  filter(species %in% exp_pca_loadings$species[1:20]) %>% 
  slice_head(n = 60) %>% 
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = abn)) +
    geom_point() +
    geom_linerange(aes(xmin = lower, xmax = upper)) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +  
    theme_bw()
```

```{r}
#| label: fig-byoc-lfcPC2
#| fig-cap: "Log-fold changes for the 20 species in artificial samples with largest loading on PC2."
byoc_logf_full %>%
  filter(species %in% exp_pca_loadings$species[1:20]) %>% 
  slice_head(n = 60) %>% 
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = abn)) +
    geom_point() +
    geom_linerange(aes(xmin = lower, xmax = upper)) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +
    theme_bw()
```

Species with the highest log-fold change mainly see a reduction from saliva to
medium, and then a slight further reduction from medium to calculus. The largest
reduction of species occurs in
`r filter(byoc_logf_full, lfc == max(lfc))$species`
(LFC = `r filter(byoc_logf_full, lfc == max(lfc))$lfc`).

### model vs. modern calculus

```{r}
head(plaque_logf_full)
```

Plots of log-fold change between oral reference samples

```{r}
#| label: fig-plaque-lfc
#| fig-cap: "Log-fold changes between sample types. Circles are species enriched in the model calculus, triangles are enriched in modern calculus, diamonds in subgingival plaque, and squares in supragingival plaque. Plot shows the top 30 absolute log-fold changes between model calculus and other sample types. Bars within shapes are standard error."
plaque_logf_full %>%
  group_by(env) %>% 
  arrange(desc(abs(lfc))) %>%
    slice_head(n = 30) %>% # causes loss of some env values
    ungroup() %>%
  slice_head(n = 30) %>% # extract top 20 lfc from modern_calculus
  bind_rows(filter( # recombine with other env values so all are included in plot
    plaque_logf_full, 
    env != "modern_calculus",
    species %in% .$species
    )
  ) %>% 
  left_join(bac_properties, by = "species") %>% # combine with species properties
   ggplot(aes(x = lfc, y = reorder(species, lfc), col = `Oxygen tolerance`, shape = abn)) +
    geom_point(size = 2) +
    geom_linerange(aes(xmin = lower, xmax = upper), size = 1) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +
    theme_bw() +
    theme(
      #legend.position = "none",
      axis.title.y = element_blank()
          ) +
  scale_shape_manual(values = c(1,2,5,7))
```

```{r}
#| label: fig-ref-lfcPC1
#| fig-cap: "Log-fold changes for the 20 species in oral reference samples with largest loading on PC1."

plaque_logf_full %>%
  filter(species %in% pca_loadings$species[1:20]) %>% 
  slice_head(n = 60) %>% # pca_loadings already arranged by PC1
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = abn)) +
    geom_point(size = 2) +
    geom_linerange(aes(xmin = lower, xmax = upper), size = 1) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +
    theme_bw() +
    theme(
      legend.position = "none",
      axis.title.y = element_blank()
          )
```

```{r}
#| label: fig-ref-lfcPC2
#| fig-cap: "Log-fold changes for the 20 species in oral reference samples with largest loading on PC2."
plaque_logf_full %>%
  filter(species %in% arrange(pca_loadings, desc(abs(PC2)))$species[1:20]) %>%
  slice_head(n = 60) %>% # pca_loadings already arranged by PC1
  ggplot(aes(x = lfc, y = reorder(species, lfc), col = abn)) +
    geom_point(size = 2) +
    geom_linerange(aes(xmin = lower, xmax = upper), size = 1) +
    geom_vline(xintercept = 0, linetype = "dashed") +
    facet_wrap(~ env) +
    theme_bw() +
    theme(
      legend.position = "none",
      axis.title.y = element_blank()
          )
```


## Discussion

*Enterococcus faecalis* has the highest log-fold change, with a higher abundance
in the artificial calculus samples compared to the reference samples and
may represent one of the main differences between artificial calculus and other
reference samples, especially modern calculus, consistent with the results of the
sPCA analysis.
